---
title: "Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure"
author: "Jared Flater"
date: "2/15/2021"
output:
  html_document:
    toc: true
    toc_float:
      collapsed: true
      smooth_scroll: false
---

```{r libraries, include=FALSE}
library(tidyverse)
library(ggh4x)
library(viridis)
library(phyloseq)
library(phylosmith)
library(venn)
library(rstatix)
library(ggpubr)
library(kableExtra)
```

# Introduction

This code was used to generate figures for a publication titled: *Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure*

The manuscript is available here:

The project was designed to evaluate if the conservation practice of planting strips of prairie grasses within agricultural fields might attenuate the transport of manure associated genes and bacteria.

Poultry manure was applied to small plots appended with prairie strips and rainfall was simulated on the manure applied ground. Runoff water samples were collected after passing through the prairie strip and soil samples were collected before and after the rainfall simulation. In both soil and water samples we looked for and tracked the presence and abundance of manure associated bacteria and genes.

# Manure associated bacteria

## Figure 1

Figure was is a taxonomic profile of the bacteria present in the poultry manure that was used as a fertilizer. Manure was applied to six of nine plots, two plots received manure each day of a rainfall simulation. Rainfall simulations were performed on the nine plots over three days, one day each for a block of plots. Each block contained one plot from each treatment. The treatments were:

**Crop + Strip (CS)**

**Crop + Strip + Manure (CSM)**

**Crop + Manure (CM)**

The manure applied to each plot came from a single \~ 60 lbs batch of manure. When manure was applied to each plot, a sample of manure was taken to be analyzed for bacteria and antibiotic resistance genes. To track manure associated bacteria, we must first characterize the bacteria present. Here we have six manure samples and a bar plot representing the relative abundance of bacterial phyla in each sample. This data was generated using NGS sequencing of the 16S gene and analyzed using DADA2 to generate a count of the different taxa present in the various samples from the rainfall simulation.

```{r data import}
worle <- readRDS("../data/Worle_curated.RDS") # Worle data

# set taxa to ASV, otherwise uses the sequence as taxa name
taxa_names(worle) <- paste0("ASV", seq(ntaxa(worle)))

Tax.Table <- data.frame(tax_table(worle)) %>%
  replace(is.na(.), "Unclassified")

head(Tax.Table)

tax_table(worle) <- as.matrix(Tax.Table)
```

```{r subset to manure}
worle.manure <- subset_samples(worle, matrix == "manure") %>%
  filter_taxa(function(x) sum(x) >= 1, T)  # Remove taxa observed less than once in these samples
```

As we make plots of taxa, it's easier to understand if consistent colors are used for the various phyla. We can assign a color to each of these phyla then manually color plots based on this.

```{r phyla colors}
set.seed(010101)
phylalist <- data.frame(tax_table(worle), row.names = NULL) %>%
  select(Phylum) %>%
  unique() 
phylalist$Phylum <- as.character(phylalist$Phylum)


library(colorspace)  # this package will generate a palette based on number and desired colors

colors <- sequential_hcl(n_distinct(phylalist), palette = "viridis") %>%
  setNames(sort(phylalist$Phylum))

head(colors)

```

```{r figure 1 plots}
global_size = 12 # Font size
phydf <- worle.manure %>%
  tax_glom(taxrank = "Phylum") %>%
  transform_sample_counts(function(x) x / sum(x)) %>%
  psmelt()  # Take manure phyloseq object and glom taxa at phyla level. Transform counts to relative abundance and melt to a data frame. 

summary <- phydf %>%  # Make a plot
  group_by(Phylum) %>%
  summarise( 
    n=n(),
    mean=mean(Abundance),
    sd=sd(Abundance)
  ) %>%
  mutate(se = sd/sqrt(n))  %>%
  mutate(ic = se * qt((1-0.05)/2 + .5, n-1)) %>%
  arrange(-mean)

summary$Phylum <- factor(summary$Phylum, levels = summary$Phylum[order(-summary$mean)]) # arrange won't change order of Phylum factor levels

p1 <- ggplot(summary) +
  theme_classic() +
  geom_bar(aes(x = Phylum, y = mean, fill = Phylum), stat = "identity", 
           position = position_dodge(width = 0.75)) +
  geom_errorbar( aes(x = reorder(Phylum, -mean), ymin = mean - ic, ymax = mean + ic), width = 0.2, colour = "black", size= 0.5) +
  labs(y = "Relative abundance of phyla", x = "Phylum") +
  scale_fill_manual(values = colors) +
  theme(legend.position = "none", 
        axis.text.x = element_text(angle = 45, hjust = 1)) +
  scale_y_continuous(labels = function(x) paste0(x*100, "%")) +

  theme(text = element_text(size = global_size, family = "Times"))
p1
p2 <- ggplot(subset(summary, Phylum %in% tail(summary$Phylum, -5))) +
  theme_classic() +
  geom_bar(aes(x = reorder(Phylum, -mean), y = mean, fill = Phylum), stat = "identity", 
           position = position_dodge(width = 0.75)) +
  geom_errorbar( aes(x = reorder(Phylum, -mean), ymin = mean - ic, ymax = mean + ic), width = 0.2, colour = "black", size= 0.5) +
  labs(x = NULL, y = NULL) +
  scale_fill_manual(values = colors) +
  theme(legend.position = "none", 
        axis.text.x = element_text(angle = 45, hjust = 1)) +
  scale_y_continuous(labels = function(x) paste0(x*100, "%")) +

  theme(text = element_text(size = global_size, family = "Times"))

p2
p3 <- p1 + annotation_custom(ggplotGrob(p2), xmin = "Deinococcus-Thermus", xmax = "Deferribacteres", ymin = .05, ymax = .65) 
p3 
```

We can see that the manure is mostly represented by five phyla, Firmicutes, Proteobacteria, Bacteroidetes, Actinobacetria, and Deinococcus-Thermus.

```{r fig 1 save}
# Use ggsave to save plot. I chose 7" wide because most word docs are 8.5" with 2 x 1" margins. Keep the height below 9.5" to save room for a caption. When you insert into your document, change the size to 6.5" wide to ensure that the plot looks visually the same. 

ggsave(filename = "../Figures/Fig1.jpeg", plot = last_plot(), device = "jpeg", width = 7.0, height = 6.0, units = "in", dpi = 350)
```
