<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang=""><head>
  <meta charset="utf-8">
  <meta name="generator" content="quarto-0.2.323">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Jared Flater">
  <title>Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Litter - 16S</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      {   }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span.al { color: #ff0000; font-weight: bold; } /* Alert */
    code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
    code span.at { color: #7d9029; } /* Attribute */
    code span.bn { color: #40a070; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #4070a0; } /* Char */
    code span.cn { color: #880000; } /* Constant */
    code span.co { color: #60a0b0; font-style: italic; } /* Comment */
    code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
    code span.do { color: #ba2121; font-style: italic; } /* Documentation */
    code span.dt { color: #902000; } /* DataType */
    code span.dv { color: #40a070; } /* DecVal */
    code span.er { color: #ff0000; font-weight: bold; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #40a070; } /* Float */
    code span.fu { color: #06287e; } /* Function */
    code span.im { } /* Import */
    code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
    code span.kw { color: #007020; font-weight: bold; } /* Keyword */
    code span.op { color: #666666; } /* Operator */
    code span.ot { color: #007020; } /* Other */
    code span.pp { color: #bc7a00; } /* Preprocessor */
    code span.sc { color: #4070a0; } /* SpecialChar */
    code span.ss { color: #bb6688; } /* SpecialString */
    code span.st { color: #4070a0; } /* String */
    code span.va { color: #19177c; } /* Variable */
    code span.vs { color: #4070a0; } /* VerbatimString */
    code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
  </style>

  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <script src="site_libs/quarto-nav/quarto-nav.js"></script>
  <script src="site_libs/quarto-nav/headroom.min.js"></script>
  <script src="site_libs/clipboard/clipboard.min.js"></script>
  <meta name="quarto:offset" content="./">
  <script src="site_libs/quarto-search/autocomplete.umd.js"></script>
  <script src="site_libs/quarto-search/fuse.min.js"></script>
  <script src="site_libs/quarto-search/quarto-search.js"></script>
  <script src="site_libs/quarto-html/quarto.js"></script>
  <script src="site_libs/quarto-html/popper.min.js"></script>
  <script src="site_libs/quarto-html/tippy.umd.min.js"></script>
  <script src="site_libs/quarto-html/anchor.min.js"></script>
  <link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
  <link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet">
  <script src="site_libs/bootstrap/bootstrap.min.js"></script>
  <link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
  <link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet">
  <script id="quarto-search-options" type="application/json">{
    "location": "navbar",
    "copy-button": false,
    "collapse-after": 2,
    "panel-placement": "end",
    "type": "overlay",
    "limit": 20,
    "language": {
      "search-no-results-text": "No results",
      "search-matching-documents-text": "matching documents",
      "search-copy-link-title": "Copy link to search",
      "search-hide-matches-text": "Hide additional matches",
      "search-more-match-text": "more match in this document",
      "search-more-matches-text": "more matches in this document",
      "search-clear-button-title": "Clear",
      "search-detached-cancel-button-title": "Cancel",
      "search-submit-button-title": "Submit"
    }
  }</script>
  <link rel="stylesheet" href="styles/styles.scss">
</head>
<body class="docked">
<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-light ">
      <div class="navbar-container container-fluid">
      <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Litter</span>
  </a>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="http://germslab.org">GERMS Lab</a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title">16S</h1>
      <button type="button" class="quarto-btn-toggle btn">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation docked overflow-auto">
  <div class="sidebar-menu-container"> 
  <ul class="list-unstyled mt-1">
      <li class="sidebar-item">
  <a href="./16S.html" class="active">16S</a>
</li>
      <li class="sidebar-item">
  <a href="./ARGs.html" class="">ARGs</a>
</li>
      <li class="sidebar-item">
  <a href="./WaterFlow.html" class="">Water Flow</a>
</li>
  </ul>
  </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
      <nav id="TOC" role="doc-toc">
<h2 id="toc-title">On this page</h2>
<ul>
<li><a href="#figure-1" class="nav-link active" data-scroll-target="#figure-1">Figure 1</a></li>
<li><a href="#supp-figure-s2" class="nav-link" data-scroll-target="#supp-figure-s2">Supp Figure S2</a></li>
<li><a href="#figure-2" class="nav-link" data-scroll-target="#figure-2">Figure 2</a></li>
<li><a href="#figure-3" class="nav-link" data-scroll-target="#figure-3">Figure 3</a></li>
</ul>
</nav>
    </div>
<!-- main -->
<main class="content">
<header id="title-block-header">
<h1 class="title d-none d-lg-block display-7">16S</h1>
<p class="author">Jared Flater</p>
<p class="date">2/15/2021</p>
</header>

<p>Introduction</p>
<p>This code was used to generate figures for a publication titled: <em>Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Litter</em></p>
<p>The manuscript is available here:</p>
<p>The project was designed to evaluate if the conservation practice of planting strips of prairie grasses within agricultural fields might attenuate the transport of Litter associated genes and bacteria.</p>
<p>Poultry Litter was applied to small plots appended with prairie strips and rainfall was simulated on the Litter applied ground. Runoff water samples were collected after passing through the prairie strip and soil samples were collected before and after the rainfall simulation. In both soil and water samples we looked for and tracked the presence and abundance of Litter associated bacteria and genes.</p>
<p>Figure 1</p>
<p>Figure was is a taxonomic profile of the bacteria present in the poultry Litter that was used as a fertilizer. Litter was applied to six of nine plots, two plots received Litter each day of a rainfall simulation. Rainfall simulations were performed on the nine plots over three days, one day each for a block of plots. Each block contained one plot from each treatment. The treatments were:</p>
<p><strong>Crop + Strip (CS)</strong></p>
<p><strong>Crop + Strip + Litter (CSL)</strong></p>
<p><strong>Crop + Litter (CL)</strong></p>
<p>The Litter applied to each plot came from a single ~ 60 lbs batch of Litter. When Litter was applied to each plot, a sample of Litter was taken to be analyzed for bacteria and antibiotic resistance genes. To track Litter associated bacteria, we must first characterize the bacteria present. Here we have six Litter samples and a bar plot representing the relative abundance of bacterial phyla in each sample. This data was generated using NGS sequencing of the 16S gene and analyzed using DADA2 to generate a count of the different taxa present in the various samples from the rainfall simulation.</p>
<p>16S Figures</p>
<div class="cell" data-hash="16S_cache/html/data import_cc9b1a2a04e10cb8880dfd02f7811cf0">
<div class="sourceCode" id="cb1"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Worle data</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>worle <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data/Worle_curated.RDS"</span>) </span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="co"># set taxa to ASV, otherwise uses the sequence as taxa name</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">taxa_names</span>(worle) <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="st">"ASV"</span>, <span class="fu">seq</span>(<span class="fu">ntaxa</span>(worle)))</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co"># rename NA taxa as "Unclassified"</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>Tax.Table <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">tax_table</span>(worle)) <span class="sc">%&gt;%</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">replace</span>(<span class="fu">is.na</span>(.), <span class="st">"Unclassified"</span>)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="co"># replace tax table </span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="fu">tax_table</span>(worle) <span class="ot">&lt;-</span> <span class="fu">as.matrix</span>(Tax.Table)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>First, visualize differences in communities by plotting PCoA ordination of the Bray-Curtis dissimilarity</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-2_6f0aa816fbfb1159171cf0bec86287c9">
<div class="sourceCode" id="cb2"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(<span class="fu">sample_data</span>(worle))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code> [1] "unique_id"        "experiment"       "matrix"           "treatment"       
 [5] "plot"             "sample_day"       "depth"            "in_plot_location"
 [9] "block"            "strip"            "manure_treatment" "soil_type"       </code></pre>
</div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sample_data</span>(worle)<span class="sc">$</span>treatment[<span class="fu">is.na</span>(<span class="fu">sample_data</span>(worle)<span class="sc">$</span>treatment)] <span class="ot">&lt;-</span> <span class="st">"manure"</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>pcoa <span class="ot">&lt;-</span> <span class="fu">pcoa_phyloseq</span>(worle, <span class="fu">c</span>(<span class="st">'matrix'</span>, <span class="st">'treatment'</span>), <span class="at">circle =</span> T) <span class="sc">+</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>() <span class="sc">+</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Scale for 'fill' is already present. Adding another scale for 'fill', which
will replace the existing scale.</code></pre>
</div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>pcoa</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/unnamed-chunk-2-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb7"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/PCoA.jpeg"</span>, <span class="at">plot =</span> pcoa, <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="fl">7.0</span>, <span class="at">height =</span> <span class="fl">6.0</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="16S_cache/html/subset to manure_7c41e68f9cea8cf7a8194bcf227c1fb7">
<div class="sourceCode" id="cb8"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>worle.manure <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"manure"</span>) <span class="sc">%&gt;%</span> <span class="co"># subset to manure</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)  <span class="co"># Remove taxa observed less than once in these samples</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>As we make plots of taxa, it’s easier to understand if consistent colors are used for the various phyla. We can assign a color to each of these phyla then manually color plots based on this.</p>
<div class="cell" data-hash="16S_cache/html/phyla colors_358c248d7473549204a2bb548cffa356">
<div class="sourceCode" id="cb9"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">010101</span>)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="co"># make a list of unique phyla</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>phylalist <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">tax_table</span>(worle), <span class="at">row.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">unique</span>() </span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>phylalist<span class="sc">$</span>Phylum <span class="ot">&lt;-</span> <span class="fu">as.character</span>(phylalist<span class="sc">$</span>Phylum)</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(colorspace)  <span class="co"># this package will generate a palette based on number and desired colors</span></span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a><span class="co"># assign color from "viridis" package to each phyla and name the list with phyla names</span></span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a>colors <span class="ot">&lt;-</span> <span class="fu">sequential_hcl</span>(<span class="fu">n_distinct</span>(phylalist), <span class="at">palette =</span> <span class="st">"viridis"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">setNames</span>(<span class="fu">sort</span>(phylalist<span class="sc">$</span>Phylum))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="figure-1" class="level1">
<h1>Figure 1</h1>
<p>Let’s calculate the relative abundance of each phyla in the manure and generate summary statistics (mean, standard error, and interquartile range)</p>
<div class="cell" data-hash="16S_cache/html/figure 1 plots_d22a9f57d2462796c1bf424329805e86">
<div class="sourceCode" id="cb10"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>global_size <span class="ot">=</span> <span class="dv">12</span> <span class="co"># Font size</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>phydf <span class="ot">&lt;-</span> worle.manure <span class="sc">%&gt;%</span></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tax_glom</span>(<span class="at">taxrank =</span> <span class="st">"Phylum"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">transform_sample_counts</span>(<span class="cf">function</span>(x) x <span class="sc">/</span> <span class="fu">sum</span>(x)) <span class="sc">%&gt;%</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">psmelt</span>()  <span class="co"># Take manure phyloseq object and glom taxa at phyla level. Transform counts to relative abundance and melt to a data frame. </span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>summary <span class="ot">&lt;-</span> phydf <span class="sc">%&gt;%</span>  <span class="co"># Make a plot</span></span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>( </span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">n=</span><span class="fu">n</span>(),</span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean=</span><span class="fu">mean</span>(Abundance),</span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">sd=</span><span class="fu">sd</span>(Abundance)</span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">se =</span> sd<span class="sc">/</span><span class="fu">sqrt</span>(n))  <span class="sc">%&gt;%</span></span>
<span id="cb10-15"><a href="#cb10-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">ic =</span> se <span class="sc">*</span> <span class="fu">qt</span>((<span class="dv">1</span><span class="fl">-0.05</span>)<span class="sc">/</span><span class="dv">2</span> <span class="sc">+</span> .<span class="dv">5</span>, n<span class="dv">-1</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb10-16"><a href="#cb10-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="sc">-</span>mean)</span>
<span id="cb10-17"><a href="#cb10-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-18"><a href="#cb10-18" aria-hidden="true" tabindex="-1"></a>summary<span class="sc">$</span>Phylum <span class="ot">&lt;-</span> <span class="fu">factor</span>(summary<span class="sc">$</span>Phylum, <span class="at">levels =</span> summary<span class="sc">$</span>Phylum[<span class="fu">order</span>(<span class="sc">-</span>summary<span class="sc">$</span>mean)]) <span class="co"># arrange won't change order of Phylum factor levels</span></span>
<span id="cb10-19"><a href="#cb10-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-20"><a href="#cb10-20" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(summary) <span class="sc">+</span></span>
<span id="cb10-21"><a href="#cb10-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb10-22"><a href="#cb10-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">x =</span> Phylum, <span class="at">y =</span> mean, <span class="at">fill =</span> Phylum), <span class="at">stat =</span> <span class="st">"identity"</span>, </span>
<span id="cb10-23"><a href="#cb10-23" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.75</span>)) <span class="sc">+</span></span>
<span id="cb10-24"><a href="#cb10-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>( <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">ymin =</span> mean <span class="sc">-</span> ic, <span class="at">ymax =</span> mean <span class="sc">+</span> ic), <span class="at">width =</span> <span class="fl">0.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size=</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb10-25"><a href="#cb10-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="st">"Relative abundance of phyla"</span>, <span class="at">x =</span> <span class="st">"Phylum"</span>) <span class="sc">+</span></span>
<span id="cb10-26"><a href="#cb10-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colors) <span class="sc">+</span></span>
<span id="cb10-27"><a href="#cb10-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>, </span>
<span id="cb10-28"><a href="#cb10-28" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb10-29"><a href="#cb10-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb10-30"><a href="#cb10-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="at">x =</span> <span class="st">"Proteobacteria"</span>, <span class="at">y =</span> <span class="fl">0.60</span>, <span class="at">label =</span> <span class="st">"A"</span>, <span class="at">size =</span> <span class="dv">8</span>, <span class="at">family =</span> <span class="st">"Times"</span>) <span class="sc">+</span></span>
<span id="cb10-31"><a href="#cb10-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>)) </span>
<span id="cb10-32"><a href="#cb10-32" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb10-33"><a href="#cb10-33" aria-hidden="true" tabindex="-1"></a>p1</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/figure%201%20plots-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb11"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># We can make a second smaller plot to show the abundances of the low abundance phyla</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">subset</span>(summary, Phylum <span class="sc">%in%</span> <span class="fu">tail</span>(summary<span class="sc">$</span>Phylum, <span class="sc">-</span><span class="dv">5</span>))) <span class="sc">+</span></span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">y =</span> mean, <span class="at">fill =</span> Phylum), <span class="at">stat =</span> <span class="st">"identity"</span>, </span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.75</span>)) <span class="sc">+</span></span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>( <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">ymin =</span> mean <span class="sc">-</span> ic, <span class="at">ymax =</span> mean <span class="sc">+</span> ic), <span class="at">width =</span> <span class="fl">0.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size=</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="cn">NULL</span>, <span class="at">y =</span> <span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colors) <span class="sc">+</span></span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>, </span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb11-13"><a href="#cb11-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="at">x =</span> <span class="st">"Tenericutes"</span>, <span class="at">y =</span> <span class="fl">0.0060</span>, <span class="at">label =</span> <span class="st">"B"</span>, <span class="at">size =</span> <span class="dv">8</span>, <span class="at">family =</span> <span class="st">"Times"</span>) <span class="sc">+</span></span>
<span id="cb11-14"><a href="#cb11-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>)) </span>
<span id="cb11-15"><a href="#cb11-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-16"><a href="#cb11-16" aria-hidden="true" tabindex="-1"></a>p2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/figure%201%20plots-2.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb12"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> p1 <span class="sc">+</span> <span class="fu">annotation_custom</span>(<span class="fu">ggplotGrob</span>(p2), <span class="at">xmin =</span> <span class="st">"Deinococcus-Thermus"</span>, <span class="at">xmax =</span> <span class="st">"Deferribacteres"</span>, <span class="at">ymin =</span> .<span class="dv">05</span>, <span class="at">ymax =</span> .<span class="dv">65</span>) </span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>p3</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/figure%201%20plots-3.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>We can see that the manure is mostly represented by five phyla, Firmicutes, Proteobacteria, Bacteroidetes, Actinobacetria, and Deinococcus-Thermus.</p>
<div class="cell" data-hash="16S_cache/html/fig 1 save_23de30cfaa4b732a41bdfa3f2a1046af">
<div class="sourceCode" id="cb13"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Use ggsave to save plot. I chose 6.5" wide because most word docs are 8.5" with 2 x 1" margins. Keep the height below 9.5" to save room for a caption. When you insert into your document, change the size to 6.5" wide to ensure that the plot looks visually the same. </span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig1.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="fl">7.0</span>, <span class="at">height =</span> <span class="fl">6.0</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The specific abundances of phyla are:</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-4_5179bd089c435619ef6c84a53f32ba47">
<div class="sourceCode" id="cb14"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>phydf <span class="sc">%&gt;%</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">Mean =</span> <span class="fu">mean</span>(Abundance), <span class="at">StdDev =</span> <span class="fu">sd</span>(Abundance), <span class="at">min =</span> <span class="fu">min</span>(Abundance), <span class="at">max =</span> <span class="fu">max</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="sc">-</span>Mean)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 20 x 5
   Phylum                        Mean    StdDev       min       max
   &lt;chr&gt;                        &lt;dbl&gt;     &lt;dbl&gt;     &lt;dbl&gt;     &lt;dbl&gt;
 1 Firmicutes               0.555     0.0907    0.441     0.678    
 2 Proteobacteria           0.200     0.0668    0.124     0.294    
 3 Bacteroidetes            0.126     0.0483    0.0714    0.191    
 4 Actinobacteria           0.0890    0.0170    0.0669    0.105    
 5 Deinococcus-Thermus      0.0187    0.00770   0.00633   0.0261   
 6 Chloroflexi              0.00405   0.00262   0.00185   0.00842  
 7 Tenericutes              0.00173   0.00166   0.0000728 0.00455  
 8 Planctomycetes           0.000927  0.000411  0.000337  0.00137  
 9 Unclassified             0.000871  0.00116   0         0.00329  
10 Euryarchaeota            0.000706  0.00108   0         0.00289  
11 Verrucomicrobia          0.000526  0.000602  0         0.00158  
12 Acidobacteria            0.000353  0.000303  0         0.000705 
13 Synergistetes            0.000261  0.000548  0         0.00137  
14 Elusimicrobia            0.000195  0.000385  0         0.000962 
15 Fusobacteria             0.000153  0.000272  0         0.000670 
16 candidate_division_WPS-1 0.000110  0.000160  0         0.000407 
17 Cloacimonetes            0.0000796 0.000142  0         0.000351 
18 Armatimonadetes          0.0000506 0.0000659 0         0.000166 
19 Spirochaetes             0.0000281 0.0000688 0         0.000169 
20 Deferribacteres          0.0000123 0.0000302 0         0.0000740</code></pre>
</div>
</div>
</section>
<section id="supp-figure-s2" class="level1">
<h1>Supp Figure S2</h1>
<p>Abundance of manure associated taxa</p>
<p>Now let’s look at the number of unique taxa in the various samples.</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-6_35af4bc745bfa3c9cfb4b2ddea06317c">
<div class="sourceCode" id="cb16"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co"># strip subset, all strip soil from baseline samples in addition to no_manure_strip treatment, note the | operator </span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>worle.strip <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> sample_day <span class="sc">==</span> <span class="st">"Baseline"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"strip"</span> <span class="sc">|</span> </span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>                                matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"strip"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T) </span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>worle.crop <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> sample_day <span class="sc">==</span> <span class="st">"Baseline"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"crop"</span> <span class="sc">|</span> </span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>                                matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"crop"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a>worle.border <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> sample_day <span class="sc">==</span> <span class="st">"Baseline"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"border"</span> <span class="sc">|</span> </span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>                                matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"border"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-12"><a href="#cb16-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span>
<span id="cb16-13"><a href="#cb16-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-14"><a href="#cb16-14" aria-hidden="true" tabindex="-1"></a>worle.manure <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"manure"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-15"><a href="#cb16-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span>
<span id="cb16-16"><a href="#cb16-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-17"><a href="#cb16-17" aria-hidden="true" tabindex="-1"></a>worle.water <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"water"</span> <span class="sc">&amp;</span> unique_id <span class="sc">!=</span> <span class="st">"Comp-from-P8-7-10-26-17"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-18"><a href="#cb16-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-8_fbc50b8fa3a88c314b57f7cf2b87e97f">
<div class="sourceCode" id="cb17"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Make a list of ASVs associated with each type of soil and manure</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>worle_manure_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.manure)</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>worle_crop_soil_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.crop)</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>worle_strip_soil_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.strip)</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>worle_border_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.border)</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>worle_water_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.water)</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a><span class="co"># venn diagram</span></span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>worle_vvv_diag <span class="ot">&lt;-</span> <span class="fu">venn</span>(<span class="fu">list</span>(<span class="st">"Manure_ASVs"</span> <span class="ot">=</span> worle_manure_asvs, <span class="st">"Crop_ASVs"</span> <span class="ot">=</span> worle_crop_soil_asvs, <span class="st">"Strip_ASVs"</span> <span class="ot">=</span> worle_strip_soil_asvs, <span class="st">"Border_ASVs"</span> <span class="ot">=</span> worle_border_asvs))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb18"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Access ASVs unique to each soil and manure and store</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>worle_manure_persitors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersections"</span>)<span class="sc">$</span>Manure_ASVs</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>worle_crop_persistors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersections"</span>)<span class="sc">$</span>Crop_ASVs</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>worle_strip_persistors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersection"</span>)<span class="sc">$</span>Strip_ASVs</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>worle_border_persistors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersections"</span>)<span class="sc">$</span>Border_ASVs</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>We have 1017 “MAB” taxa</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-10_a377fa27d480152f48cff7842c185dc6">
<div class="sourceCode" id="cb19"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>Sample_Type <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Manure"</span>, <span class="st">"Crop"</span>, <span class="st">"Strip"</span>, <span class="st">"Border"</span>)</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>Number_of_taxa <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">ntaxa</span>(worle.manure), <span class="fu">ntaxa</span>(worle.crop), <span class="fu">ntaxa</span>(worle.strip), <span class="fu">ntaxa</span>(worle.border))</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>Taxa_unique_to <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">length</span>(worle_manure_persitors), <span class="fu">length</span>(worle_crop_persistors), <span class="fu">length</span>(worle_strip_persistors), <span class="fu">length</span>(worle_border_persistors))</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(Sample_Type, Number_of_taxa, Taxa_unique_to)</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>  Sample_Type Number_of_taxa Taxa_unique_to
1      Manure           1073           1017
2        Crop          19002           9548
3       Strip          18043           8518
4      Border           8140           1868</code></pre>
</div>
</div>
<p>Make a taxonomy table of the ASVs unique to manure</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-12_9ac74532643945ea51ab37b4849ad748">
<div class="sourceCode" id="cb21"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>MAB <span class="ot">&lt;-</span> <span class="fu">tax_table</span>(worle.manure) <span class="sc">%&gt;%</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>() <span class="sc">%&gt;%</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="st">"ASV"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(ASV <span class="sc">%in%</span> worle_manure_persitors)</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(MAB)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>     ASV  Kingdom         Phylum          Class           Order
1   ASV4 Bacteria     Firmicutes        Bacilli      Bacillales
2 ASV184 Bacteria     Firmicutes        Bacilli      Bacillales
3 ASV187 Bacteria     Firmicutes        Bacilli      Bacillales
4 ASV244 Bacteria     Firmicutes        Bacilli Lactobacillales
5 ASV246 Bacteria Actinobacteria Actinobacteria Actinomycetales
6 ASV269 Bacteria     Firmicutes        Bacilli      Bacillales
              Family           Genus
1      Bacillaceae_2    Unclassified
2      Bacillaceae_2  Oceanobacillus
3      Bacillaceae_2    Unclassified
4    Enterococcaceae    Enterococcus
5 Corynebacteriaceae Corynebacterium
6      Bacillaceae_2    Unclassified</code></pre>
</div>
<div class="sourceCode" id="cb23"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">write.table</span>(MAB, <span class="at">file =</span> <span class="st">"Tables/MAB.csv"</span>, <span class="at">sep =</span> <span class="st">","</span>, <span class="at">col.names =</span> T,</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>            <span class="at">qmethod =</span> <span class="st">"double"</span>, <span class="at">row.names =</span> F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-14_7f43de9f06031d3a9b464e606cf0c7f6">
<div class="sourceCode" id="cb24"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggvenn</span>(<span class="fu">list</span>(<span class="st">"Manure"</span> <span class="ot">=</span> worle_manure_asvs, <span class="st">"Crop"</span> <span class="ot">=</span> worle_crop_soil_asvs, <span class="st">"Strip"</span> <span class="ot">=</span> worle_strip_soil_asvs, <span class="st">"Border"</span> <span class="ot">=</span> worle_border_asvs), <span class="at">set_name_size =</span> <span class="dv">4</span>, <span class="at">show_percentage =</span> F) <span class="sc">+</span> <span class="fu">scale_fill_viridis_d</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Scale for 'fill' is already present. Adding another scale for 'fill', which
will replace the existing scale.</code></pre>
</div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb26"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"Figures/SuppFig2.png"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">width =</span> <span class="fl">6.5</span>, <span class="at">dpi =</span> <span class="dv">300</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Saving 6.5 x 5 in image</code></pre>
</div>
</div>
<p>We will assign taxa to two groups, soils associated and manure associated bacteria: “SAB” and “MAB”</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-16_c11a781db88cb72b268aeeec2bd3767b">
<div class="sourceCode" id="cb28"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>tax_association <span class="ot">&lt;-</span> <span class="fu">tax_table</span>(worle.water) <span class="sc">%&gt;%</span> </span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>() <span class="sc">%&gt;%</span></span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="st">"ASV"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">ASV_Association =</span> <span class="fu">ifelse</span>(ASV <span class="sc">%in%</span> worle_manure_persitors, <span class="st">"MAB"</span>,<span class="st">"SAB"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">asv =</span> ASV) <span class="sc">%&gt;%</span></span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_to_rownames</span>(<span class="st">"ASV"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(<span class="at">ASV =</span> asv) <span class="sc">%&gt;%</span></span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.matrix</span>() </span>
<span id="cb28-9"><a href="#cb28-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-10"><a href="#cb28-10" aria-hidden="true" tabindex="-1"></a><span class="fu">tax_table</span>(worle.water) <span class="ot">&lt;-</span> tax_association</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Now subset water samples to just “MAB”.</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-18_f340dcc3c6b2724c516b70e5942a08d2">
<div class="sourceCode" id="cb29"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Due to sequencing depth, we have a few MAB detected in CS plots, let's note them and remove them for "clean" plot. </span></span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>ASVsinCS <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle.water, treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">subset_taxa</span>(ASV_Association <span class="sc">==</span> <span class="st">"MAB"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">psmelt</span>()</span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a>ASVsinCS <span class="ot">&lt;-</span> ASVsinCS <span class="sc">%&gt;%</span></span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Abundance <span class="sc">&gt;</span> <span class="dv">0</span>) <span class="sc">%&gt;%</span></span>
<span id="cb29-8"><a href="#cb29-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(OTU, Abundance, Phylum, Class, Order, Family, Genus)</span>
<span id="cb29-9"><a href="#cb29-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-10"><a href="#cb29-10" aria-hidden="true" tabindex="-1"></a>removeASVs <span class="ot">&lt;-</span> ASVsinCS<span class="sc">$</span>OTU</span>
<span id="cb29-11"><a href="#cb29-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-12"><a href="#cb29-12" aria-hidden="true" tabindex="-1"></a>cleanwater <span class="ot">&lt;-</span> <span class="fu">subset_taxa</span>(worle.water, <span class="sc">!</span>ASV <span class="sc">%in%</span> removeASVs)</span>
<span id="cb29-13"><a href="#cb29-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-14"><a href="#cb29-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-15"><a href="#cb29-15" aria-hidden="true" tabindex="-1"></a>sp <span class="ot">&lt;-</span> cleanwater <span class="sc">%&gt;%</span></span>
<span id="cb29-16"><a href="#cb29-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">transform_sample_counts</span>(<span class="cf">function</span>(x) x <span class="sc">/</span> <span class="fu">sum</span>(x)) <span class="sc">%&gt;%</span></span>
<span id="cb29-17"><a href="#cb29-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">subset_taxa</span>(ASV_Association <span class="sc">==</span> <span class="st">"MAB"</span>)</span>
<span id="cb29-18"><a href="#cb29-18" aria-hidden="true" tabindex="-1"></a>sp</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 489 taxa and 51 samples ]
sample_data() Sample Data:       [ 51 samples by 12 sample variables ]
tax_table()   Taxonomy Table:    [ 489 taxa by 8 taxonomic ranks ]
refseq()      DNAStringSet:      [ 489 reference sequences ]</code></pre>
</div>
</div>
</section>
<section id="figure-2" class="level1">
<h1>Figure 2</h1>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-20_cc8848e931d155b22347fe5f85759f85">
<div class="sourceCode" id="cb31"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">phylogeny_profile</span>(sp, <span class="at">classification =</span> <span class="st">'Phylum'</span>, <span class="at">treatment =</span> <span class="fu">c</span>(<span class="st">"treatment"</span>), <span class="at">merge =</span> <span class="cn">TRUE</span>, <span class="at">relative_abundance =</span> F) <span class="sc">+</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(. <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span></span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>        <span class="at">legend.position =</span> <span class="st">"top"</span>) <span class="sc">+</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">ncol=</span><span class="dv">3</span>)) <span class="sc">+</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="st">"Relative abundance of MAB"</span>) <span class="sc">+</span></span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Scale for 'x' is already present. Adding another scale for 'x', which will
replace the existing scale.</code></pre>
</div>
<div class="sourceCode" id="cb33"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> p<span class="sc">$</span>data <span class="sc">%&gt;%</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">separate</span>(<span class="at">col =</span> unique_id, <span class="at">into =</span> <span class="fu">c</span>(<span class="st">"p"</span>, <span class="st">"time"</span>), <span class="at">sep =</span> <span class="st">"-"</span>) </span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(df<span class="sc">$</span>treatment) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"CL"</span>, <span class="st">"CSL"</span>, <span class="st">"CS"</span>)</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>df<span class="sc">$</span>treatment <span class="ot">&lt;-</span> <span class="fu">ordered</span>(df<span class="sc">$</span>treatment, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"CS"</span>, <span class="st">"CSL"</span>, <span class="st">"CL"</span>)) </span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>breaks <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(plot <span class="sc">==</span> <span class="dv">4</span>, time <span class="sc">==</span> <span class="dv">3</span>) <span class="sc">%&gt;%</span></span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">mean =</span> <span class="fu">mean</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(mean) <span class="sc">%&gt;%</span> </span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Phylum) </span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>df<span class="sc">$</span>Phylum <span class="ot">&lt;-</span> <span class="fu">factor</span>(df<span class="sc">$</span>Phylum, <span class="at">levels =</span> breaks<span class="sc">$</span>Phylum)</span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a>mabbarplot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> df, <span class="fu">aes</span>(<span class="at">x =</span> time, <span class="at">y =</span> Abundance, <span class="at">fill =</span> Phylum)) <span class="sc">+</span></span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">colour =</span> <span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(. <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span></span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb33-21"><a href="#cb33-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb33-22"><a href="#cb33-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb33-23"><a href="#cb33-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb33-24"><a href="#cb33-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb33-25"><a href="#cb33-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb33-26"><a href="#cb33-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb33-27"><a href="#cb33-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb33-28"><a href="#cb33-28" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">aesthetics =</span> <span class="st">"fill"</span>, <span class="at">values =</span> <span class="fu">subset</span>(colors, <span class="fu">names</span>(colors) <span class="sc">%in%</span> df<span class="sc">$</span>Phylum), <span class="at">breaks =</span> breaks<span class="sc">$</span>Phylum) <span class="sc">+</span></span>
<span id="cb33-29"><a href="#cb33-29" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample time (min) after runoff"</span>, <span class="at">y =</span> <span class="st">"Relative abundance of LAB"</span>) <span class="sc">+</span></span>
<span id="cb33-30"><a href="#cb33-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>(<span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"2.5"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"7.5"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"12.5"</span>, <span class="st">"4"</span> <span class="ot">=</span> <span class="st">"17.5"</span>, <span class="st">"5"</span> <span class="ot">=</span> <span class="st">"22.5"</span>, <span class="st">"6"</span> <span class="ot">=</span> <span class="st">"27.5"</span>)) <span class="sc">+</span></span>
<span id="cb33-31"><a href="#cb33-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb33-32"><a href="#cb33-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">nrow =</span> <span class="dv">4</span>))</span>
<span id="cb33-33"><a href="#cb33-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-34"><a href="#cb33-34" aria-hidden="true" tabindex="-1"></a>mabbarplot</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>The relative abundance of phyla that represent the “MAB” taxa detected in runoff water.</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-22_0b58d75a6a0d32cbf40396e4d29c01e9">
<div class="sourceCode" id="cb34"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig2.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="dv">7</span>, <span class="at">height =</span> <span class="dv">6</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="figure-3" class="level1">
<h1>Figure 3</h1>
<p>Only plot 8 most abundant phyla to keep plot cleaner</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-24_95b5e41e579c45a2e2b8053b7c978d4e">
<div class="sourceCode" id="cb35"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>summary <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>treatment <span class="sc">==</span> <span class="st">"CS"</span>) <span class="sc">%&gt;%</span> <span class="co"># Remove no manure treatment</span></span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum, </span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a>           plot, </span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>           treatment) <span class="sc">%&gt;%</span></span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>( </span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">n=</span><span class="fu">n</span>(),</span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean=</span><span class="fu">mean</span>(Abundance),</span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">sd=</span><span class="fu">sd</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">se =</span> sd<span class="sc">/</span><span class="fu">sqrt</span>(n))  <span class="sc">%&gt;%</span></span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">ic =</span> se <span class="sc">*</span> <span class="fu">qt</span>((<span class="dv">1</span><span class="fl">-0.05</span>)<span class="sc">/</span><span class="dv">2</span> <span class="sc">+</span> .<span class="dv">5</span>, n<span class="dv">-1</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(plot, <span class="sc">-</span>mean) <span class="sc">%&gt;%</span> </span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'Phylum', 'plot'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="sourceCode" id="cb37"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>EightPhyla <span class="ot">&lt;-</span> summary <span class="sc">%&gt;%</span></span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">mean =</span> <span class="fu">mean</span>(mean)) <span class="sc">%&gt;%</span></span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="sc">-</span>mean) <span class="sc">%&gt;%</span></span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>() <span class="sc">%&gt;%</span></span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">head</span>(<span class="dv">8</span>) <span class="co"># Use head with n = 8 to select eight phyla</span></span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>summary<span class="sc">$</span>plot <span class="ot">&lt;-</span> <span class="fu">as.factor</span>(summary<span class="sc">$</span>plot)</span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>summary <span class="ot">&lt;-</span> summary <span class="sc">%&gt;%</span></span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Phylum <span class="sc">%in%</span> EightPhyla<span class="sc">$</span>Phylum)</span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(summary, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">y =</span> mean, <span class="at">fill =</span> plot)) <span class="sc">+</span></span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, </span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">ymin =</span> mean <span class="sc">-</span> ic, <span class="at">ymax =</span> mean <span class="sc">+</span> ic), </span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="dv">1</span>), <span class="at">width =</span> <span class="fl">0.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size=</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="st">"Relative abundance of phyla"</span>, <span class="at">x =</span> <span class="st">"Phylum"</span>) <span class="sc">+</span></span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_grid</span>( <span class="sc">~</span> treatment) <span class="sc">+</span></span>
<span id="cb37-21"><a href="#cb37-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span> </span>
<span id="cb37-22"><a href="#cb37-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb37-23"><a href="#cb37-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb37-24"><a href="#cb37-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">name =</span> <span class="st">"Plot"</span>) <span class="sc">+</span></span>
<span id="cb37-25"><a href="#cb37-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="co"># Here we can multiply the relative abundance by 100 and add a % mark, note that any tables generated will still need to be multiplied by 100. </span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="16S_files/figure-html/unnamed-chunk-24-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb38"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig3.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="dv">7</span>, <span class="at">height =</span> <span class="dv">6</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>LAB treatment mean</p>
<div class="cell" data-hash="16S_cache/html/unnamed-chunk-26_4e99398f41044007c3ffac3ad48ac524">
<div class="sourceCode" id="cb39"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>df <span class="sc">%&gt;%</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(treatment, plot, time) <span class="sc">%&gt;%</span></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">MAB_Abundance =</span> <span class="fu">sum</span>(Abundance)) <span class="sc">%&gt;%</span> </span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(treatment, plot) <span class="sc">%&gt;%</span></span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean_mab =</span> <span class="fu">mean</span>(MAB_Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(plot) <span class="sc">%&gt;%</span></span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">trtMeanMab =</span> <span class="fu">mean</span>(mean_mab))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'treatment', 'plot'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'treatment'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="cell-output-stdout">
<pre><code># A tibble: 9 x 2
   plot trtMeanMab
  &lt;dbl&gt;      &lt;dbl&gt;
1     1    0      
2     2    0      
3     3    0      
4     4    0.552  
5     5    0.0110 
6     6    0.00284
7     7    0.0893 
8     8    0.100  
9     9    0.312  </code></pre>
</div>
</div>


</section>
</main> <!-- /main -->
<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    setTimeout(function() {
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
  </div>
  <div class="nav-page nav-page-next">
  </div>
</nav>
</div> <!-- /content -->


</body></html>