<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang=""><head>
  <meta charset="utf-8">
  <meta name="generator" content="quarto-0.2.323">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Jared Flater">
  <title>Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      {   }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span.al { color: #ff0000; font-weight: bold; } /* Alert */
    code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
    code span.at { color: #7d9029; } /* Attribute */
    code span.bn { color: #40a070; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #4070a0; } /* Char */
    code span.cn { color: #880000; } /* Constant */
    code span.co { color: #60a0b0; font-style: italic; } /* Comment */
    code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
    code span.do { color: #ba2121; font-style: italic; } /* Documentation */
    code span.dt { color: #902000; } /* DataType */
    code span.dv { color: #40a070; } /* DecVal */
    code span.er { color: #ff0000; font-weight: bold; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #40a070; } /* Float */
    code span.fu { color: #06287e; } /* Function */
    code span.im { } /* Import */
    code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
    code span.kw { color: #007020; font-weight: bold; } /* Keyword */
    code span.op { color: #666666; } /* Operator */
    code span.ot { color: #007020; } /* Other */
    code span.pp { color: #bc7a00; } /* Preprocessor */
    code span.sc { color: #4070a0; } /* SpecialChar */
    code span.ss { color: #bb6688; } /* SpecialString */
    code span.st { color: #4070a0; } /* String */
    code span.va { color: #19177c; } /* Variable */
    code span.vs { color: #4070a0; } /* VerbatimString */
    code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
  </style>

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <script src="site_libs/quarto-nav/quarto-nav.js"></script>
  <script src="site_libs/quarto-nav/headroom.min.js"></script>
  <script src="site_libs/clipboard/clipboard.min.js"></script>
  <meta name="quarto:offset" content="./">
  <script src="site_libs/quarto-search/autocomplete.umd.js"></script>
  <script src="site_libs/quarto-search/fuse.min.js"></script>
  <script src="site_libs/quarto-search/quarto-search.js"></script>
  <script src="site_libs/quarto-html/quarto.js"></script>
  <script src="site_libs/quarto-html/popper.min.js"></script>
  <script src="site_libs/quarto-html/tippy.umd.min.js"></script>
  <script src="site_libs/quarto-html/anchor.min.js"></script>
  <link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
  <link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet">
  <script src="site_libs/bootstrap/bootstrap.min.js"></script>
  <link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
  <link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet">
  <script id="quarto-search-options" type="application/json">{
    "location": "navbar",
    "copy-button": false,
    "collapse-after": 2,
    "panel-placement": "end",
    "type": "overlay",
    "limit": 20,
    "language": {
      "search-no-results-text": "No results",
      "search-matching-documents-text": "matching documents",
      "search-copy-link-title": "Copy link to search",
      "search-hide-matches-text": "Hide additional matches",
      "search-more-match-text": "more match in this document",
      "search-more-matches-text": "more matches in this document",
      "search-clear-button-title": "Clear",
      "search-detached-cancel-button-title": "Cancel",
      "search-submit-button-title": "Submit"
    }
  }</script>
  <link rel="stylesheet" href="styles/styles.scss">
</head>
<body class="docked">
<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-light ">
      <div class="navbar-container container-fluid">
      <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure</span>
  </a>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="http://germslab.org">GERMS Lab</a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title">Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure</h1>
      <button type="button" class="quarto-btn-toggle btn">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation docked overflow-auto">
  <div class="sidebar-menu-container"> 
  <ul class="list-unstyled mt-1">
      <li class="sidebar-item">
  <a href="./16S.html" class="">16S</a>
</li>
      <li class="sidebar-item">
  <a href="./ARGs.html" class="">ARGs</a>
</li>
  </ul>
  </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
      <nav id="TOC" role="doc-toc">
<h2 id="toc-title">On this page</h2>
<ul>
<li><a href="#figure-1" class="nav-link active" data-scroll-target="#figure-1">Figure 1</a></li>
<li><a href="#supp-figure-s2" class="nav-link" data-scroll-target="#supp-figure-s2">Supp Figure S2</a></li>
<li><a href="#figure-2" class="nav-link" data-scroll-target="#figure-2">Figure 2</a></li>
<li><a href="#figure-3" class="nav-link" data-scroll-target="#figure-3">Figure 3</a></li>
<li><a href="#figure-4" class="nav-link" data-scroll-target="#figure-4">Figure 4</a></li>
<li><a href="#supp-figure-s3" class="nav-link" data-scroll-target="#supp-figure-s3">Supp Figure S3</a></li>
<li><a href="#supp-figure-s4" class="nav-link" data-scroll-target="#supp-figure-s4">Supp Figure S4</a></li>
<li><a href="#supp-figure-s5" class="nav-link" data-scroll-target="#supp-figure-s5">Supp Figure S5</a></li>
<li><a href="#supp-figure-s6" class="nav-link" data-scroll-target="#supp-figure-s6">Supp Figure S6</a></li>
<li><a href="#supp-figure-s7" class="nav-link" data-scroll-target="#supp-figure-s7">Supp Figure S7</a></li>
</ul>
</nav>
    </div>
<!-- main -->
<main class="content">
<header id="title-block-header">
<h1 class="title d-none d-lg-block display-7">Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure</h1>
<p class="author">Jared Flater</p>
<p class="date">2/15/2021</p>
</header>

<p>Introduction</p>
<p>This code was used to generate figures for a publication titled: <em>Prairie Strips Impact on Transport of Antibiotic Resistance Indicators in Poultry Manure</em></p>
<p>The manuscript is available here:</p>
<p>The project was designed to evaluate if the conservation practice of planting strips of prairie grasses within agricultural fields might attenuate the transport of manure associated genes and bacteria.</p>
<p>Poultry manure was applied to small plots appended with prairie strips and rainfall was simulated on the manure applied ground. Runoff water samples were collected after passing through the prairie strip and soil samples were collected before and after the rainfall simulation. In both soil and water samples we looked for and tracked the presence and abundance of manure associated bacteria and genes.</p>
<p>Figure 1</p>
<p>Figure was is a taxonomic profile of the bacteria present in the poultry manure that was used as a fertilizer. Manure was applied to six of nine plots, two plots received manure each day of a rainfall simulation. Rainfall simulations were performed on the nine plots over three days, one day each for a block of plots. Each block contained one plot from each treatment. The treatments were:</p>
<p><strong>Crop + Strip (CS)</strong></p>
<p><strong>Crop + Strip + Manure (CSM)</strong></p>
<p><strong>Crop + Manure (CM)</strong></p>
<p>The manure applied to each plot came from a single ~ 60 lbs batch of manure. When manure was applied to each plot, a sample of manure was taken to be analyzed for bacteria and antibiotic resistance genes. To track manure associated bacteria, we must first characterize the bacteria present. Here we have six manure samples and a bar plot representing the relative abundance of bacterial phyla in each sample. This data was generated using NGS sequencing of the 16S gene and analyzed using DADA2 to generate a count of the different taxa present in the various samples from the rainfall simulation.</p>
<p>16S Figures</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/data import_5a2bc61b5d7114fa366b6143d6b847a7">
<div class="sourceCode" id="cb1"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Worle data</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>worle <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data/Worle_curated.RDS"</span>) </span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="co"># set taxa to ASV, otherwise uses the sequence as taxa name</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">taxa_names</span>(worle) <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="st">"ASV"</span>, <span class="fu">seq</span>(<span class="fu">ntaxa</span>(worle)))</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co"># rename NA taxa as "Unclassified"</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>Tax.Table <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">tax_table</span>(worle)) <span class="sc">%&gt;%</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">replace</span>(<span class="fu">is.na</span>(.), <span class="st">"Unclassified"</span>)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="co"># replace tax table </span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="fu">tax_table</span>(worle) <span class="ot">&lt;-</span> <span class="fu">as.matrix</span>(Tax.Table)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>First, visualize differences in communities by plotting PCoA ordination of the Bray-Curtis dissimilarity</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-2_698d19e539b315fc4e8554b26a95b930">
<div class="sourceCode" id="cb2"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(<span class="fu">sample_data</span>(worle))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code> [1] "unique_id"        "experiment"       "matrix"           "treatment"       
 [5] "plot"             "sample_day"       "depth"            "in_plot_location"
 [9] "block"            "strip"            "manure_treatment" "soil_type"       </code></pre>
</div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sample_data</span>(worle)<span class="sc">$</span>treatment[<span class="fu">is.na</span>(<span class="fu">sample_data</span>(worle)<span class="sc">$</span>treatment)] <span class="ot">&lt;-</span> <span class="st">"manure"</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>pcoa <span class="ot">&lt;-</span> <span class="fu">pcoa_phyloseq</span>(worle, <span class="fu">c</span>(<span class="st">'matrix'</span>, <span class="st">'treatment'</span>), <span class="at">circle =</span> T) <span class="sc">+</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>() <span class="sc">+</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Scale for 'fill' is already present. Adding another scale for 'fill', which
will replace the existing scale.</code></pre>
</div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>pcoa</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-2-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb7"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/PCoA.jpeg"</span>, <span class="at">plot =</span> pcoa, <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="fl">7.0</span>, <span class="at">height =</span> <span class="fl">6.0</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/subset to manure_3eb59a34f9db4074306c538c53b1b039">
<div class="sourceCode" id="cb8"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>worle.manure <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"manure"</span>) <span class="sc">%&gt;%</span> <span class="co"># subset to manure</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)  <span class="co"># Remove taxa observed less than once in these samples</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>As we make plots of taxa, it’s easier to understand if consistent colors are used for the various phyla. We can assign a color to each of these phyla then manually color plots based on this.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/phyla colors_4985c4b9db350a630a0dd0c11cd78179">
<div class="sourceCode" id="cb9"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">010101</span>)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="co"># make a list of unique phyla</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>phylalist <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">tax_table</span>(worle), <span class="at">row.names =</span> <span class="cn">NULL</span>) <span class="sc">%&gt;%</span></span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">unique</span>() </span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>phylalist<span class="sc">$</span>Phylum <span class="ot">&lt;-</span> <span class="fu">as.character</span>(phylalist<span class="sc">$</span>Phylum)</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(colorspace)  <span class="co"># this package will generate a palette based on number and desired colors</span></span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a><span class="co"># assign color from "viridis" package to each phyla and name the list with phyla names</span></span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a>colors <span class="ot">&lt;-</span> <span class="fu">sequential_hcl</span>(<span class="fu">n_distinct</span>(phylalist), <span class="at">palette =</span> <span class="st">"viridis"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">setNames</span>(<span class="fu">sort</span>(phylalist<span class="sc">$</span>Phylum))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="figure-1" class="level1">
<h1>Figure 1</h1>
<p>Let’s calculate the relative abundance of each phyla in the manure and generate summary statistics (mean, standard error, and interquartile range) ::: {.cell hash=“ManuscriptWalkthrough_cache/html/figure 1 plots_92386224e66a6e3b36ddb4facb2095b8”}</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>global_size <span class="ot">=</span> <span class="dv">12</span> <span class="co"># Font size</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>phydf <span class="ot">&lt;-</span> worle.manure <span class="sc">%&gt;%</span></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tax_glom</span>(<span class="at">taxrank =</span> <span class="st">"Phylum"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">transform_sample_counts</span>(<span class="cf">function</span>(x) x <span class="sc">/</span> <span class="fu">sum</span>(x)) <span class="sc">%&gt;%</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">psmelt</span>()  <span class="co"># Take manure phyloseq object and glom taxa at phyla level. Transform counts to relative abundance and melt to a data frame. </span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>summary <span class="ot">&lt;-</span> phydf <span class="sc">%&gt;%</span>  <span class="co"># Make a plot</span></span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>( </span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">n=</span><span class="fu">n</span>(),</span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean=</span><span class="fu">mean</span>(Abundance),</span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">sd=</span><span class="fu">sd</span>(Abundance)</span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">se =</span> sd<span class="sc">/</span><span class="fu">sqrt</span>(n))  <span class="sc">%&gt;%</span></span>
<span id="cb10-15"><a href="#cb10-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">ic =</span> se <span class="sc">*</span> <span class="fu">qt</span>((<span class="dv">1</span><span class="fl">-0.05</span>)<span class="sc">/</span><span class="dv">2</span> <span class="sc">+</span> .<span class="dv">5</span>, n<span class="dv">-1</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb10-16"><a href="#cb10-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="sc">-</span>mean)</span>
<span id="cb10-17"><a href="#cb10-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-18"><a href="#cb10-18" aria-hidden="true" tabindex="-1"></a>summary<span class="sc">$</span>Phylum <span class="ot">&lt;-</span> <span class="fu">factor</span>(summary<span class="sc">$</span>Phylum, <span class="at">levels =</span> summary<span class="sc">$</span>Phylum[<span class="fu">order</span>(<span class="sc">-</span>summary<span class="sc">$</span>mean)]) <span class="co"># arrange won't change order of Phylum factor levels</span></span>
<span id="cb10-19"><a href="#cb10-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-20"><a href="#cb10-20" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(summary) <span class="sc">+</span></span>
<span id="cb10-21"><a href="#cb10-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb10-22"><a href="#cb10-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">x =</span> Phylum, <span class="at">y =</span> mean, <span class="at">fill =</span> Phylum), <span class="at">stat =</span> <span class="st">"identity"</span>, </span>
<span id="cb10-23"><a href="#cb10-23" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.75</span>)) <span class="sc">+</span></span>
<span id="cb10-24"><a href="#cb10-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>( <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">ymin =</span> mean <span class="sc">-</span> ic, <span class="at">ymax =</span> mean <span class="sc">+</span> ic), <span class="at">width =</span> <span class="fl">0.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size=</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb10-25"><a href="#cb10-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="st">"Relative abundance of phyla"</span>, <span class="at">x =</span> <span class="st">"Phylum"</span>) <span class="sc">+</span></span>
<span id="cb10-26"><a href="#cb10-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colors) <span class="sc">+</span></span>
<span id="cb10-27"><a href="#cb10-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>, </span>
<span id="cb10-28"><a href="#cb10-28" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb10-29"><a href="#cb10-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb10-30"><a href="#cb10-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="at">x =</span> <span class="st">"Proteobacteria"</span>, <span class="at">y =</span> <span class="fl">0.60</span>, <span class="at">label =</span> <span class="st">"A"</span>, <span class="at">size =</span> <span class="dv">8</span>, <span class="at">family =</span> <span class="st">"Times"</span>) <span class="sc">+</span></span>
<span id="cb10-31"><a href="#cb10-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>)) </span>
<span id="cb10-32"><a href="#cb10-32" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb10-33"><a href="#cb10-33" aria-hidden="true" tabindex="-1"></a>p1</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/figure%201%20plots-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb11"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># We can make a second smaller plot to show the abundances of the low abundance phyla</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">subset</span>(summary, Phylum <span class="sc">%in%</span> <span class="fu">tail</span>(summary<span class="sc">$</span>Phylum, <span class="sc">-</span><span class="dv">5</span>))) <span class="sc">+</span></span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">y =</span> mean, <span class="at">fill =</span> Phylum), <span class="at">stat =</span> <span class="st">"identity"</span>, </span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.75</span>)) <span class="sc">+</span></span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>( <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">ymin =</span> mean <span class="sc">-</span> ic, <span class="at">ymax =</span> mean <span class="sc">+</span> ic), <span class="at">width =</span> <span class="fl">0.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size=</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="cn">NULL</span>, <span class="at">y =</span> <span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colors) <span class="sc">+</span></span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>, </span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb11-13"><a href="#cb11-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="at">x =</span> <span class="st">"Tenericutes"</span>, <span class="at">y =</span> <span class="fl">0.0060</span>, <span class="at">label =</span> <span class="st">"B"</span>, <span class="at">size =</span> <span class="dv">8</span>, <span class="at">family =</span> <span class="st">"Times"</span>) <span class="sc">+</span></span>
<span id="cb11-14"><a href="#cb11-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>)) </span>
<span id="cb11-15"><a href="#cb11-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-16"><a href="#cb11-16" aria-hidden="true" tabindex="-1"></a>p2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/figure%201%20plots-2.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb12"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> p1 <span class="sc">+</span> <span class="fu">annotation_custom</span>(<span class="fu">ggplotGrob</span>(p2), <span class="at">xmin =</span> <span class="st">"Deinococcus-Thermus"</span>, <span class="at">xmax =</span> <span class="st">"Deferribacteres"</span>, <span class="at">ymin =</span> .<span class="dv">05</span>, <span class="at">ymax =</span> .<span class="dv">65</span>) </span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>p3</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/figure%201%20plots-3.png" class="img-fluid" width="672"></p>
</div>
<p>:::</p>
<p>We can see that the manure is mostly represented by five phyla, Firmicutes, Proteobacteria, Bacteroidetes, Actinobacetria, and Deinococcus-Thermus.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/fig 1 save_8ec2bf0b3f05c911fcaef7592b068b43">
<div class="sourceCode" id="cb13"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Use ggsave to save plot. I chose 6.5" wide because most word docs are 8.5" with 2 x 1" margins. Keep the height below 9.5" to save room for a caption. When you insert into your document, change the size to 6.5" wide to ensure that the plot looks visually the same. </span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig1.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="fl">7.0</span>, <span class="at">height =</span> <span class="fl">6.0</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The specific abundances of phyla are:</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-4_9085ae358d06b32141ca827a30d47fb2">
<div class="sourceCode" id="cb14"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>phydf <span class="sc">%&gt;%</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">Mean =</span> <span class="fu">mean</span>(Abundance), <span class="at">StdDev =</span> <span class="fu">sd</span>(Abundance), <span class="at">min =</span> <span class="fu">min</span>(Abundance), <span class="at">max =</span> <span class="fu">max</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="sc">-</span>Mean)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 20 x 5
   Phylum                        Mean    StdDev       min       max
   &lt;chr&gt;                        &lt;dbl&gt;     &lt;dbl&gt;     &lt;dbl&gt;     &lt;dbl&gt;
 1 Firmicutes               0.555     0.0907    0.441     0.678    
 2 Proteobacteria           0.200     0.0668    0.124     0.294    
 3 Bacteroidetes            0.126     0.0483    0.0714    0.191    
 4 Actinobacteria           0.0890    0.0170    0.0669    0.105    
 5 Deinococcus-Thermus      0.0187    0.00770   0.00633   0.0261   
 6 Chloroflexi              0.00405   0.00262   0.00185   0.00842  
 7 Tenericutes              0.00173   0.00166   0.0000728 0.00455  
 8 Planctomycetes           0.000927  0.000411  0.000337  0.00137  
 9 Unclassified             0.000871  0.00116   0         0.00329  
10 Euryarchaeota            0.000706  0.00108   0         0.00289  
11 Verrucomicrobia          0.000526  0.000602  0         0.00158  
12 Acidobacteria            0.000353  0.000303  0         0.000705 
13 Synergistetes            0.000261  0.000548  0         0.00137  
14 Elusimicrobia            0.000195  0.000385  0         0.000962 
15 Fusobacteria             0.000153  0.000272  0         0.000670 
16 candidate_division_WPS-1 0.000110  0.000160  0         0.000407 
17 Cloacimonetes            0.0000796 0.000142  0         0.000351 
18 Armatimonadetes          0.0000506 0.0000659 0         0.000166 
19 Spirochaetes             0.0000281 0.0000688 0         0.000169 
20 Deferribacteres          0.0000123 0.0000302 0         0.0000740</code></pre>
</div>
</div>
</section>
<section id="supp-figure-s2" class="level1">
<h1>Supp Figure S2</h1>
<p>Abundance of manure associated taxa</p>
<p>Now let’s look at the number of unique taxa in the various samples.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-6_ee89531bebc5ba3ff49fa59074bbf265">
<div class="sourceCode" id="cb16"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co"># strip subset, all strip soil from baseline samples in addition to no_manure_strip treatment, note the | operator </span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>worle.strip <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> sample_day <span class="sc">==</span> <span class="st">"Baseline"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"strip"</span> <span class="sc">|</span> </span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>                                matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"strip"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T) </span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>worle.crop <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> sample_day <span class="sc">==</span> <span class="st">"Baseline"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"crop"</span> <span class="sc">|</span> </span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>                                matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"crop"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a>worle.border <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> sample_day <span class="sc">==</span> <span class="st">"Baseline"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"border"</span> <span class="sc">|</span> </span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>                                matrix <span class="sc">==</span> <span class="st">"soil"</span> <span class="sc">&amp;</span> treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span> <span class="sc">&amp;</span> soil_type <span class="sc">==</span> <span class="st">"border"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-12"><a href="#cb16-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span>
<span id="cb16-13"><a href="#cb16-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-14"><a href="#cb16-14" aria-hidden="true" tabindex="-1"></a>worle.manure <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"manure"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-15"><a href="#cb16-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span>
<span id="cb16-16"><a href="#cb16-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-17"><a href="#cb16-17" aria-hidden="true" tabindex="-1"></a>worle.water <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle, matrix <span class="sc">==</span> <span class="st">"water"</span> <span class="sc">&amp;</span> unique_id <span class="sc">!=</span> <span class="st">"Comp-from-P8-7-10-26-17"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-18"><a href="#cb16-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter_taxa</span>(<span class="cf">function</span>(x) <span class="fu">sum</span>(x) <span class="sc">&gt;=</span> <span class="dv">1</span>, T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-8_713f2d562615040b9037ef59b123e035">
<div class="sourceCode" id="cb17"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Make a list of ASVs associated with each type of soil and manure</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>worle_manure_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.manure)</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>worle_crop_soil_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.crop)</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>worle_strip_soil_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.strip)</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>worle_border_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.border)</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>worle_water_asvs <span class="ot">&lt;-</span> <span class="fu">taxa_names</span>(worle.water)</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a><span class="co"># venn diagram</span></span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>worle_vvv_diag <span class="ot">&lt;-</span> <span class="fu">venn</span>(<span class="fu">list</span>(<span class="st">"Manure_ASVs"</span> <span class="ot">=</span> worle_manure_asvs, <span class="st">"Crop_ASVs"</span> <span class="ot">=</span> worle_crop_soil_asvs, <span class="st">"Strip_ASVs"</span> <span class="ot">=</span> worle_strip_soil_asvs, <span class="st">"Border_ASVs"</span> <span class="ot">=</span> worle_border_asvs))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb18"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Access ASVs unique to each soil and manure and store</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>worle_manure_persitors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersections"</span>)<span class="sc">$</span>Manure_ASVs</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>worle_crop_persistors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersections"</span>)<span class="sc">$</span>Crop_ASVs</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>worle_strip_persistors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersection"</span>)<span class="sc">$</span>Strip_ASVs</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>worle_border_persistors <span class="ot">&lt;-</span> <span class="fu">attr</span>(worle_vvv_diag, <span class="st">"intersections"</span>)<span class="sc">$</span>Border_ASVs</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>We have 1017 “MAB” taxa</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-10_5dde6c7495cada85845eb82c39db5f67">
<div class="sourceCode" id="cb19"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>Sample_Type <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Manure"</span>, <span class="st">"Crop"</span>, <span class="st">"Strip"</span>, <span class="st">"Border"</span>)</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>Number_of_taxa <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">ntaxa</span>(worle.manure), <span class="fu">ntaxa</span>(worle.crop), <span class="fu">ntaxa</span>(worle.strip), <span class="fu">ntaxa</span>(worle.border))</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>Taxa_unique_to <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">length</span>(worle_manure_persitors), <span class="fu">length</span>(worle_crop_persistors), <span class="fu">length</span>(worle_strip_persistors), <span class="fu">length</span>(worle_border_persistors))</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(Sample_Type, Number_of_taxa, Taxa_unique_to)</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>  Sample_Type Number_of_taxa Taxa_unique_to
1      Manure           1073           1017
2        Crop          19002           9548
3       Strip          18043           8518
4      Border           8140           1868</code></pre>
</div>
</div>
<p>Make a taxonomy table of the ASVs unique to manure</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-12_a73189534a5549db030def56ffed9a6f">
<div class="sourceCode" id="cb21"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>MAB <span class="ot">&lt;-</span> <span class="fu">tax_table</span>(worle.manure) <span class="sc">%&gt;%</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>() <span class="sc">%&gt;%</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="st">"ASV"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(ASV <span class="sc">%in%</span> worle_manure_persitors)</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(MAB)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>     ASV  Kingdom         Phylum          Class           Order
1   ASV4 Bacteria     Firmicutes        Bacilli      Bacillales
2 ASV184 Bacteria     Firmicutes        Bacilli      Bacillales
3 ASV187 Bacteria     Firmicutes        Bacilli      Bacillales
4 ASV244 Bacteria     Firmicutes        Bacilli Lactobacillales
5 ASV246 Bacteria Actinobacteria Actinobacteria Actinomycetales
6 ASV269 Bacteria     Firmicutes        Bacilli      Bacillales
              Family           Genus
1      Bacillaceae_2    Unclassified
2      Bacillaceae_2  Oceanobacillus
3      Bacillaceae_2    Unclassified
4    Enterococcaceae    Enterococcus
5 Corynebacteriaceae Corynebacterium
6      Bacillaceae_2    Unclassified</code></pre>
</div>
<div class="sourceCode" id="cb23"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">write.table</span>(MAB, <span class="at">file =</span> <span class="st">"Tables/MAB.csv"</span>, <span class="at">sep =</span> <span class="st">","</span>, <span class="at">col.names =</span> T,</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>            <span class="at">qmethod =</span> <span class="st">"double"</span>, <span class="at">row.names =</span> F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-14_d27803d2aa365bf95615d80f2235e7c8">
<div class="sourceCode" id="cb24"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggvenn</span>(<span class="fu">list</span>(<span class="st">"Manure"</span> <span class="ot">=</span> worle_manure_asvs, <span class="st">"Crop"</span> <span class="ot">=</span> worle_crop_soil_asvs, <span class="st">"Strip"</span> <span class="ot">=</span> worle_strip_soil_asvs, <span class="st">"Border"</span> <span class="ot">=</span> worle_border_asvs), <span class="at">set_name_size =</span> <span class="dv">4</span>, <span class="at">show_percentage =</span> F) <span class="sc">+</span> <span class="fu">scale_fill_viridis_d</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Scale for 'fill' is already present. Adding another scale for 'fill', which
will replace the existing scale.</code></pre>
</div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb26"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"Figures/SuppFig2.png"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">width =</span> <span class="fl">6.5</span>, <span class="at">dpi =</span> <span class="dv">300</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Saving 6.5 x 5 in image</code></pre>
</div>
</div>
<p>We will assign taxa to two groups, soils associated and manure associated bacteria: “SAB” and “MAB”</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-16_27046e8bc559caa60fc84cbaf8af8ac4">
<div class="sourceCode" id="cb28"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>tax_association <span class="ot">&lt;-</span> <span class="fu">tax_table</span>(worle.water) <span class="sc">%&gt;%</span> </span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>() <span class="sc">%&gt;%</span></span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="st">"ASV"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">ASV_Association =</span> <span class="fu">ifelse</span>(ASV <span class="sc">%in%</span> worle_manure_persitors, <span class="st">"MAB"</span>,<span class="st">"SAB"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">asv =</span> ASV) <span class="sc">%&gt;%</span></span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">column_to_rownames</span>(<span class="st">"ASV"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(<span class="at">ASV =</span> asv) <span class="sc">%&gt;%</span></span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.matrix</span>() </span>
<span id="cb28-9"><a href="#cb28-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-10"><a href="#cb28-10" aria-hidden="true" tabindex="-1"></a><span class="fu">tax_table</span>(worle.water) <span class="ot">&lt;-</span> tax_association</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Now subset water samples to just “MAB”.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-18_12506e69a278b9791775a7603cd1db7e">
<div class="sourceCode" id="cb29"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Due to sequencing depth, we have a few MAB detected in CS plots, let's note them and remove them for "clean" plot. </span></span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>ASVsinCS <span class="ot">&lt;-</span> <span class="fu">subset_samples</span>(worle.water, treatment <span class="sc">==</span> <span class="st">"no_manure_strip"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">subset_taxa</span>(ASV_Association <span class="sc">==</span> <span class="st">"MAB"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">psmelt</span>()</span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a>ASVsinCS <span class="ot">&lt;-</span> ASVsinCS <span class="sc">%&gt;%</span></span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Abundance <span class="sc">&gt;</span> <span class="dv">0</span>) <span class="sc">%&gt;%</span></span>
<span id="cb29-8"><a href="#cb29-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(OTU, Abundance, Phylum, Class, Order, Family, Genus)</span>
<span id="cb29-9"><a href="#cb29-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-10"><a href="#cb29-10" aria-hidden="true" tabindex="-1"></a>removeASVs <span class="ot">&lt;-</span> ASVsinCS<span class="sc">$</span>OTU</span>
<span id="cb29-11"><a href="#cb29-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-12"><a href="#cb29-12" aria-hidden="true" tabindex="-1"></a>cleanwater <span class="ot">&lt;-</span> <span class="fu">subset_taxa</span>(worle.water, <span class="sc">!</span>ASV <span class="sc">%in%</span> removeASVs)</span>
<span id="cb29-13"><a href="#cb29-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-14"><a href="#cb29-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-15"><a href="#cb29-15" aria-hidden="true" tabindex="-1"></a>sp <span class="ot">&lt;-</span> cleanwater <span class="sc">%&gt;%</span></span>
<span id="cb29-16"><a href="#cb29-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">transform_sample_counts</span>(<span class="cf">function</span>(x) x <span class="sc">/</span> <span class="fu">sum</span>(x)) <span class="sc">%&gt;%</span></span>
<span id="cb29-17"><a href="#cb29-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">subset_taxa</span>(ASV_Association <span class="sc">==</span> <span class="st">"MAB"</span>)</span>
<span id="cb29-18"><a href="#cb29-18" aria-hidden="true" tabindex="-1"></a>sp</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 489 taxa and 51 samples ]
sample_data() Sample Data:       [ 51 samples by 12 sample variables ]
tax_table()   Taxonomy Table:    [ 489 taxa by 8 taxonomic ranks ]
refseq()      DNAStringSet:      [ 489 reference sequences ]</code></pre>
</div>
</div>
</section>
<section id="figure-2" class="level1">
<h1>Figure 2</h1>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-20_b4d70823f2785c13e73ac0be28871254">
<div class="sourceCode" id="cb31"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">phylogeny_profile</span>(sp, <span class="at">classification =</span> <span class="st">'Phylum'</span>, <span class="at">treatment =</span> <span class="fu">c</span>(<span class="st">"treatment"</span>), <span class="at">merge =</span> <span class="cn">TRUE</span>, <span class="at">relative_abundance =</span> F) <span class="sc">+</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(. <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span></span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>        <span class="at">legend.position =</span> <span class="st">"top"</span>) <span class="sc">+</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">ncol=</span><span class="dv">3</span>)) <span class="sc">+</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="st">"Relative abundance of MAB"</span>) <span class="sc">+</span></span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Scale for 'x' is already present. Adding another scale for 'x', which will
replace the existing scale.</code></pre>
</div>
<div class="sourceCode" id="cb33"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> p<span class="sc">$</span>data <span class="sc">%&gt;%</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">separate</span>(<span class="at">col =</span> unique_id, <span class="at">into =</span> <span class="fu">c</span>(<span class="st">"p"</span>, <span class="st">"time"</span>), <span class="at">sep =</span> <span class="st">"-"</span>) </span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(df<span class="sc">$</span>treatment) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"CL"</span>, <span class="st">"CSL"</span>, <span class="st">"CS"</span>)</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>df<span class="sc">$</span>treatment <span class="ot">&lt;-</span> <span class="fu">ordered</span>(df<span class="sc">$</span>treatment, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"CS"</span>, <span class="st">"CSL"</span>, <span class="st">"CL"</span>)) </span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>breaks <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(plot <span class="sc">==</span> <span class="dv">4</span>, time <span class="sc">==</span> <span class="dv">3</span>) <span class="sc">%&gt;%</span></span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">mean =</span> <span class="fu">mean</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(mean) <span class="sc">%&gt;%</span> </span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Phylum) </span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>df<span class="sc">$</span>Phylum <span class="ot">&lt;-</span> <span class="fu">factor</span>(df<span class="sc">$</span>Phylum, <span class="at">levels =</span> breaks<span class="sc">$</span>Phylum)</span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a>mabbarplot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> df, <span class="fu">aes</span>(<span class="at">x =</span> time, <span class="at">y =</span> Abundance, <span class="at">fill =</span> Phylum)) <span class="sc">+</span></span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">colour =</span> <span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(. <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span></span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb33-21"><a href="#cb33-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb33-22"><a href="#cb33-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb33-23"><a href="#cb33-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb33-24"><a href="#cb33-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb33-25"><a href="#cb33-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb33-26"><a href="#cb33-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb33-27"><a href="#cb33-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb33-28"><a href="#cb33-28" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">aesthetics =</span> <span class="st">"fill"</span>, <span class="at">values =</span> <span class="fu">subset</span>(colors, <span class="fu">names</span>(colors) <span class="sc">%in%</span> df<span class="sc">$</span>Phylum), <span class="at">breaks =</span> breaks<span class="sc">$</span>Phylum) <span class="sc">+</span></span>
<span id="cb33-29"><a href="#cb33-29" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample time (min) after runoff"</span>, <span class="at">y =</span> <span class="st">"Relative abundance of LAB"</span>) <span class="sc">+</span></span>
<span id="cb33-30"><a href="#cb33-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>(<span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"2.5"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"7.5"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"12.5"</span>, <span class="st">"4"</span> <span class="ot">=</span> <span class="st">"17.5"</span>, <span class="st">"5"</span> <span class="ot">=</span> <span class="st">"22.5"</span>, <span class="st">"6"</span> <span class="ot">=</span> <span class="st">"27.5"</span>)) <span class="sc">+</span></span>
<span id="cb33-31"><a href="#cb33-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb33-32"><a href="#cb33-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">nrow =</span> <span class="dv">4</span>))</span>
<span id="cb33-33"><a href="#cb33-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-34"><a href="#cb33-34" aria-hidden="true" tabindex="-1"></a>mabbarplot</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>The relative abundance of phyla that represent the “MAB” taxa detected in runoff water.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-22_c1cf2240495c343c1d01d7ca6fb525ad">
<div class="sourceCode" id="cb34"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig2.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="dv">7</span>, <span class="at">height =</span> <span class="dv">6</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="figure-3" class="level1">
<h1>Figure 3</h1>
<p>Only plot 8 most abundant phyla to keep plot cleaner</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-24_d7991002da1520b84a2ce34b1cb7439e">
<div class="sourceCode" id="cb35"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>summary <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>treatment <span class="sc">==</span> <span class="st">"CS"</span>) <span class="sc">%&gt;%</span> <span class="co"># Remove no manure treatment</span></span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum, </span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a>           plot, </span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>           treatment) <span class="sc">%&gt;%</span></span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>( </span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">n=</span><span class="fu">n</span>(),</span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean=</span><span class="fu">mean</span>(Abundance),</span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">sd=</span><span class="fu">sd</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">se =</span> sd<span class="sc">/</span><span class="fu">sqrt</span>(n))  <span class="sc">%&gt;%</span></span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">ic =</span> se <span class="sc">*</span> <span class="fu">qt</span>((<span class="dv">1</span><span class="fl">-0.05</span>)<span class="sc">/</span><span class="dv">2</span> <span class="sc">+</span> .<span class="dv">5</span>, n<span class="dv">-1</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(plot, <span class="sc">-</span>mean) <span class="sc">%&gt;%</span> </span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'Phylum', 'plot'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="sourceCode" id="cb37"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>EightPhyla <span class="ot">&lt;-</span> summary <span class="sc">%&gt;%</span></span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">mean =</span> <span class="fu">mean</span>(mean)) <span class="sc">%&gt;%</span></span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="sc">-</span>mean) <span class="sc">%&gt;%</span></span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Phylum) <span class="sc">%&gt;%</span></span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>() <span class="sc">%&gt;%</span></span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">head</span>(<span class="dv">8</span>) <span class="co"># Use head with n = 8 to select eight phyla</span></span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>summary<span class="sc">$</span>plot <span class="ot">&lt;-</span> <span class="fu">as.factor</span>(summary<span class="sc">$</span>plot)</span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>summary <span class="ot">&lt;-</span> summary <span class="sc">%&gt;%</span></span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Phylum <span class="sc">%in%</span> EightPhyla<span class="sc">$</span>Phylum)</span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(summary, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">y =</span> mean, <span class="at">fill =</span> plot)) <span class="sc">+</span></span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, </span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Phylum, <span class="sc">-</span>mean), <span class="at">ymin =</span> mean <span class="sc">-</span> ic, <span class="at">ymax =</span> mean <span class="sc">+</span> ic), </span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a>           <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="dv">1</span>), <span class="at">width =</span> <span class="fl">0.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size=</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="st">"Relative abundance of phyla"</span>, <span class="at">x =</span> <span class="st">"Phylum"</span>) <span class="sc">+</span></span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_grid</span>( <span class="sc">~</span> treatment) <span class="sc">+</span></span>
<span id="cb37-21"><a href="#cb37-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span> </span>
<span id="cb37-22"><a href="#cb37-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> global_size, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb37-23"><a href="#cb37-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb37-24"><a href="#cb37-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">name =</span> <span class="st">"Plot"</span>) <span class="sc">+</span></span>
<span id="cb37-25"><a href="#cb37-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="co"># Here we can multiply the relative abundance by 100 and add a % mark, note that any tables generated will still need to be multiplied by 100. </span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-24-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb38"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig3.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">width =</span> <span class="dv">7</span>, <span class="at">height =</span> <span class="dv">6</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>LAB treatment mean ::: {.cell hash=“ManuscriptWalkthrough_cache/html/unnamed-chunk-26_3bab828b21c571d2e8da0531b5f4b565”}</p>
<div class="sourceCode" id="cb39"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>df <span class="sc">%&gt;%</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(treatment, plot, time) <span class="sc">%&gt;%</span></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">MAB_Abundance =</span> <span class="fu">sum</span>(Abundance)) <span class="sc">%&gt;%</span> </span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(treatment, plot) <span class="sc">%&gt;%</span></span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean_mab =</span> <span class="fu">mean</span>(MAB_Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(plot) <span class="sc">%&gt;%</span></span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">trtMeanMab =</span> <span class="fu">mean</span>(mean_mab))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'treatment', 'plot'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'treatment'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="cell-output-stdout">
<pre><code># A tibble: 9 x 2
   plot trtMeanMab
  &lt;dbl&gt;      &lt;dbl&gt;
1     1    0      
2     2    0      
3     3    0      
4     4    0.552  
5     5    0.0110 
6     6    0.00284
7     7    0.0893 
8     8    0.100  
9     9    0.312  </code></pre>
</div>
<p>:::</p>
<p>ARG Figures</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-28_7dc20bddee5e6383faf643ce84a0c326">
<div class="sourceCode" id="cb43"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb43-2"><a href="#cb43-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(viridisLite)</span>
<span id="cb43-3"><a href="#cb43-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggh4x)</span>
<span id="cb43-4"><a href="#cb43-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpubr)</span>
<span id="cb43-5"><a href="#cb43-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(rstatix)</span>
<span id="cb43-6"><a href="#cb43-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggtext)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-30_c6c4f30e3d299a3e1e486c55d4820364">
<div class="sourceCode" id="cb44"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data/metawithbiomark.RDS"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-32_fd3fee2131f5b6d2d442528a57a1edff">
<div class="sourceCode" id="cb45"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a>standard <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(sample_type <span class="sc">%in%</span> <span class="st">"standard"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-34_40d6f3f7eb308df9440caa554c40e548">
<div class="sourceCode" id="cb46"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a>PC_Standards_no_NA <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data/ampliconlengths.RDS"</span>)</span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a>PC_Standards_no_NA</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 46 x 3
   gene          Assay       amp_len
   &lt;chr&gt;         &lt;chr&gt;       &lt;chr&gt;  
 1 aadA9_F       aadA9       251    
 2 ant6-ia_F     ant6-ia     313    
 3 ant6-ib_F     ant6-ib     251    
 4 aph(3'')-ia_F aph(3'')-ia 301    
 5 erm(35)_F     erm(35)     251    
 6 erm(B)_F      erm(B)      251    
 7 erm(C)_F      erm(C)      251    
 8 erm(F)_F      erm(F)      251    
 9 erm(Q)_F      erm(Q)      287    
10 ermA/ermTR_F  ermA/ermTR  251    
# … with 36 more rows</code></pre>
</div>
</div>
<p>Calculating limit of detection for each gene</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-36_c3ccbd3d331f2105d5910f0b4a830b76">
<div class="sourceCode" id="cb48"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb48-1"><a href="#cb48-1" aria-hidden="true" tabindex="-1"></a>filtered_standard <span class="ot">&lt;-</span> standard <span class="sc">%&gt;%</span> </span>
<span id="cb48-2"><a href="#cb48-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Sample_Name <span class="sc">==</span> Assay)</span>
<span id="cb48-3"><a href="#cb48-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-4"><a href="#cb48-4" aria-hidden="true" tabindex="-1"></a>lod_max_by_gene <span class="ot">&lt;-</span> filtered_standard <span class="sc">%&gt;%</span> </span>
<span id="cb48-5"><a href="#cb48-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(rConc, Sample_Name) <span class="sc">%&gt;%</span></span>
<span id="cb48-6"><a href="#cb48-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="fu">any</span>(Call <span class="sc">==</span> <span class="st">"Pass"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb48-7"><a href="#cb48-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Value =</span> <span class="fu">as.numeric</span>(Value)) <span class="sc">%&gt;%</span> </span>
<span id="cb48-8"><a href="#cb48-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>Call <span class="sc">==</span> <span class="st">"Flag"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb48-9"><a href="#cb48-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">max_lod =</span> <span class="fu">max</span>(Value)) <span class="sc">%&gt;%</span> </span>
<span id="cb48-10"><a href="#cb48-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample_Name) <span class="sc">%&gt;%</span> </span>
<span id="cb48-11"><a href="#cb48-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">slice_max</span>(max_lod) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`summarise()` has grouped output by 'rConc'. You can override using the `.groups` argument.</code></pre>
</div>
<div class="sourceCode" id="cb50"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a>lod_max_by_gene <span class="sc">%&gt;%</span> </span>
<span id="cb50-2"><a href="#cb50-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(Sample_Name, max_lod)) <span class="sc">+</span></span>
<span id="cb50-3"><a href="#cb50-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>(<span class="fu">aes</span>(<span class="at">fill =</span> rConc)) <span class="sc">+</span></span>
<span id="cb50-4"><a href="#cb50-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_light</span>() <span class="sc">+</span></span>
<span id="cb50-5"><a href="#cb50-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb50-6"><a href="#cb50-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>)</span>
<span id="cb50-7"><a href="#cb50-7" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb50-8"><a href="#cb50-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">expand =</span> <span class="fu">expansion</span>(<span class="at">add =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">0</span>), <span class="at">mult =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.1</span>))) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-36-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb51"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a>counts <span class="ot">&lt;-</span> filtered_standard <span class="sc">%&gt;%</span></span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(rConc, Sample_Name) <span class="sc">%&gt;%</span> </span>
<span id="cb51-3"><a href="#cb51-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="fu">any</span>(Call <span class="sc">==</span> <span class="st">"Pass"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb51-4"><a href="#cb51-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Sample_Name, Assay, rConc, Value, Call)  <span class="sc">%&gt;%</span></span>
<span id="cb51-5"><a href="#cb51-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span>  </span>
<span id="cb51-6"><a href="#cb51-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>Call <span class="sc">==</span> <span class="st">"Flag"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb51-7"><a href="#cb51-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">left_join</span>(PC_Standards_no_NA) <span class="sc">%&gt;%</span></span>
<span id="cb51-8"><a href="#cb51-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(Assay, rConc, Value, amp_len) <span class="sc">%&gt;%</span></span>
<span id="cb51-9"><a href="#cb51-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">counts =</span> <span class="fu">as.numeric</span>(rConc) <span class="sc">*</span> (<span class="dv">1</span><span class="sc">/</span>(<span class="dv">10</span><span class="sc">^</span><span class="dv">9</span>)) <span class="sc">*</span> (<span class="dv">1</span><span class="sc">/</span><span class="dv">660</span>) <span class="sc">*</span> ((<span class="fl">6.023</span><span class="sc">*</span><span class="dv">10</span><span class="sc">^</span><span class="dv">23</span>)<span class="sc">/</span><span class="dv">1</span>) <span class="sc">*</span> (<span class="dv">1</span><span class="sc">/</span><span class="fu">as.numeric</span>(amp_len)) <span class="sc">*</span> (.<span class="dv">0067</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb51-10"><a href="#cb51-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">Value =</span> <span class="fu">as.numeric</span>(Value))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>Joining, by = "Assay"</code></pre>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-38_5186db1fa413f38859fb90d05a89794a">
<div class="sourceCode" id="cb53"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a>lod_max_by_gene</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 16 x 3
# Groups:   Sample_Name [16]
   rConc Sample_Name        max_lod
   &lt;chr&gt; &lt;chr&gt;                &lt;dbl&gt;
 1 1e-07 16S_Eub_338_518       24.7
 2 1e-06 aadA9                 23.4
 3 1e-07 erm(C)                26.4
 4 0.001 intI1F165_clinical    27.3
 5 1e-05 IS6100                22.2
 6 1e-06 strB                  22.9
 7 1e-06 sul1 NEW              22.7
 8 1e-06 sul2                  22.6
 9 1e-07 tetbP                 24.7
10 1e-06 tetG_F                23.5
11 1e-06 tetH                  23.3
12 1e-05 tetL                  19.5
13 1e-06 tetM                  22.5
14 1e-06 tetT                  21.9
15 1e-06 tetW                  21.5
16 1e-06 tetX                  22.6</code></pre>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-40_e71e991b8407172d30b43a94df19c029">
<div class="sourceCode" id="cb55"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a>lm_eqn <span class="ot">=</span> <span class="cf">function</span>(df){</span>
<span id="cb55-2"><a href="#cb55-2" aria-hidden="true" tabindex="-1"></a>    m <span class="ot">=</span> <span class="fu">lm</span>(<span class="at">data =</span> df, <span class="at">formula =</span> <span class="fu">as.numeric</span>(Value) <span class="sc">~</span> <span class="fu">log10</span>(counts <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb55-3"><a href="#cb55-3" aria-hidden="true" tabindex="-1"></a>    b <span class="ot">=</span> <span class="fu">coef</span>(m)[<span class="dv">2</span>]  </span>
<span id="cb55-4"><a href="#cb55-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">data.frame</span>(</span>
<span id="cb55-5"><a href="#cb55-5" aria-hidden="true" tabindex="-1"></a>      <span class="at">eff =</span> <span class="fu">round</span>(<span class="dv">100</span><span class="sc">*</span>((<span class="dv">10</span><span class="sc">^</span>(<span class="sc">-</span><span class="dv">1</span><span class="sc">/</span>b))<span class="sc">-</span><span class="dv">1</span>), <span class="at">digits =</span> <span class="dv">2</span>),</span>
<span id="cb55-6"><a href="#cb55-6" aria-hidden="true" tabindex="-1"></a>      <span class="at">r2 =</span> <span class="fu">format</span>(<span class="fu">summary</span>(m)<span class="sc">$</span>r.squared, <span class="at">digits =</span> <span class="dv">3</span>),</span>
<span id="cb55-7"><a href="#cb55-7" aria-hidden="true" tabindex="-1"></a>      <span class="at">LOD =</span> <span class="fu">round</span>(<span class="fu">max</span>(df<span class="sc">$</span>Value), <span class="at">digits =</span> <span class="dv">2</span>),</span>
<span id="cb55-8"><a href="#cb55-8" aria-hidden="true" tabindex="-1"></a>      <span class="at">Intercept =</span> <span class="fu">round</span>(<span class="fu">coef</span>(m)[<span class="dv">1</span>], <span class="at">digits =</span> <span class="dv">2</span>),</span>
<span id="cb55-9"><a href="#cb55-9" aria-hidden="true" tabindex="-1"></a>      <span class="at">Slope =</span> <span class="fu">round</span>(<span class="fu">coef</span>(m)[<span class="dv">2</span>], <span class="at">digits =</span> <span class="dv">2</span>)</span>
<span id="cb55-10"><a href="#cb55-10" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb55-11"><a href="#cb55-11" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb55-12"><a href="#cb55-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-13"><a href="#cb55-13" aria-hidden="true" tabindex="-1"></a>st_splits <span class="ot">&lt;-</span>  counts <span class="sc">%&gt;%</span> </span>
<span id="cb55-14"><a href="#cb55-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Assay) <span class="sc">%&gt;%</span> </span>
<span id="cb55-15"><a href="#cb55-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_split</span>() </span>
<span id="cb55-16"><a href="#cb55-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-17"><a href="#cb55-17" aria-hidden="true" tabindex="-1"></a>eqs <span class="ot">&lt;-</span> st_splits <span class="sc">%&gt;%</span> </span>
<span id="cb55-18"><a href="#cb55-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">lapply</span>(., lm_eqn) <span class="sc">%&gt;%</span> </span>
<span id="cb55-19"><a href="#cb55-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">bind_rows</span>()</span>
<span id="cb55-20"><a href="#cb55-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-21"><a href="#cb55-21" aria-hidden="true" tabindex="-1"></a>labels <span class="ot">&lt;-</span> <span class="fu">lapply</span>(st_splits, slice_head, <span class="at">n =</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb55-22"><a href="#cb55-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">bind_rows</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb55-23"><a href="#cb55-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Assay) <span class="sc">%&gt;%</span> </span>
<span id="cb55-24"><a href="#cb55-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">bind_cols</span>(eqs) <span class="sc">%&gt;%</span> </span>
<span id="cb55-25"><a href="#cb55-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">eq_label =</span> <span class="fu">paste0</span>(<span class="st">"LOD = "</span>, LOD, <span class="st">"&lt;br&gt;r^2 = "</span>, r2, <span class="st">"&lt;br&gt;Efficiency = "</span>, eff, <span class="st">"&lt;br&gt;Slope = "</span>, Slope, <span class="st">"&lt;br&gt;Intercept = "</span>, Intercept))</span>
<span id="cb55-26"><a href="#cb55-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-27"><a href="#cb55-27" aria-hidden="true" tabindex="-1"></a>SCPlots <span class="ot">&lt;-</span> counts <span class="sc">%&gt;%</span></span>
<span id="cb55-28"><a href="#cb55-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="fu">as.numeric</span>(<span class="fu">log10</span>(counts <span class="sc">+</span> <span class="dv">1</span>)), <span class="fu">as.numeric</span>(Value))) <span class="sc">+</span></span>
<span id="cb55-29"><a href="#cb55-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb55-30"><a href="#cb55-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>( <span class="sc">~</span> Assay, <span class="at">scales =</span> <span class="st">"free"</span>) <span class="sc">+</span> </span>
<span id="cb55-31"><a href="#cb55-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb55-32"><a href="#cb55-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.border =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">1</span>, <span class="at">fill =</span> <span class="cn">NA</span>),</span>
<span id="cb55-33"><a href="#cb55-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.minor.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb55-34"><a href="#cb55-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.minor.y =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb55-35"><a href="#cb55-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.spacing =</span> <span class="fu">unit</span>(<span class="fl">0.5</span>, <span class="st">"lines"</span>),</span>
<span id="cb55-36"><a href="#cb55-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">1</span>, <span class="at">fill =</span> <span class="cn">NA</span>),</span>
<span id="cb55-37"><a href="#cb55-37" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span> </span>
<span id="cb55-38"><a href="#cb55-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb55-39"><a href="#cb55-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">"Log10 (gene copies per reaction)"</span>,</span>
<span id="cb55-40"><a href="#cb55-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Ct"</span></span>
<span id="cb55-41"><a href="#cb55-41" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span> </span>
<span id="cb55-42"><a href="#cb55-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">7</span>), <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">7</span>, <span class="dv">1</span>), <span class="at">expand =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">0</span>)) <span class="sc">+</span> </span>
<span id="cb55-43"><a href="#cb55-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">25</span>)) <span class="sc">+</span></span>
<span id="cb55-44"><a href="#cb55-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="fu">aes</span>(<span class="at">group=</span><span class="dv">1</span>), <span class="at">method=</span><span class="st">"lm"</span>, <span class="at">se=</span><span class="cn">FALSE</span>) <span class="sc">+</span> </span>
<span id="cb55-45"><a href="#cb55-45" aria-hidden="true" tabindex="-1"></a>  ggtext<span class="sc">::</span><span class="fu">geom_richtext</span>(<span class="at">data =</span> labels, <span class="fu">aes</span>(<span class="at">x =</span> <span class="dv">2</span>, <span class="at">y =</span> <span class="dv">5</span>, <span class="at">label =</span> eq_label),</span>
<span id="cb55-46"><a href="#cb55-46" aria-hidden="true" tabindex="-1"></a>                        <span class="at">size =</span> <span class="dv">4</span>, <span class="at">fontface =</span> <span class="st">"bold"</span>, <span class="at">inherit.aes =</span> <span class="cn">FALSE</span>) <span class="sc">+</span></span>
<span id="cb55-47"><a href="#cb55-47" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb55-48"><a href="#cb55-48" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb55-49"><a href="#cb55-49" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb55-50"><a href="#cb55-50" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb55-51"><a href="#cb55-51" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb55-52"><a href="#cb55-52" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb55-53"><a href="#cb55-53" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb55-54"><a href="#cb55-54" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.key.width =</span> <span class="fu">unit</span>(.<span class="dv">75</span>, <span class="st">"in"</span>)</span>
<span id="cb55-55"><a href="#cb55-55" aria-hidden="true" tabindex="-1"></a>  ) </span>
<span id="cb55-56"><a href="#cb55-56" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-57"><a href="#cb55-57" aria-hidden="true" tabindex="-1"></a>SCPlots</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`geom_smooth()` using formula 'y ~ x'</code></pre>
</div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-40-1.png" class="img-fluid" width="1152"></p>
</div>
<div class="sourceCode" id="cb57"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save a table of the information from each curve</span></span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a>LODtable <span class="ot">&lt;-</span> labels <span class="sc">%&gt;%</span></span>
<span id="cb57-3"><a href="#cb57-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="sc">!</span>eq_label)</span>
<span id="cb57-4"><a href="#cb57-4" aria-hidden="true" tabindex="-1"></a>LODtable</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 16 x 6
   Assay                eff r2      LOD Intercept Slope
   &lt;chr&gt;              &lt;dbl&gt; &lt;chr&gt; &lt;dbl&gt;     &lt;dbl&gt; &lt;dbl&gt;
 1 16S_Eub_338_518     95.1 0.952  24.7      28.3 -3.45
 2 aadA9               81.7 0.932  23.4      29.4 -3.86
 3 erm(C)              82.6 0.969  26.4      29.2 -3.82
 4 intI1F165_clinical  49.4 0.855  27.3      47.0 -5.73
 5 IS6100              89.7 0.947  22.2      26.1 -3.6 
 6 strB                81.5 0.981  22.9      29.5 -3.86
 7 sul1 NEW            94.4 0.989  22.7      26.1 -3.46
 8 sul2               106.  0.975  22.6      25.1 -3.19
 9 tetbP               92.7 0.99   24.7      25.9 -3.51
10 tetG_F              87.7 0.985  23.5      26.2 -3.66
11 tetH                82.2 0.999  23.3      28.3 -3.84
12 tetL                85.9 0.999  19.5      28.3 -3.71
13 tetM                78.3 0.997  22.5      27.4 -3.98
14 tetT                86.3 0.998  21.9      26.4 -3.7 
15 tetW                97.6 0.997  21.5      26.5 -3.38
16 tetX                77.6 0.998  22.6      27.8 -4.01</code></pre>
</div>
<div class="sourceCode" id="cb59"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a><span class="fu">write_csv</span>(LODtable, <span class="at">file =</span> <span class="st">"Data/standardcurves.csv"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Calculating counts in an environmental sample</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-42_2f31234896efcfe3e554682a9ea13e47">
<div class="sourceCode" id="cb60"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a>counts</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 425 x 5
   Assay           rConc Value amp_len counts
   &lt;chr&gt;           &lt;chr&gt; &lt;dbl&gt; &lt;chr&gt;    &lt;dbl&gt;
 1 tetbP           1e-07  24.7 251       2.44
 2 sul1 NEW        1e-07  21.8 251       2.44
 3 IS6100          1e-07  21.4 251       2.44
 4 erm(C)          1e-07  26.4 251       2.44
 5 16S_Eub_338_518 1e-07  24.7 257       2.38
 6 tetbP           1e-06  22.1 251      24.4 
 7 tetbP           1e-06  20.7 251      24.4 
 8 tetbP           1e-06  20.5 251      24.4 
 9 sul1 NEW        1e-06  22.7 251      24.4 
10 sul1 NEW        1e-06  20.8 251      24.4 
# … with 415 more rows</code></pre>
</div>
</div>
<p>Environment Ct values</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-44_99814bcbd39971305183ff11d2a845aa">
<div class="sourceCode" id="cb62"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a>water <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb62-2"><a href="#cb62-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(sample_type <span class="sc">%in%</span> <span class="st">"water"</span>, <span class="sc">!</span>Sample_Name <span class="sc">==</span> <span class="st">"Supply_Water"</span>)</span>
<span id="cb62-3"><a href="#cb62-3" aria-hidden="true" tabindex="-1"></a>water</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 6,336 x 21
   Sample_Name Sample_Type Assay  Assay_Type ID     rConc Value `Calibrated rCo…
   &lt;chr&gt;       &lt;chr&gt;       &lt;chr&gt;  &lt;chr&gt;      &lt;chr&gt;  &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;           
 1 P2_5        Unknown     16S_E… Test       S06-A… 1     18.3… 4.9938279547795…
 2 P2_5        Unknown     16S_E… Test       S06-A… 1     18.3… 5.0738800340691…
 3 P2_5        Unknown     16S_E… Test       S06-A… 1     17.7… 7.6293726452979…
 4 P2_5        Unknown     sul1 … Test       S06-A… 1     999   -1              
 5 P2_5        Unknown     sul1 … Test       S06-A… 1     999   -1              
 6 P2_5        Unknown     sul1 … Test       S06-A… 1     999   -1              
 7 P2_5        Unknown     erm(3… Test       S06-A… 1     999   -1              
 8 P2_5        Unknown     erm(3… Test       S06-A… 1     999   -1              
 9 P2_5        Unknown     erm(3… Test       S06-A… 1     999   -1              
10 P2_5        Unknown     tetW   Test       S06-A… 1     999   -1              
# … with 6,326 more rows, and 13 more variables: Quality &lt;chr&gt;, Call &lt;chr&gt;,
#   Threshold &lt;chr&gt;, In Range &lt;chr&gt;, Out Range &lt;chr&gt;, Peak Ratio &lt;chr&gt;,
#   soil_type &lt;chr&gt;, plot &lt;chr&gt;, treatment &lt;chr&gt;, block &lt;chr&gt;,
#   sample_day &lt;fct&gt;, sample_number &lt;chr&gt;, sample_type &lt;chr&gt;</code></pre>
</div>
<div class="sourceCode" id="cb64"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a>soil <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb64-2"><a href="#cb64-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(sample_type <span class="sc">%in%</span> <span class="st">"soil"</span>)</span>
<span id="cb64-3"><a href="#cb64-3" aria-hidden="true" tabindex="-1"></a>soil</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 10,944 x 21
   Sample_Name Sample_Type Assay  Assay_Type ID     rConc Value `Calibrated rCo…
   &lt;chr&gt;       &lt;chr&gt;       &lt;chr&gt;  &lt;chr&gt;      &lt;chr&gt;  &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;           
 1 C_1_TB_1    Unknown     16S_E… Test       S95-A… 1     15.1… 0.0001944385882…
 2 C_1_TB_1    Unknown     16S_E… Test       S95-A… 1     14.8… 0.0002366823806…
 3 C_1_TB_1    Unknown     16S_E… Test       S95-A… 1     14.9… 0.0002276031810…
 4 C_1_TB_1    Unknown     sul1 … Test       S95-A… 1     999   -1              
 5 C_1_TB_1    Unknown     sul1 … Test       S95-A… 1     999   -1              
 6 C_1_TB_1    Unknown     sul1 … Test       S95-A… 1     999   -1              
 7 C_1_TB_1    Unknown     erm(3… Test       S95-A… 1     999   -1              
 8 C_1_TB_1    Unknown     erm(3… Test       S95-A… 1     999   -1              
 9 C_1_TB_1    Unknown     erm(3… Test       S95-A… 1     999   -1              
10 C_1_TB_1    Unknown     tetW   Test       S95-A… 1     999   -1              
# … with 10,934 more rows, and 13 more variables: Quality &lt;chr&gt;, Call &lt;chr&gt;,
#   Threshold &lt;chr&gt;, In Range &lt;chr&gt;, Out Range &lt;chr&gt;, Peak Ratio &lt;chr&gt;,
#   soil_type &lt;chr&gt;, plot &lt;chr&gt;, treatment &lt;chr&gt;, block &lt;chr&gt;,
#   sample_day &lt;fct&gt;, sample_number &lt;chr&gt;, sample_type &lt;chr&gt;</code></pre>
</div>
<div class="sourceCode" id="cb66"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb66-1"><a href="#cb66-1" aria-hidden="true" tabindex="-1"></a><span class="fu">unique</span>(water<span class="sc">$</span>Sample_Name)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code> [1] "P2_5" "P2_3" "P2_1" "P1_5" "P1_3" "P1_1" "P2_6" "P2_4" "P2_2" "P1_6"
[11] "P1_4" "P1_2" "P3_5" "P3_3" "P3_1" "P3_6" "P3_4" "P3_2" "P9_6" "P4_5"
[21] "P9_4" "P4_3" "P9_2" "P4_1" "P9_5" "P4_6" "P9_3" "P4_4" "P9_1" "P4_2"
[31] "P8_6" "P5_5" "P8_4" "P5_3" "P8_2" "P5_1" "P8_5" "P5_6" "P8_3" "P5_4"
[41] "P8_1" "P5_2" "P7_6" "P6_5" "P7_4" "P6_3" "P7_2" "P6_1" "P7_5" "P6_6"
[51] "P7_3" "P6_4" "P7_1" "P6_2"</code></pre>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-46_466b126e4c64155f3830e6d0070067bc">
<div class="sourceCode" id="cb68"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb68-1"><a href="#cb68-1" aria-hidden="true" tabindex="-1"></a>get_counts <span class="ot">&lt;-</span> <span class="cf">function</span>(stddf, ctdf, gene){</span>
<span id="cb68-2"><a href="#cb68-2" aria-hidden="true" tabindex="-1"></a>  <span class="co">#stddf &lt;- counts # use to test</span></span>
<span id="cb68-3"><a href="#cb68-3" aria-hidden="true" tabindex="-1"></a>  <span class="co">#ctdf &lt;- water # use to test</span></span>
<span id="cb68-4"><a href="#cb68-4" aria-hidden="true" tabindex="-1"></a>  <span class="co">#gene &lt;- "tetM" # use to test</span></span>
<span id="cb68-5"><a href="#cb68-5" aria-hidden="true" tabindex="-1"></a>  std <span class="ot">&lt;-</span> stddf <span class="sc">%&gt;%</span></span>
<span id="cb68-6"><a href="#cb68-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(Assay <span class="sc">==</span> gene)</span>
<span id="cb68-7"><a href="#cb68-7" aria-hidden="true" tabindex="-1"></a>  ctd <span class="ot">&lt;-</span> ctdf <span class="sc">%&gt;%</span></span>
<span id="cb68-8"><a href="#cb68-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(Assay <span class="sc">==</span> gene)</span>
<span id="cb68-9"><a href="#cb68-9" aria-hidden="true" tabindex="-1"></a>  inverse.lm <span class="ot">&lt;-</span> <span class="fu">lm</span>(<span class="at">data =</span> std, <span class="at">formula =</span> <span class="fu">log10</span>(counts<span class="sc">+</span><span class="dv">1</span>) <span class="sc">~</span> <span class="fu">as.numeric</span>(Value)) <span class="co"># Note log10 transformation</span></span>
<span id="cb68-10"><a href="#cb68-10" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb68-11"><a href="#cb68-11" aria-hidden="true" tabindex="-1"></a>  val2 <span class="ot">&lt;-</span> ctd<span class="sc">$</span>Value</span>
<span id="cb68-12"><a href="#cb68-12" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb68-13"><a href="#cb68-13" aria-hidden="true" tabindex="-1"></a>  ctd<span class="sc">$</span>counts <span class="ot">&lt;-</span> <span class="dv">10</span> <span class="sc">^</span> <span class="fu">predict</span>(inverse.lm ,<span class="fu">data.frame</span>(<span class="at">Value =</span> val2), <span class="at">interval =</span> <span class="st">"predict"</span>)[,<span class="dv">1</span>] </span>
<span id="cb68-14"><a href="#cb68-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb68-15"><a href="#cb68-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(ctd)</span>
<span id="cb68-16"><a href="#cb68-16" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-48_b036187e847818ee1a0aeffdfa68e6ba">
<div class="sourceCode" id="cb69"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a><span class="fu">unique</span>(counts<span class="sc">$</span>Assay)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code> [1] "tetbP"              "sul1 NEW"           "IS6100"            
 [4] "erm(C)"             "16S_Eub_338_518"    "strB"              
 [7] "intI1F165_clinical" "aadA9"              "tetG_F"            
[10] "sul2"               "tetW"               "tetT"              
[13] "tetL"               "tetM"               "tetH"              
[16] "tetX"              </code></pre>
</div>
<div class="sourceCode" id="cb71"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a><span class="fu">unique</span>(water<span class="sc">$</span>Assay)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code> [1] "16S_Eub_338_518"    "sul1 NEW"           "erm(36)"           
 [4] "tetW"               "aac3-Via"           "sul2"              
 [7] "erm(C)"             "tetX"               "aadA17"            
[10] "tetbP"              "int1-a-marko"       "tnpA1"             
[13] "aadA9"              "tetG_F"             "intI1F165_clinical"
[16] "tnpA2"              "tetH"               "aadD"              
[19] "tnpA3"              "lnuA"               "tetL"              
[22] "aphA3"              "tnpA5"              "IS1247"            
[25] "tetM"               "cadC"               "tnpA6"             
[28] "IS6100"             "tetT"               "cmr"               
[31] "tnpA7"              "strB"              </code></pre>
</div>
</div>
<p>By changing high Ct values to NA, we clean up the heatmap by converting these “non detects” to gray color tiles</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-50_73fe2b2d7830f8f1123dfd807708b028">
<div class="sourceCode" id="cb73"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Change high Ct values to NA</span></span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a>water<span class="sc">$</span>Value[water<span class="sc">$</span>Value <span class="sc">&gt;</span> <span class="dv">30</span>] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb73-3"><a href="#cb73-3" aria-hidden="true" tabindex="-1"></a>soil<span class="sc">$</span>Value[soil<span class="sc">$</span>Value <span class="sc">&gt;</span> <span class="dv">30</span>] <span class="ot">&lt;-</span> <span class="cn">NA</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-52_d2b28dc6dd1d689601de278655c43c98">
<div class="cell-output-stdout">
<pre><code>c("tetbP", "sul1 NEW", "IS6100", "erm(C)", "16S_Eub_338_518", 
"strB", "intI1F165_clinical", "aadA9", "tetG_F", "sul2", "tetW", 
"tetT", "tetL", "tetM", "tetH", "tetX")</code></pre>
</div>
<div class="cell-output-stderr">
<pre><code>Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")
Joining, by = c("Sample_Name", "Sample_Type", "Assay", "Assay_Type", "ID", "rConc", "Value", "Calibrated rConc", "Quality", "Call", "Threshold", "In Range", "Out Range", "Peak Ratio", "soil_type", "plot", "treatment", "block", "sample_day", "sample_number", "sample_type", "counts")</code></pre>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-54_c3da39808bcc2fb0c83d86b15e1eef65">
<div class="sourceCode" id="cb76"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb76-1"><a href="#cb76-1" aria-hidden="true" tabindex="-1"></a>WaterCounts <span class="ot">&lt;-</span> WaterCounts <span class="sc">%&gt;%</span> <span class="co"># Add classes to ARGs</span></span>
<span id="cb76-2"><a href="#cb76-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">gene_class =</span> <span class="fu">case_when</span>(Assay <span class="sc">%in%</span> <span class="fu">unique</span>(WaterCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"^[aa]"</span>, WaterCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"AMG"</span>,</span>
<span id="cb76-3"><a href="#cb76-3" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">unique</span>(WaterCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"^[erm]"</span>, WaterCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"MLSB"</span>,</span>
<span id="cb76-4"><a href="#cb76-4" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">unique</span>(WaterCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"^[sul]"</span>, WaterCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"Sulfa"</span>,</span>
<span id="cb76-5"><a href="#cb76-5" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">unique</span>(WaterCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"tet"</span>, WaterCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"Tet"</span>,</span>
<span id="cb76-6"><a href="#cb76-6" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"intl1"</span>, <span class="st">"intl2"</span>, <span class="st">"intl3"</span>, <span class="st">"IS1247"</span>, <span class="st">"IS6100"</span>, <span class="st">"tnpA2"</span>,</span>
<span id="cb76-7"><a href="#cb76-7" aria-hidden="true" tabindex="-1"></a>                                              <span class="st">"tnpA5"</span>, <span class="st">"tnpA6"</span>, <span class="st">"tnpA3"</span>, <span class="st">"intI1F165_clinical"</span>, <span class="st">"tnpA1"</span>, <span class="st">"int1-a-marko"</span>, <span class="st">"tnpA7"</span>) <span class="sc">~</span> <span class="st">"MGE"</span>, </span>
<span id="cb76-8"><a href="#cb76-8" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"cadC"</span>, <span class="st">"cmr"</span>) <span class="sc">~</span> <span class="st">"HM"</span>,</span>
<span id="cb76-9"><a href="#cb76-9" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">==</span> <span class="fu">c</span>(<span class="st">"16S_Eub_338_518"</span>) <span class="sc">~</span> <span class="st">"16S"</span>))</span>
<span id="cb76-10"><a href="#cb76-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb76-11"><a href="#cb76-11" aria-hidden="true" tabindex="-1"></a>WaterCounts<span class="sc">$</span>gene_class_f <span class="ot">=</span> <span class="fu">factor</span>(WaterCounts<span class="sc">$</span>gene_class, <span class="co"># Order the classes</span></span>
<span id="cb76-12"><a href="#cb76-12" aria-hidden="true" tabindex="-1"></a>                                <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">'AMG'</span>, <span class="st">'MLSB'</span>, <span class="st">'Sulfa'</span>, <span class="st">"Tet"</span>, <span class="st">"MGE"</span>, <span class="st">"HM"</span>, <span class="st">"16S"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-56_38abba28ba2a47ad19de0905c14396a5">
<div class="sourceCode" id="cb77"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a>SoilCounts <span class="ot">&lt;-</span> SoilCounts <span class="sc">%&gt;%</span> <span class="co"># Add classes to ARGs</span></span>
<span id="cb77-2"><a href="#cb77-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">gene_class =</span> <span class="fu">case_when</span>(Assay <span class="sc">%in%</span> <span class="fu">unique</span>(SoilCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"^[aa]"</span>, SoilCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"AMG"</span>,</span>
<span id="cb77-3"><a href="#cb77-3" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">unique</span>(SoilCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"^[erm]"</span>, SoilCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"MLSB"</span>,</span>
<span id="cb77-4"><a href="#cb77-4" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">unique</span>(SoilCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"^[sul]"</span>, SoilCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"Sulfa"</span>,</span>
<span id="cb77-5"><a href="#cb77-5" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">unique</span>(SoilCounts<span class="sc">$</span>Assay[<span class="fu">grepl</span>(<span class="st">"tet"</span>, SoilCounts<span class="sc">$</span>Assay)]) <span class="sc">~</span> <span class="st">"Tet"</span>,</span>
<span id="cb77-6"><a href="#cb77-6" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"intl1"</span>, <span class="st">"intl2"</span>, <span class="st">"intl3"</span>, <span class="st">"IS1247"</span>, <span class="st">"IS6100"</span>, <span class="st">"tnpA2"</span>,</span>
<span id="cb77-7"><a href="#cb77-7" aria-hidden="true" tabindex="-1"></a>                                              <span class="st">"tnpA5"</span>, <span class="st">"tnpA6"</span>, <span class="st">"tnpA3"</span>, <span class="st">"intI1F165_clinical"</span>, <span class="st">"tnpA1"</span>, <span class="st">"int1-a-marko"</span>, <span class="st">"tnpA7"</span>) <span class="sc">~</span> <span class="st">"MGE"</span>, </span>
<span id="cb77-8"><a href="#cb77-8" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"cadC"</span>, <span class="st">"cmr"</span>) <span class="sc">~</span> <span class="st">"HM"</span>,</span>
<span id="cb77-9"><a href="#cb77-9" aria-hidden="true" tabindex="-1"></a>                                Assay <span class="sc">==</span> <span class="fu">c</span>(<span class="st">"16S_Eub_338_518"</span>) <span class="sc">~</span> <span class="st">"16S"</span>))</span>
<span id="cb77-10"><a href="#cb77-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb77-11"><a href="#cb77-11" aria-hidden="true" tabindex="-1"></a>SoilCounts<span class="sc">$</span>gene_class_f <span class="ot">=</span> <span class="fu">factor</span>(SoilCounts<span class="sc">$</span>gene_class, <span class="co"># Order the classes</span></span>
<span id="cb77-12"><a href="#cb77-12" aria-hidden="true" tabindex="-1"></a>                                <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">'AMG'</span>, <span class="st">'MLSB'</span>, <span class="st">'Sulfa'</span>, <span class="st">"Tet"</span>, <span class="st">"MGE"</span>, <span class="st">"HM"</span>, <span class="st">"16S"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-58_95ab34543aeade857a9f995243374d65">
<div class="sourceCode" id="cb78"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb78-1"><a href="#cb78-1" aria-hidden="true" tabindex="-1"></a>norm <span class="ot">&lt;-</span> WaterCounts <span class="sc">%&gt;%</span></span>
<span id="cb78-2"><a href="#cb78-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">unite</span>(ID, <span class="fu">c</span>(<span class="st">"plot"</span>, <span class="st">"sample_number"</span>, <span class="st">"Assay"</span>), <span class="at">sep =</span> <span class="st">"."</span>, <span class="at">remove =</span> F) <span class="sc">%&gt;%</span></span>
<span id="cb78-3"><a href="#cb78-3" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">group_by</span>(plot, sample_number, Assay) <span class="sc">%&gt;%</span></span>
<span id="cb78-4"><a href="#cb78-4" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">rep =</span> <span class="fu">seq_along</span>(ID)) <span class="sc">%&gt;%</span></span>
<span id="cb78-5"><a href="#cb78-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(plot, sample_number, rep, treatment, Assay, gene_class, counts, Value) <span class="sc">%&gt;%</span></span>
<span id="cb78-6"><a href="#cb78-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-60_789faf5884bbcf9b57016911a3181cf7">
<div class="sourceCode" id="cb79"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb79-1"><a href="#cb79-1" aria-hidden="true" tabindex="-1"></a>soilnorm <span class="ot">&lt;-</span> SoilCounts <span class="sc">%&gt;%</span></span>
<span id="cb79-2"><a href="#cb79-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">unite</span>(ID, <span class="fu">c</span>(<span class="st">"plot"</span>, <span class="st">"sample_day"</span>, <span class="st">"sample_number"</span>, <span class="st">"soil_type"</span>, <span class="st">"Assay"</span>), <span class="at">sep =</span> <span class="st">"."</span>, <span class="at">remove =</span> F) <span class="sc">%&gt;%</span></span>
<span id="cb79-3"><a href="#cb79-3" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">group_by</span>(plot, sample_day, sample_number, soil_type, Assay) <span class="sc">%&gt;%</span></span>
<span id="cb79-4"><a href="#cb79-4" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">rep =</span> <span class="fu">seq_along</span>(ID)) <span class="sc">%&gt;%</span></span>
<span id="cb79-5"><a href="#cb79-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(plot, sample_number, sample_day, soil_type, rep, treatment, Assay, gene_class, counts, Value) <span class="sc">%&gt;%</span></span>
<span id="cb79-6"><a href="#cb79-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-62_d0c74e6c727077d5d18220bc5bc55bcc">
<div class="sourceCode" id="cb80"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb80-1"><a href="#cb80-1" aria-hidden="true" tabindex="-1"></a>SixS <span class="ot">&lt;-</span> norm <span class="sc">%&gt;%</span></span>
<span id="cb80-2"><a href="#cb80-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Assay <span class="sc">==</span> <span class="st">"16S_Eub_338_518"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb80-3"><a href="#cb80-3" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">rename</span>(<span class="at">SixSCount =</span> counts) <span class="sc">%&gt;%</span></span>
<span id="cb80-4"><a href="#cb80-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(plot, sample_number, rep, SixSCount, treatment) </span>
<span id="cb80-5"><a href="#cb80-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb80-6"><a href="#cb80-6" aria-hidden="true" tabindex="-1"></a>SoilSixS <span class="ot">&lt;-</span> soilnorm <span class="sc">%&gt;%</span></span>
<span id="cb80-7"><a href="#cb80-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Assay <span class="sc">==</span> <span class="st">"16S_Eub_338_518"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb80-8"><a href="#cb80-8" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">rename</span>(<span class="at">SixSCount =</span> counts) <span class="sc">%&gt;%</span></span>
<span id="cb80-9"><a href="#cb80-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(plot, sample_number, sample_day, soil_type, rep, SixSCount, treatment)</span>
<span id="cb80-10"><a href="#cb80-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb80-11"><a href="#cb80-11" aria-hidden="true" tabindex="-1"></a>dfdf <span class="ot">&lt;-</span> norm <span class="sc">%&gt;%</span></span>
<span id="cb80-12"><a href="#cb80-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(SixS, <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"plot"</span>, <span class="st">"sample_number"</span>, <span class="st">"rep"</span>, <span class="st">"treatment"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb80-13"><a href="#cb80-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>Assay <span class="sc">==</span> <span class="st">"16S_Eub_338_518"</span>)</span>
<span id="cb80-14"><a href="#cb80-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb80-15"><a href="#cb80-15" aria-hidden="true" tabindex="-1"></a>soildfdf <span class="ot">&lt;-</span> soilnorm <span class="sc">%&gt;%</span></span>
<span id="cb80-16"><a href="#cb80-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(SoilSixS, <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"plot"</span>, <span class="st">"sample_number"</span>, <span class="st">"sample_day"</span>, <span class="st">"soil_type"</span>, <span class="st">"rep"</span>, <span class="st">"treatment"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb80-17"><a href="#cb80-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>Assay <span class="sc">==</span> <span class="st">"16S_Eub_338_518"</span>)</span>
<span id="cb80-18"><a href="#cb80-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb80-19"><a href="#cb80-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Now got to divide each count of each gene by the 16s number</span></span>
<span id="cb80-20"><a href="#cb80-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb80-21"><a href="#cb80-21" aria-hidden="true" tabindex="-1"></a>test <span class="ot">&lt;-</span> WaterCounts <span class="sc">%&gt;%</span> <span class="co"># Order ARGs based on class</span></span>
<span id="cb80-22"><a href="#cb80-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(gene_class) </span>
<span id="cb80-23"><a href="#cb80-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb80-24"><a href="#cb80-24" aria-hidden="true" tabindex="-1"></a>levels <span class="ot">&lt;-</span> <span class="fu">dput</span>(<span class="fu">levels</span>(<span class="fu">factor</span>(test<span class="sc">$</span>Assay))) <span class="co"># dput ftw!!!</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>c("16S_Eub_338_518", "aadA9", "erm(C)", "IS6100", "strB", "sul1 NEW", 
"sul2", "tetbP", "tetG_F", "tetH", "tetL", "tetM", "tetT", "tetW", 
"tetX")</code></pre>
</div>
<div class="sourceCode" id="cb82"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a>dfdf<span class="sc">$</span>treatment <span class="ot">=</span> <span class="fu">factor</span>(dfdf<span class="sc">$</span>treatment, </span>
<span id="cb82-2"><a href="#cb82-2" aria-hidden="true" tabindex="-1"></a>                             <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">'WCS'</span>, <span class="st">'WCSM'</span>, <span class="st">'WCM'</span>))</span>
<span id="cb82-3"><a href="#cb82-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-4"><a href="#cb82-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Also need to set the order of the soil data</span></span>
<span id="cb82-5"><a href="#cb82-5" aria-hidden="true" tabindex="-1"></a>soildfdf<span class="sc">$</span>treatment <span class="ot">=</span> <span class="fu">factor</span>(soildfdf<span class="sc">$</span>treatment, </span>
<span id="cb82-6"><a href="#cb82-6" aria-hidden="true" tabindex="-1"></a>                             <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">'WCS'</span>, <span class="st">'WCSM'</span>, <span class="st">'WCM'</span>))</span>
<span id="cb82-7"><a href="#cb82-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-8"><a href="#cb82-8" aria-hidden="true" tabindex="-1"></a>dfdf<span class="sc">$</span>sample_number <span class="ot">&lt;-</span> <span class="fu">as.factor</span>(dfdf<span class="sc">$</span>sample_number)</span>
<span id="cb82-9"><a href="#cb82-9" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(dfdf<span class="sc">$</span>sample_number) <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="st">"2.5"</span> <span class="ot">=</span> <span class="st">"1"</span>, <span class="st">"7.5"</span> <span class="ot">=</span> <span class="st">"2"</span>, <span class="st">"12.5"</span> <span class="ot">=</span> <span class="st">"3"</span>, <span class="st">"17.5"</span> <span class="ot">=</span> <span class="st">"4"</span>, <span class="st">"22.5"</span> <span class="ot">=</span> <span class="st">"5"</span>, <span class="st">"27.5"</span> <span class="ot">=</span> <span class="st">"6"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-64_37cc65840da0f58f995b03aa3722e65e">
<div class="sourceCode" id="cb83"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb83-1"><a href="#cb83-1" aria-hidden="true" tabindex="-1"></a>waterdata <span class="ot">&lt;-</span> dfdf <span class="sc">%&gt;%</span></span>
<span id="cb83-2"><a href="#cb83-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">norm =</span> counts<span class="sc">/</span>SixSCount)</span>
<span id="cb83-3"><a href="#cb83-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb83-4"><a href="#cb83-4" aria-hidden="true" tabindex="-1"></a>soildata <span class="ot">&lt;-</span> soildfdf <span class="sc">%&gt;%</span></span>
<span id="cb83-5"><a href="#cb83-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">norm =</span> counts<span class="sc">/</span>SixSCount)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Unfiltered counts in water</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-66_19ba12c441763a7ff09b7e901ef22424">
<div class="sourceCode" id="cb84"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb84-1"><a href="#cb84-1" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(waterdata<span class="sc">$</span>treatment) <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="st">"CS"</span> <span class="ot">=</span> <span class="st">"WCS"</span>, <span class="st">"CSL"</span> <span class="ot">=</span> <span class="st">"WCSM"</span>, <span class="st">"CL"</span> <span class="ot">=</span> <span class="st">"WCM"</span>)</span>
<span id="cb84-2"><a href="#cb84-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-3"><a href="#cb84-3" aria-hidden="true" tabindex="-1"></a>heatmap2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(waterdata, <span class="fu">aes</span>(<span class="fu">as.factor</span>(sample_number), <span class="fu">ordered</span>(Assay, <span class="at">levels =</span> levels))) <span class="sc">+</span></span>
<span id="cb84-4"><a href="#cb84-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="fu">aes</span>(<span class="at">fill =</span> norm), <span class="at">colour =</span> <span class="st">"white"</span>, <span class="at">size =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb84-5"><a href="#cb84-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_c</span>(<span class="at">name =</span> <span class="st">"ARG copies </span><span class="sc">\n</span><span class="st">normalized by 16S"</span>)  <span class="sc">+</span></span>
<span id="cb84-6"><a href="#cb84-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample time (min) after runoff"</span>, <span class="at">y =</span> <span class="st">"ARG"</span>) <span class="sc">+</span></span>
<span id="cb84-7"><a href="#cb84-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(gene_class <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span> <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb84-8"><a href="#cb84-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb84-9"><a href="#cb84-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb84-10"><a href="#cb84-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb84-11"><a href="#cb84-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb84-12"><a href="#cb84-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb84-13"><a href="#cb84-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb84-14"><a href="#cb84-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb84-15"><a href="#cb84-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.key.width =</span> <span class="fu">unit</span>(.<span class="dv">75</span>, <span class="st">"in"</span>)</span>
<span id="cb84-16"><a href="#cb84-16" aria-hidden="true" tabindex="-1"></a>  ) </span>
<span id="cb84-17"><a href="#cb84-17" aria-hidden="true" tabindex="-1"></a>heatmap2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-66-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-68_cdaa740d136764f3f691ce08ce1ae196">
<div class="sourceCode" id="cb85"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb85-1"><a href="#cb85-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(waterdata, <span class="fu">aes</span>(<span class="fu">as.factor</span>(sample_number), <span class="fu">ordered</span>(Assay, <span class="at">levels =</span> levels))) <span class="sc">+</span></span>
<span id="cb85-2"><a href="#cb85-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="fu">aes</span>(<span class="at">fill =</span> counts), <span class="at">colour =</span> <span class="st">"white"</span>, <span class="at">size =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb85-3"><a href="#cb85-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_c</span>(<span class="at">name =</span> <span class="st">"ARG copies"</span>)  <span class="sc">+</span></span>
<span id="cb85-4"><a href="#cb85-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample time (min) after runoff"</span>, <span class="at">y =</span> <span class="st">"ARG"</span>) <span class="sc">+</span></span>
<span id="cb85-5"><a href="#cb85-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(gene_class <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span> <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb85-6"><a href="#cb85-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb85-7"><a href="#cb85-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb85-8"><a href="#cb85-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb85-9"><a href="#cb85-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb85-10"><a href="#cb85-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb85-11"><a href="#cb85-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb85-12"><a href="#cb85-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb85-13"><a href="#cb85-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.key.width =</span> <span class="fu">unit</span>(.<span class="dv">75</span>, <span class="st">"in"</span>)</span>
<span id="cb85-14"><a href="#cb85-14" aria-hidden="true" tabindex="-1"></a>  ) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-68-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Filtered counts in water</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-70_b8417dccf51e1158d6bdcd97c5b6574b">
<div class="sourceCode" id="cb86"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb86-1"><a href="#cb86-1" aria-hidden="true" tabindex="-1"></a>waterwithlod <span class="ot">&lt;-</span> waterdata <span class="sc">%&gt;%</span></span>
<span id="cb86-2"><a href="#cb86-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(lod_max_by_gene, <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"Assay"</span> <span class="ot">=</span> <span class="st">"Sample_Name"</span>)) </span>
<span id="cb86-3"><a href="#cb86-3" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb86-4"><a href="#cb86-4" aria-hidden="true" tabindex="-1"></a>waterwithlod<span class="sc">$</span>norm[waterwithlod<span class="sc">$</span>Value <span class="sc">&gt;=</span> waterwithlod<span class="sc">$</span>max_lod] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb86-5"><a href="#cb86-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb86-6"><a href="#cb86-6" aria-hidden="true" tabindex="-1"></a>waterwithlod<span class="sc">$</span>counts[waterwithlod<span class="sc">$</span>Value <span class="sc">&gt;=</span> waterwithlod<span class="sc">$</span>max_lod] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb86-7"><a href="#cb86-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb86-8"><a href="#cb86-8" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(waterwithlod<span class="sc">$</span>treatment) <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="st">"CS"</span> <span class="ot">=</span> <span class="st">"WCS"</span>, <span class="st">"CSL"</span> <span class="ot">=</span> <span class="st">"WCSM"</span>, <span class="st">"CL"</span> <span class="ot">=</span> <span class="st">"WCM"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="figure-4" class="level1">
<h1>Figure 4</h1>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-72_035df06b453fe24ac361266155b042fd">
<div class="sourceCode" id="cb87"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb87-1"><a href="#cb87-1" aria-hidden="true" tabindex="-1"></a><span class="co">#waterwithlod &lt;- waterwithlod %&gt;% mutate(Assay = str_replace(Assay, "tetG_F", "tetG"))</span></span>
<span id="cb87-2"><a href="#cb87-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb87-3"><a href="#cb87-3" aria-hidden="true" tabindex="-1"></a><span class="co">#waterwithlod %&gt;% </span></span>
<span id="cb87-4"><a href="#cb87-4" aria-hidden="true" tabindex="-1"></a><span class="co">#  mutate(Assay = str_replace(Assay, "tetG_F", "tetG")) %&gt;%</span></span>
<span id="cb87-5"><a href="#cb87-5" aria-hidden="true" tabindex="-1"></a><span class="co">#  count(Assay)</span></span>
<span id="cb87-6"><a href="#cb87-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb87-7"><a href="#cb87-7" aria-hidden="true" tabindex="-1"></a>heatmap <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(waterwithlod, <span class="fu">aes</span>(<span class="fu">as.factor</span>(sample_number), <span class="fu">ordered</span>(Assay, <span class="at">levels =</span> levels))) <span class="sc">+</span></span>
<span id="cb87-8"><a href="#cb87-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="fu">aes</span>(<span class="at">fill =</span> norm), <span class="at">colour =</span> <span class="st">"white"</span>, <span class="at">size =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb87-9"><a href="#cb87-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_c</span>(<span class="at">name =</span> <span class="st">"ARG copies </span><span class="sc">\n</span><span class="st">normalized by 16S"</span>)  <span class="sc">+</span></span>
<span id="cb87-10"><a href="#cb87-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample time (min) after runoff"</span>, <span class="at">y =</span> <span class="st">"ARG"</span>) <span class="sc">+</span></span>
<span id="cb87-11"><a href="#cb87-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(gene_class <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span> <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb87-12"><a href="#cb87-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb87-13"><a href="#cb87-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb87-14"><a href="#cb87-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb87-15"><a href="#cb87-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb87-16"><a href="#cb87-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb87-17"><a href="#cb87-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb87-18"><a href="#cb87-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb87-19"><a href="#cb87-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.key.width =</span> <span class="fu">unit</span>(.<span class="dv">75</span>, <span class="st">"in"</span>)) </span>
<span id="cb87-20"><a href="#cb87-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb87-21"><a href="#cb87-21" aria-hidden="true" tabindex="-1"></a>heatmap</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-72-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb88"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb88-1"><a href="#cb88-1" aria-hidden="true" tabindex="-1"></a>heatmap<span class="sc">$</span>data <span class="sc">%&gt;%</span></span>
<span id="cb88-2"><a href="#cb88-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(Assay)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 14 x 2
   Assay        n
   &lt;chr&gt;    &lt;int&gt;
 1 aadA9      198
 2 erm(C)     198
 3 IS6100     198
 4 strB       198
 5 sul1 NEW   198
 6 sul2       198
 7 tetbP      198
 8 tetG_F     198
 9 tetH       198
10 tetL       198
11 tetM       198
12 tetT       198
13 tetW       198
14 tetX       198</code></pre>
</div>
</div>
<ul>
<li>Astrik on manure ARGs ::: {.cell hash=“ManuscriptWalkthrough_cache/html/unnamed-chunk-74_304361dff4835c2df529ca1a354f2540”}</li>
</ul>
<div class="sourceCode" id="cb90"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb90-1"><a href="#cb90-1" aria-hidden="true" tabindex="-1"></a>labels <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">tetbP =</span> <span class="st">"* tetbP "</span>, </span>
<span id="cb90-2"><a href="#cb90-2" aria-hidden="true" tabindex="-1"></a>            <span class="at">aphA3 =</span> <span class="st">"* aphA3 "</span>, </span>
<span id="cb90-3"><a href="#cb90-3" aria-hidden="true" tabindex="-1"></a>            <span class="at">tetT =</span> <span class="st">"* tetT "</span>, </span>
<span id="cb90-4"><a href="#cb90-4" aria-hidden="true" tabindex="-1"></a>            <span class="at">tetW =</span> <span class="st">"* tetW "</span>, </span>
<span id="cb90-5"><a href="#cb90-5" aria-hidden="true" tabindex="-1"></a>            <span class="st">`</span><span class="at">erm(C)</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"* erm(C)"</span>, </span>
<span id="cb90-6"><a href="#cb90-6" aria-hidden="true" tabindex="-1"></a>            <span class="at">tetX =</span> <span class="st">"* tetX "</span>, </span>
<span id="cb90-7"><a href="#cb90-7" aria-hidden="true" tabindex="-1"></a>            <span class="at">tnpA1 =</span> <span class="st">"* tnpA1 "</span>,</span>
<span id="cb90-8"><a href="#cb90-8" aria-hidden="true" tabindex="-1"></a>            <span class="at">tetH =</span> <span class="st">"* tetH "</span>, </span>
<span id="cb90-9"><a href="#cb90-9" aria-hidden="true" tabindex="-1"></a>            <span class="at">lnuA =</span> <span class="st">"* lnuA "</span>, </span>
<span id="cb90-10"><a href="#cb90-10" aria-hidden="true" tabindex="-1"></a>            <span class="at">tnpA5 =</span> <span class="st">"* tnpA5 "</span>, </span>
<span id="cb90-11"><a href="#cb90-11" aria-hidden="true" tabindex="-1"></a>            <span class="at">tetM =</span> <span class="st">"* tetM "</span>, </span>
<span id="cb90-12"><a href="#cb90-12" aria-hidden="true" tabindex="-1"></a>            <span class="at">cmr =</span> <span class="st">"* cmr "</span>,</span>
<span id="cb90-13"><a href="#cb90-13" aria-hidden="true" tabindex="-1"></a>            <span class="at">tetG_F =</span> <span class="st">"tetG "</span>,</span>
<span id="cb90-14"><a href="#cb90-14" aria-hidden="true" tabindex="-1"></a>            <span class="st">`</span><span class="at">sul1 NEW</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"sul1"</span>)</span>
<span id="cb90-15"><a href="#cb90-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb90-16"><a href="#cb90-16" aria-hidden="true" tabindex="-1"></a>labels</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>     tetbP      aphA3       tetT       tetW     erm(C)       tetX      tnpA1 
"* tetbP " "* aphA3 "  "* tetT "  "* tetW " "* erm(C)"  "* tetX " "* tnpA1 " 
      tetH       lnuA      tnpA5       tetM        cmr     tetG_F   sul1 NEW 
 "* tetH "  "* lnuA " "* tnpA5 "  "* tetM "   "* cmr "    "tetG "     "sul1" </code></pre>
</div>
<p>::: ::: {.cell hash=“ManuscriptWalkthrough_cache/html/unnamed-chunk-76_c0215bc00ba722fc1308db7e652b55d6”}</p>
<div class="sourceCode" id="cb92"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb92-1"><a href="#cb92-1" aria-hidden="true" tabindex="-1"></a>heatmap <span class="sc">+</span></span>
<span id="cb92-2"><a href="#cb92-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_discrete</span>(</span>
<span id="cb92-3"><a href="#cb92-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">name =</span> <span class="cn">NULL</span>,</span>
<span id="cb92-4"><a href="#cb92-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">labels =</span> labels</span>
<span id="cb92-5"><a href="#cb92-5" aria-hidden="true" tabindex="-1"></a>  ) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-76-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb93"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb93-1"><a href="#cb93-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/Fig4.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">width =</span> <span class="dv">7</span>, <span class="at">height =</span> <span class="fl">7.5</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<p>:::</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-78_5788829ff73ed45d2d457243321ff5f6">
<div class="sourceCode" id="cb94"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb94-1"><a href="#cb94-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(waterwithlod, <span class="fu">aes</span>(<span class="fu">as.factor</span>(sample_number), <span class="fu">ordered</span>(Assay, <span class="at">levels =</span> levels))) <span class="sc">+</span></span>
<span id="cb94-2"><a href="#cb94-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="fu">aes</span>(<span class="at">fill =</span> counts), <span class="at">colour =</span> <span class="st">"white"</span>, <span class="at">size =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb94-3"><a href="#cb94-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_c</span>(<span class="at">name =</span> <span class="st">"ARG copies"</span>)  <span class="sc">+</span></span>
<span id="cb94-4"><a href="#cb94-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample time (min) after runoff"</span>, <span class="at">y =</span> <span class="st">"ARG"</span>) <span class="sc">+</span></span>
<span id="cb94-5"><a href="#cb94-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(gene_class <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span> <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb94-6"><a href="#cb94-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb94-7"><a href="#cb94-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb94-8"><a href="#cb94-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb94-9"><a href="#cb94-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb94-10"><a href="#cb94-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb94-11"><a href="#cb94-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb94-12"><a href="#cb94-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb94-13"><a href="#cb94-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.key.width =</span> <span class="fu">unit</span>(.<span class="dv">75</span>, <span class="st">"in"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-78-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>A few genes were detected in low enough quantities that they are no longer visible in the filtered data set. Overall, the results do not change much, but there are changes in pairwise significance for some genes that I need to evaluate further.</p>
</section>
<section id="supp-figure-s3" class="level1">
<h1>Supp Figure S3</h1>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-80_bb6d15e55c7e16cbd8c73f9b4de8bb66">
<div class="sourceCode" id="cb95"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb95-1"><a href="#cb95-1" aria-hidden="true" tabindex="-1"></a>soilwithlod <span class="ot">&lt;-</span> soildata <span class="sc">%&gt;%</span></span>
<span id="cb95-2"><a href="#cb95-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(lod_max_by_gene, <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"Assay"</span> <span class="ot">=</span> <span class="st">"Sample_Name"</span>)) </span>
<span id="cb95-3"><a href="#cb95-3" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb95-4"><a href="#cb95-4" aria-hidden="true" tabindex="-1"></a>soilwithlod<span class="sc">$</span>norm[soilwithlod<span class="sc">$</span>Value <span class="sc">&gt;=</span> soilwithlod<span class="sc">$</span>max_lod] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb95-5"><a href="#cb95-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb95-6"><a href="#cb95-6" aria-hidden="true" tabindex="-1"></a>soilwithlod<span class="sc">$</span>counts[soilwithlod<span class="sc">$</span>Value <span class="sc">&gt;=</span> soilwithlod<span class="sc">$</span>max_lod] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb95-7"><a href="#cb95-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb95-8"><a href="#cb95-8" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(soilwithlod<span class="sc">$</span>treatment) <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="st">"CS"</span> <span class="ot">=</span> <span class="st">"WCS"</span>, <span class="st">"CSL"</span> <span class="ot">=</span> <span class="st">"WCSM"</span>, <span class="st">"CL"</span> <span class="ot">=</span> <span class="st">"WCM"</span>)</span>
<span id="cb95-9"><a href="#cb95-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb95-10"><a href="#cb95-10" aria-hidden="true" tabindex="-1"></a>heatmap2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(soilwithlod, <span class="fu">aes</span>(<span class="fu">as.factor</span>(sample_day), <span class="fu">ordered</span>(Assay, <span class="at">levels =</span> levels))) <span class="sc">+</span></span>
<span id="cb95-11"><a href="#cb95-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="fu">aes</span>(<span class="at">fill =</span> norm), <span class="at">colour =</span> <span class="st">"white"</span>, <span class="at">size =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb95-12"><a href="#cb95-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_c</span>(<span class="at">name =</span> <span class="st">"ARG copies </span><span class="sc">\n</span><span class="st">normalized by 16S"</span>)  <span class="sc">+</span></span>
<span id="cb95-13"><a href="#cb95-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sample day"</span>, <span class="at">y =</span> <span class="st">"ARG"</span>) <span class="sc">+</span></span>
<span id="cb95-14"><a href="#cb95-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_nested</span>(gene_class <span class="sc">~</span> treatment <span class="sc">+</span> plot, <span class="at">scales =</span> <span class="st">"free"</span>, <span class="at">space =</span> <span class="st">"free"</span>) <span class="sc">+</span> <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb95-15"><a href="#cb95-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb95-16"><a href="#cb95-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">family =</span> <span class="st">"Times"</span>),</span>
<span id="cb95-17"><a href="#cb95-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb95-18"><a href="#cb95-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb95-19"><a href="#cb95-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb95-20"><a href="#cb95-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>, <span class="at">hjust=</span><span class="dv">1</span>),</span>
<span id="cb95-21"><a href="#cb95-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb95-22"><a href="#cb95-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.key.width =</span> <span class="fu">unit</span>(.<span class="dv">75</span>, <span class="st">"in"</span>)</span>
<span id="cb95-23"><a href="#cb95-23" aria-hidden="true" tabindex="-1"></a>  ) </span>
<span id="cb95-24"><a href="#cb95-24" aria-hidden="true" tabindex="-1"></a>heatmap2 <span class="sc">+</span></span>
<span id="cb95-25"><a href="#cb95-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_discrete</span>(</span>
<span id="cb95-26"><a href="#cb95-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">name =</span> <span class="cn">NULL</span>,</span>
<span id="cb95-27"><a href="#cb95-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">labels =</span> labels</span>
<span id="cb95-28"><a href="#cb95-28" aria-hidden="true" tabindex="-1"></a>  ) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-80-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb96"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb96-1"><a href="#cb96-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/SupFigS3.jpeg"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"jpeg"</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">width =</span> <span class="dv">7</span>, <span class="at">height =</span> <span class="fl">7.5</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="supp-figure-s4" class="level1">
<h1>Supp Figure S4</h1>
<p>This plot shows the flow rate for each sample from each plot.</p>
<p>If we plot MAB vs.&nbsp;flow, do prairie strips have an impact?</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-82_f5377b4a511e14cbdef70ee22e932b63">
<div class="sourceCode" id="cb97"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb97-1"><a href="#cb97-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readxl)</span>
<span id="cb97-2"><a href="#cb97-2" aria-hidden="true" tabindex="-1"></a>mabbarplot<span class="sc">$</span>data</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>     Sample                 OTU              Phylum   Abundance  p time
  1:   P1-1      Actinobacteria      Actinobacteria 0.000000000 P1    1
  2:   P1-1       Bacteroidetes       Bacteroidetes 0.000000000 P1    1
  3:   P1-1         Chloroflexi         Chloroflexi 0.000000000 P1    1
  4:   P1-1       Cloacimonetes       Cloacimonetes 0.000000000 P1    1
  5:   P1-1 Deinococcus-Thermus Deinococcus-Thermus 0.000000000 P1    1
 ---                                                                   
659:   P9-6      Proteobacteria      Proteobacteria 0.007088626 P9    6
660:   P9-6         Chloroflexi         Chloroflexi 0.007844266 P9    6
661:   P9-6      Planctomycetes      Planctomycetes 0.008527941 P9    6
662:   P9-6      Actinobacteria      Actinobacteria 0.071893779 P9    6
663:   P9-6          Firmicutes          Firmicutes 0.205822029 P9    6
         experiment matrix treatment plot sample_day depth in_plot_location
  1: Worle_Rainfall  water        CS    1       T042  &lt;NA&gt;             &lt;NA&gt;
  2: Worle_Rainfall  water        CS    1       T042  &lt;NA&gt;             &lt;NA&gt;
  3: Worle_Rainfall  water        CS    1       T042  &lt;NA&gt;             &lt;NA&gt;
  4: Worle_Rainfall  water        CS    1       T042  &lt;NA&gt;             &lt;NA&gt;
  5: Worle_Rainfall  water        CS    1       T042  &lt;NA&gt;             &lt;NA&gt;
 ---                                                                       
659: Worle_Rainfall  water        CL    9       T042  &lt;NA&gt;             &lt;NA&gt;
660: Worle_Rainfall  water        CL    9       T042  &lt;NA&gt;             &lt;NA&gt;
661: Worle_Rainfall  water        CL    9       T042  &lt;NA&gt;             &lt;NA&gt;
662: Worle_Rainfall  water        CL    9       T042  &lt;NA&gt;             &lt;NA&gt;
663: Worle_Rainfall  water        CL    9       T042  &lt;NA&gt;             &lt;NA&gt;
     block strip manure_treatment soil_type
  1:     1     Y                N    border
  2:     1     Y                N    border
  3:     1     Y                N    border
  4:     1     Y                N    border
  5:     1     Y                N    border
 ---                                       
659:     3     N                Y    border
660:     3     N                Y    border
661:     3     N                Y    border
662:     3     N                Y    border
663:     3     N                Y    border</code></pre>
</div>
<div class="sourceCode" id="cb99"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb99-1"><a href="#cb99-1" aria-hidden="true" tabindex="-1"></a>flow <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(<span class="st">"Data/WOR_Water_flow.xlsx"</span>)</span>
<span id="cb99-2"><a href="#cb99-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb99-3"><a href="#cb99-3" aria-hidden="true" tabindex="-1"></a><span class="fu">glimpse</span>(flow)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>Rows: 54
Columns: 8
$ plot                    &lt;dbl&gt; 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3…
$ treatment               &lt;chr&gt; "CS", "CS", "CS", "CS", "CS", "CS", "CS", "CS"…
$ sample_number           &lt;dbl&gt; 1, 2, 3, 4, 5, 6, 1, 2, 3, 4, 5, 6, 1, 2, 3, 4…
$ time_after_runoff_start &lt;dbl&gt; 2.5, 7.5, 12.5, 17.5, 22.5, 27.5, 2.5, 7.5, 12…
$ collection_time_sec     &lt;dbl&gt; 210, 210, 150, 120, 75, 75, 180, 150, 135, 120…
$ jare_tare_wt_g          &lt;dbl&gt; 88, 88, 88, 88, 88, 88, 87, 88, 88, 88, 88, 88…
$ jare_plus_runoff_wt_g   &lt;dbl&gt; 919, 906, 996, 1067, 1027, 993, 731, 772, 859,…
$ `flow_g/s`              &lt;dbl&gt; 3.957143, 3.895238, 6.053333, 8.158333, 12.520…</code></pre>
</div>
<div class="sourceCode" id="cb101"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb101-1"><a href="#cb101-1" aria-hidden="true" tabindex="-1"></a>flow<span class="sc">$</span>sample_number <span class="ot">&lt;-</span> <span class="fu">as.character</span>(flow<span class="sc">$</span>sample_number)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-84_997505ff7b272642e2c549ec366ed2b4">
<div class="sourceCode" id="cb102"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb102-1"><a href="#cb102-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(flow, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.factor</span>(sample_number), <span class="at">y =</span> <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>, <span class="at">group =</span> <span class="fu">as.factor</span>(plot))) <span class="sc">+</span></span>
<span id="cb102-2"><a href="#cb102-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> .<span class="dv">3</span>)) <span class="sc">+</span></span>
<span id="cb102-3"><a href="#cb102-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">color =</span> <span class="fu">as.factor</span>(treatment)), <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> .<span class="dv">3</span>), <span class="at">size =</span> <span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb102-4"><a href="#cb102-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> plot), <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> .<span class="dv">3</span>), <span class="at">show.legend =</span> F) <span class="sc">+</span></span>
<span id="cb102-5"><a href="#cb102-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb102-6"><a href="#cb102-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb102-7"><a href="#cb102-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>, </span>
<span id="cb102-8"><a href="#cb102-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb102-9"><a href="#cb102-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb102-10"><a href="#cb102-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb102-11"><a href="#cb102-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">vjust =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb102-12"><a href="#cb102-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_viridis_d</span>() <span class="sc">+</span></span>
<span id="cb102-13"><a href="#cb102-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Time after runoff start (min)"</span>, <span class="at">y =</span> <span class="st">"Flow (mL/s)"</span>, <span class="at">color =</span> <span class="st">"Treatment </span><span class="sc">\n</span><span class="st">(points labeled with plot)"</span>) <span class="sc">+</span></span>
<span id="cb102-14"><a href="#cb102-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_x_discrete</span>(<span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"2.5"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"7.5"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"12.5"</span>, <span class="st">"4"</span> <span class="ot">=</span> <span class="st">"17.5"</span>, <span class="st">"5"</span> <span class="ot">=</span> <span class="st">"22.5"</span>, <span class="st">"6"</span> <span class="ot">=</span> <span class="st">"27.5"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-84-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb103"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb103-1"><a href="#cb103-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/SupFig4.png"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"png"</span>, <span class="at">width =</span> <span class="fl">6.5</span>, <span class="at">height =</span> <span class="fl">7.5</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="supp-figure-s5" class="level1">
<h1>Supp Figure S5</h1>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-86_f4fb8b4c9f56317e8d8f1c498371c784">
<div class="sourceCode" id="cb104"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb104-1"><a href="#cb104-1" aria-hidden="true" tabindex="-1"></a>SixS <span class="ot">&lt;-</span> norm <span class="sc">%&gt;%</span></span>
<span id="cb104-2"><a href="#cb104-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Assay <span class="sc">==</span> <span class="st">"16S_Eub_338_518"</span>) </span>
<span id="cb104-3"><a href="#cb104-3" aria-hidden="true" tabindex="-1"></a>SixS</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 198 x 8
   plot  sample_number   rep treatment Assay       gene_class counts Value      
   &lt;chr&gt; &lt;chr&gt;         &lt;int&gt; &lt;chr&gt;     &lt;chr&gt;       &lt;chr&gt;       &lt;dbl&gt; &lt;chr&gt;      
 1 2     5                 1 WCS       16S_Eub_33… 16S          844. 18.3926544…
 2 2     5                 2 WCS       16S_Eub_33… 16S          858. 18.3668106…
 3 2     5                 3 WCS       16S_Eub_33… 16S         1308. 17.7039409…
 4 2     3                 1 WCS       16S_Eub_33… 16S          715. 18.6550587…
 5 2     3                 2 WCS       16S_Eub_33… 16S          733. 18.6149361…
 6 2     3                 3 WCS       16S_Eub_33… 16S         1173. 17.8757951…
 7 2     1                 1 WCS       16S_Eub_33… 16S         3359. 16.2208645…
 8 2     1                 2 WCS       16S_Eub_33… 16S         3564. 16.1278807…
 9 2     1                 3 WCS       16S_Eub_33… 16S         4399. 15.7965356…
10 1     5                 1 WCS       16S_Eub_33… 16S           NA  &lt;NA&gt;       
# … with 188 more rows</code></pre>
</div>
<div class="sourceCode" id="cb106"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb106-1"><a href="#cb106-1" aria-hidden="true" tabindex="-1"></a>SixS<span class="sc">$</span>treatment <span class="ot">=</span> <span class="fu">factor</span>(SixS<span class="sc">$</span>treatment, </span>
<span id="cb106-2"><a href="#cb106-2" aria-hidden="true" tabindex="-1"></a>                             <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">'WCS'</span>, <span class="st">'WCSM'</span>, <span class="st">'WCM'</span>))</span>
<span id="cb106-3"><a href="#cb106-3" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(SixS<span class="sc">$</span>treatment) <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="st">"CS"</span><span class="ot">=</span><span class="st">"WCS"</span>, <span class="st">"CSM"</span><span class="ot">=</span><span class="st">"WCSM"</span>, <span class="st">"CM"</span><span class="ot">=</span><span class="st">"WCM"</span>)</span>
<span id="cb106-4"><a href="#cb106-4" aria-hidden="true" tabindex="-1"></a>SixS<span class="sc">$</span>sample_number <span class="ot">&lt;-</span> <span class="fu">as.factor</span>(SixS<span class="sc">$</span>sample_number)</span>
<span id="cb106-5"><a href="#cb106-5" aria-hidden="true" tabindex="-1"></a><span class="fu">levels</span>(SixS<span class="sc">$</span>sample_number) <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="st">"2.5"</span> <span class="ot">=</span> <span class="st">"1"</span>, <span class="st">"7.5"</span> <span class="ot">=</span> <span class="st">"2"</span>, <span class="st">"12.5"</span> <span class="ot">=</span> <span class="st">"3"</span>, <span class="st">"17.5"</span> <span class="ot">=</span> <span class="st">"4"</span>, <span class="st">"22.5"</span> <span class="ot">=</span> <span class="st">"5"</span>, <span class="st">"27.5"</span> <span class="ot">=</span> <span class="st">"6"</span>)</span>
<span id="cb106-6"><a href="#cb106-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb106-7"><a href="#cb106-7" aria-hidden="true" tabindex="-1"></a>my_comparisons <span class="ot">&lt;-</span> <span class="fu">list</span>( <span class="fu">c</span>(<span class="st">"WCS"</span>, <span class="st">"WCSM"</span>), <span class="fu">c</span>(<span class="st">"WCS"</span>, <span class="st">"WCM"</span>), <span class="fu">c</span>(<span class="st">"WCM"</span>, <span class="st">"WCSM"</span>))</span>
<span id="cb106-8"><a href="#cb106-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb106-9"><a href="#cb106-9" aria-hidden="true" tabindex="-1"></a>sig <span class="ot">&lt;-</span> SixS <span class="sc">%&gt;%</span></span>
<span id="cb106-10"><a href="#cb106-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>rep <span class="sc">%in%</span> <span class="fu">c</span>(<span class="dv">4</span>,<span class="dv">5</span>,<span class="dv">6</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb106-11"><a href="#cb106-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">droplevels</span>()</span>
<span id="cb106-12"><a href="#cb106-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb106-13"><a href="#cb106-13" aria-hidden="true" tabindex="-1"></a>sig</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 162 x 8
   plot  sample_number   rep treatment Assay       gene_class counts Value      
   &lt;chr&gt; &lt;fct&gt;         &lt;int&gt; &lt;fct&gt;     &lt;chr&gt;       &lt;chr&gt;       &lt;dbl&gt; &lt;chr&gt;      
 1 2     22.5              1 CS        16S_Eub_33… 16S          844. 18.3926544…
 2 2     22.5              2 CS        16S_Eub_33… 16S          858. 18.3668106…
 3 2     22.5              3 CS        16S_Eub_33… 16S         1308. 17.7039409…
 4 2     12.5              1 CS        16S_Eub_33… 16S          715. 18.6550587…
 5 2     12.5              2 CS        16S_Eub_33… 16S          733. 18.6149361…
 6 2     12.5              3 CS        16S_Eub_33… 16S         1173. 17.8757951…
 7 2     2.5               1 CS        16S_Eub_33… 16S         3359. 16.2208645…
 8 2     2.5               2 CS        16S_Eub_33… 16S         3564. 16.1278807…
 9 2     2.5               3 CS        16S_Eub_33… 16S         4399. 15.7965356…
10 1     22.5              1 CS        16S_Eub_33… 16S           NA  &lt;NA&gt;       
# … with 152 more rows</code></pre>
</div>
<div class="sourceCode" id="cb108"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb108-1"><a href="#cb108-1" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(sig, <span class="fu">aes</span>(<span class="at">x=</span>treatment, <span class="at">y=</span>counts, <span class="at">fill =</span> treatment)) <span class="sc">+</span> </span>
<span id="cb108-2"><a href="#cb108-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb108-3"><a href="#cb108-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Treatment"</span>, <span class="at">y =</span> <span class="st">"Log 10 gene 16s copies"</span>) <span class="sc">+</span></span>
<span id="cb108-4"><a href="#cb108-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">trans =</span> <span class="st">'log10'</span>) <span class="sc">+</span></span>
<span id="cb108-5"><a href="#cb108-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>() <span class="sc">+</span></span>
<span id="cb108-6"><a href="#cb108-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>( <span class="sc">~</span> sample_number) <span class="sc">+</span></span>
<span id="cb108-7"><a href="#cb108-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_pubr</span>() <span class="sc">+</span></span>
<span id="cb108-8"><a href="#cb108-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb108-9"><a href="#cb108-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.background =</span> <span class="fu">element_rect</span>(</span>
<span id="cb108-10"><a href="#cb108-10" aria-hidden="true" tabindex="-1"></a>      <span class="at">fill =</span> (<span class="at">alpha =</span> <span class="dv">0</span>),</span>
<span id="cb108-11"><a href="#cb108-11" aria-hidden="true" tabindex="-1"></a>      <span class="at">color =</span> <span class="st">'black'</span>,</span>
<span id="cb108-12"><a href="#cb108-12" aria-hidden="true" tabindex="-1"></a>      <span class="at">size =</span> <span class="fl">0.1</span>),</span>
<span id="cb108-13"><a href="#cb108-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb108-14"><a href="#cb108-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">face =</span> <span class="st">'bold'</span>),</span>
<span id="cb108-15"><a href="#cb108-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">colour =</span> <span class="st">'black'</span>, <span class="at">size =</span> <span class="fl">1.4</span>),</span>
<span id="cb108-16"><a href="#cb108-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span>) </span>
<span id="cb108-17"><a href="#cb108-17" aria-hidden="true" tabindex="-1"></a>p  </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-86-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-88_f5dc5a0a3abe5bcef1eb8ee81a2d9077">
<div class="sourceCode" id="cb109"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb109-1"><a href="#cb109-1" aria-hidden="true" tabindex="-1"></a>krusk.test <span class="ot">&lt;-</span> sig <span class="sc">%&gt;%</span></span>
<span id="cb109-2"><a href="#cb109-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(sample_number) <span class="sc">%&gt;%</span></span>
<span id="cb109-3"><a href="#cb109-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kruskal_test</span>(counts <span class="sc">~</span> treatment)</span>
<span id="cb109-4"><a href="#cb109-4" aria-hidden="true" tabindex="-1"></a>krusk.test</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 6 x 7
  sample_number .y.        n statistic    df        p method        
* &lt;fct&gt;         &lt;chr&gt;  &lt;int&gt;     &lt;dbl&gt; &lt;int&gt;    &lt;dbl&gt; &lt;chr&gt;         
1 2.5           counts    27      9.00     2 0.0111   Kruskal-Wallis
2 7.5           counts    27      9.43     2 0.00895  Kruskal-Wallis
3 12.5          counts    27     17.0      2 0.000206 Kruskal-Wallis
4 17.5          counts    27      5.66     2 0.0591   Kruskal-Wallis
5 22.5          counts    27      3.41     2 0.182    Kruskal-Wallis
6 27.5          counts    27      4.02     2 0.134    Kruskal-Wallis</code></pre>
</div>
<div class="sourceCode" id="cb111"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb111-1"><a href="#cb111-1" aria-hidden="true" tabindex="-1"></a>stat.test <span class="ot">&lt;-</span> sig <span class="sc">%&gt;%</span></span>
<span id="cb111-2"><a href="#cb111-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(sample_number) <span class="sc">%&gt;%</span></span>
<span id="cb111-3"><a href="#cb111-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">dunn_test</span>(counts <span class="sc">~</span> treatment) <span class="sc">%&gt;%</span></span>
<span id="cb111-4"><a href="#cb111-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">adjust_pvalue</span>(<span class="at">method =</span> <span class="st">"hochberg"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb111-5"><a href="#cb111-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">y.position =</span> <span class="fu">rep</span>(<span class="fu">c</span>(<span class="fu">log10</span>(<span class="dv">100000</span>), <span class="fu">log10</span>(<span class="dv">300000</span>), <span class="fu">log10</span>(<span class="dv">1000000</span>)), <span class="dv">6</span>))</span>
<span id="cb111-6"><a href="#cb111-6" aria-hidden="true" tabindex="-1"></a>stat.test</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code># A tibble: 18 x 11
   sample_number .y.    group1 group2    n1    n2 statistic        p   p.adj
 * &lt;fct&gt;         &lt;chr&gt;  &lt;chr&gt;  &lt;chr&gt;  &lt;int&gt; &lt;int&gt;     &lt;dbl&gt;    &lt;dbl&gt;   &lt;dbl&gt;
 1 2.5           counts CS     CSM        9     9     1.46  0.146    0.878  
 2 2.5           counts CS     CM         9     9     3.00  0.00271  0.0433 
 3 2.5           counts CSM    CM         9     9     1.54  0.123    0.878  
 4 7.5           counts CS     CSM        8     9     2.52  0.0117   0.163  
 5 7.5           counts CS     CM         8     9     2.82  0.00478  0.0717 
 6 7.5           counts CSM    CM         9     9     0.308 0.758    0.878  
 7 12.5          counts CS     CSM        9     9     3.18  0.00149  0.0253 
 8 12.5          counts CS     CM         9     9     3.86  0.000113 0.00204
 9 12.5          counts CSM    CM         9     9     0.683 0.495    0.878  
10 17.5          counts CS     CSM        8     9     1.82  0.0693   0.693  
11 17.5          counts CS     CM         8     9     2.26  0.0235   0.306  
12 17.5          counts CSM    CM         9     9     0.462 0.644    0.878  
13 22.5          counts CS     CSM        6     9     1.27  0.205    0.878  
14 22.5          counts CS     CM         6     9     1.83  0.0667   0.693  
15 22.5          counts CSM    CM         9     9     0.633 0.527    0.878  
16 27.5          counts CS     CSM        8     9     1.67  0.0941   0.847  
17 27.5          counts CS     CM         8     9     1.82  0.0682   0.693  
18 27.5          counts CSM    CM         9     9     0.154 0.878    0.878  
# … with 2 more variables: p.adj.signif &lt;chr&gt;, y.position &lt;dbl&gt;</code></pre>
</div>
<div class="sourceCode" id="cb113"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb113-1"><a href="#cb113-1" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggboxplot</span>(sig, <span class="at">x =</span> <span class="st">"treatment"</span>, <span class="at">y =</span> <span class="st">"counts"</span>, <span class="at">facet.by =</span> <span class="st">"sample_number"</span>) <span class="sc">+</span></span>
<span id="cb113-2"><a href="#cb113-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">trans =</span> <span class="st">"log10"</span>)</span>
<span id="cb113-3"><a href="#cb113-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb113-4"><a href="#cb113-4" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">scipen =</span> <span class="dv">5</span>)</span>
<span id="cb113-5"><a href="#cb113-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb113-6"><a href="#cb113-6" aria-hidden="true" tabindex="-1"></a>p <span class="sc">+</span> </span>
<span id="cb113-7"><a href="#cb113-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_pvalue_manual</span>(stat.test, <span class="at">hide.ns =</span> T) <span class="sc">+</span></span>
<span id="cb113-8"><a href="#cb113-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_pubr</span>() <span class="sc">+</span></span>
<span id="cb113-9"><a href="#cb113-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Treatment"</span>, <span class="at">y =</span> <span class="st">"Log 10 gene 16s copies"</span>,</span>
<span id="cb113-10"><a href="#cb113-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">caption =</span> <span class="fu">get_pwc_label</span>(stat.test)) <span class="sc">+</span></span>
<span id="cb113-11"><a href="#cb113-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>(<span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"CSM"</span> <span class="ot">=</span> <span class="st">"CSL"</span>, <span class="st">"CM"</span> <span class="ot">=</span> <span class="st">"CL"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-88-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb114"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb114-1"><a href="#cb114-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/SuppFiggS5.png"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"png"</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">width =</span> <span class="fl">6.5</span>, <span class="at">height =</span> <span class="fl">7.5</span>, <span class="at">dpi =</span> <span class="dv">300</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="supp-figure-s6" class="level1">
<h1>Supp Figure S6</h1>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-90_d79610b2ad84709a53d25918791adb22">
<div class="sourceCode" id="cb115"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb115-1"><a href="#cb115-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> mabbarplot<span class="sc">$</span>data <span class="sc">%&gt;%</span></span>
<span id="cb115-2"><a href="#cb115-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(plot, time) <span class="sc">%&gt;%</span></span>
<span id="cb115-3"><a href="#cb115-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">SumAbund =</span> <span class="fu">sum</span>(Abundance)) <span class="sc">%&gt;%</span></span>
<span id="cb115-4"><a href="#cb115-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(<span class="at">sample_number =</span> time) <span class="sc">%&gt;%</span></span>
<span id="cb115-5"><a href="#cb115-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb115-6"><a href="#cb115-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(flow, <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"plot"</span>, <span class="st">"sample_number"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb115-7"><a href="#cb115-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(treatment.x, plot, sample_number, SumAbund, time_after_runoff_start, collection_time_sec, jare_tare_wt_g, jare_plus_runoff_wt_g, <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>)</span>
<span id="cb115-8"><a href="#cb115-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb115-9"><a href="#cb115-9" aria-hidden="true" tabindex="-1"></a><span class="fu">write_excel_csv</span>(df, <span class="at">file =</span> <span class="st">"Data/water_flow_with_MAB.csv"</span>)</span>
<span id="cb115-10"><a href="#cb115-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb115-11"><a href="#cb115-11" aria-hidden="true" tabindex="-1"></a>lmplot <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb115-12"><a href="#cb115-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>treatment.x <span class="sc">==</span> <span class="st">"CS"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb115-13"><a href="#cb115-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>, <span class="at">y =</span> SumAbund, <span class="at">color =</span> treatment.x)) <span class="sc">+</span></span>
<span id="cb115-14"><a href="#cb115-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb115-15"><a href="#cb115-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="at">method =</span> lm, <span class="at">se =</span> <span class="cn">FALSE</span>)  <span class="sc">+</span></span>
<span id="cb115-16"><a href="#cb115-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"darkgreen"</span>, <span class="st">"tan4"</span>), <span class="at">name =</span> <span class="st">"Treatment"</span>) <span class="sc">+</span></span>
<span id="cb115-17"><a href="#cb115-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>()</span>
<span id="cb115-18"><a href="#cb115-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb115-19"><a href="#cb115-19" aria-hidden="true" tabindex="-1"></a>lmplot</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`geom_smooth()` using formula 'y ~ x'</code></pre>
</div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-90-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-92_ad8c20e1dd51cf1769c17cd52e2fad95">
<div class="sourceCode" id="cb117"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb117-1"><a href="#cb117-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Two seperate lm's to calculated where they intersect</span></span>
<span id="cb117-2"><a href="#cb117-2" aria-hidden="true" tabindex="-1"></a>CM <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb117-3"><a href="#cb117-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>treatment.x <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"CS"</span>, <span class="st">"CSL"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb117-4"><a href="#cb117-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">lm</span>(SumAbund <span class="sc">~</span> <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>, <span class="at">data =</span> .)</span>
<span id="cb117-5"><a href="#cb117-5" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(CM)<span class="sc">$</span>r.squared</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 0.5325833</code></pre>
</div>
<div class="sourceCode" id="cb119"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb119-1"><a href="#cb119-1" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(CM)<span class="sc">$</span>adj.r.squared</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 0.5305686</code></pre>
</div>
<div class="sourceCode" id="cb121"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb121-1"><a href="#cb121-1" aria-hidden="true" tabindex="-1"></a>CSM <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb121-2"><a href="#cb121-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>treatment.x <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"CS"</span>, <span class="st">"CL"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb121-3"><a href="#cb121-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">lm</span>(SumAbund <span class="sc">~</span> <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>, <span class="at">data =</span> .)</span>
<span id="cb121-4"><a href="#cb121-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(CSM)<span class="sc">$</span>r.squared</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 0.5541166</code></pre>
</div>
<div class="sourceCode" id="cb123"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb123-1"><a href="#cb123-1" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(CSM)<span class="sc">$</span>adj.r.squared</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 0.5521947</code></pre>
</div>
<div class="sourceCode" id="cb125"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb125-1"><a href="#cb125-1" aria-hidden="true" tabindex="-1"></a><span class="co"># where do the two lines intersect? </span></span>
<span id="cb125-2"><a href="#cb125-2" aria-hidden="true" tabindex="-1"></a>cm <span class="ot">&lt;-</span> <span class="fu">rbind</span>(<span class="fu">coef</span>(CM),<span class="fu">coef</span>(CSM)) <span class="co"># Coefficient matrix</span></span>
<span id="cb125-3"><a href="#cb125-3" aria-hidden="true" tabindex="-1"></a><span class="fu">c</span>(<span class="sc">-</span><span class="fu">solve</span>(<span class="fu">cbind</span>(cm[,<span class="dv">2</span>],<span class="sc">-</span><span class="dv">1</span>)) <span class="sc">%*%</span> cm[,<span class="dv">1</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 7.373952 0.286310</code></pre>
</div>
<div class="sourceCode" id="cb127"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb127-1"><a href="#cb127-1" aria-hidden="true" tabindex="-1"></a>lmplot <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb127-2"><a href="#cb127-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>treatment.x <span class="sc">==</span> <span class="st">"CS"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb127-3"><a href="#cb127-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>, <span class="at">y =</span> SumAbund, <span class="at">color =</span> treatment.x)) <span class="sc">+</span></span>
<span id="cb127-4"><a href="#cb127-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb127-5"><a href="#cb127-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="at">method =</span> lm, <span class="at">se =</span> <span class="cn">FALSE</span>) <span class="sc">+</span></span>
<span id="cb127-6"><a href="#cb127-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">x =</span> <span class="fl">7.373952</span>, <span class="at">y =</span> <span class="fl">0.286310</span>, <span class="at">color =</span> <span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb127-7"><a href="#cb127-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"text"</span>, <span class="at">x =</span> <span class="fl">8.1</span>, <span class="at">y =</span> <span class="fl">0.24</span>, <span class="at">label =</span> <span class="st">"7.37 (mL/s)"</span>) <span class="sc">+</span></span>
<span id="cb127-8"><a href="#cb127-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_cor</span>(<span class="fu">aes</span>(<span class="at">label =</span> <span class="fu">paste</span>(..rr.label.., ..p.label.., <span class="at">sep =</span> <span class="st">"~`,`~"</span>)), </span>
<span id="cb127-9"><a href="#cb127-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">label.x =</span> <span class="dv">3</span>, <span class="at">show.legend =</span> F) <span class="sc">+</span></span>
<span id="cb127-10"><a href="#cb127-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Flow (mL/s)"</span>, <span class="at">y =</span> <span class="st">"Relative abundance (%)"</span>) <span class="sc">+</span></span>
<span id="cb127-11"><a href="#cb127-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">paste0</span>(x<span class="sc">*</span><span class="dv">100</span>, <span class="st">"%"</span>)) <span class="sc">+</span></span>
<span id="cb127-12"><a href="#cb127-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"segment"</span>, <span class="at">x =</span> <span class="fl">7.373952</span>, <span class="at">xend =</span> <span class="fl">7.6</span>, <span class="at">y =</span> <span class="fl">0.286310</span>, <span class="at">yend =</span> <span class="fl">0.26</span>,</span>
<span id="cb127-13"><a href="#cb127-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">colour =</span> <span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb127-14"><a href="#cb127-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">name =</span> <span class="st">"Treatment"</span>, <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"CSL"</span>, <span class="st">"CL"</span>), <span class="at">values =</span> <span class="fu">c</span>(<span class="st">"darkgreen"</span>, <span class="st">"tan4"</span>)) <span class="sc">+</span></span>
<span id="cb127-15"><a href="#cb127-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() </span>
<span id="cb127-16"><a href="#cb127-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb127-17"><a href="#cb127-17" aria-hidden="true" tabindex="-1"></a>lmplot</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`geom_smooth()` using formula 'y ~ x'</code></pre>
</div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-92-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb129"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb129-1"><a href="#cb129-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="at">filename =</span> <span class="st">"Figures/SupFig6.png"</span>, <span class="at">plot =</span> <span class="fu">last_plot</span>(), <span class="at">device =</span> <span class="st">"png"</span>, <span class="at">width =</span> <span class="fl">6.5</span>, <span class="at">height =</span> <span class="fl">7.5</span>, <span class="at">units =</span> <span class="st">"in"</span>, <span class="at">dpi =</span> <span class="dv">350</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stderr">
<pre><code>`geom_smooth()` using formula 'y ~ x'</code></pre>
</div>
</div>
</section>
<section id="supp-figure-s7" class="level1">
<h1>Supp Figure S7</h1>
<p>If we plot MAB vs.&nbsp;flow, do prairie strips have an impact?</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-94_01b5e934735627a7ecdb9b3706dd6a9f">

</div>
<p>Look at differences in sample volume, time to collect ::: {.cell hash=“ManuscriptWalkthrough_cache/html/unnamed-chunk-96_66549b02ddee0c0a1f13b5db9ce4e20e”}</p>
<div class="sourceCode" id="cb131"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb131-1"><a href="#cb131-1" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(df)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] "treatment.x"             "plot"                   
[3] "sample_number"           "SumAbund"               
[5] "time_after_runoff_start" "collection_time_sec"    
[7] "jare_tare_wt_g"          "jare_plus_runoff_wt_g"  
[9] "flow_g/s"               </code></pre>
</div>
<div class="sourceCode" id="cb133"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb133-1"><a href="#cb133-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(df, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.factor</span>(sample_number), <span class="at">y =</span> collection_time_sec, <span class="at">fill =</span> <span class="fu">as.factor</span>(plot))) <span class="sc">+</span></span>
<span id="cb133-2"><a href="#cb133-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb133-3"><a href="#cb133-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_grid</span>(<span class="sc">~</span> treatment.x) <span class="sc">+</span></span>
<span id="cb133-4"><a href="#cb133-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">"Sample Collection time"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-96-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb134"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb134-1"><a href="#cb134-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(df, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.factor</span>(sample_number), <span class="at">y =</span> jare_plus_runoff_wt_g <span class="sc">-</span> jare_tare_wt_g, <span class="at">fill =</span> <span class="fu">as.factor</span>(plot))) <span class="sc">+</span></span>
<span id="cb134-2"><a href="#cb134-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb134-3"><a href="#cb134-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_grid</span>(<span class="sc">~</span> treatment.x) <span class="sc">+</span></span>
<span id="cb134-4"><a href="#cb134-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">"Sample Volume"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-96-2.png" class="img-fluid" width="672"></p>
</div>
<p>:::</p>
<p>A simple scatter plot looking at flow vs abundance, with a lm line fitted to each treatment.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-98_664af1af7427b3b6ee3b488b8ea20c74">
<div class="cell-output-stdout">
<pre><code>[1] "numeric"</code></pre>
</div>
<div class="cell-output-stderr">
<pre><code>`geom_smooth()` using formula 'y ~ x'</code></pre>
</div>
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-98-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>It appears that up to a flow of 7.5 g/s, strips may be effective at removing MAB in runoff water when compared to if no strips are present.</p>
<p>Calculate corrected (cor) MAB abundance by dividing MAB abundance by flow in each sample. Then calculate the treatment mean, sd, se and interquartile.</p>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-100_a88adbd208c87ad96498c989a810b2b3">

</div>
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-102_4b37b0a6e53e21b0e92ea10d17139dcf">
<div class="cell-output-display">
<p><img src="ManuscriptWalkthrough_files/figure-html/unnamed-chunk-102-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Here, despite the error bars, we have indications that when strips are present, there is less MAB in runoff water.</p>
<p>Event mean concentration (EMC)</p>
<p>EMC calculation</p>
<p>For a plot we have</p>
<ol type="1">
<li><p>6 water samples</p></li>
<li><p>Mass (vol) of water collected at each sample</p></li>
<li><p>Time to collect each sample</p></li>
<li><p>Count of ARGs filtered out of sample</p></li>
</ol>
<p>The calculation</p>
<p>Event load (EL)</p>
<p><span class="math display">\[
=\frac{{T1ARGConc}*{T1Vol}+{T2ARGConc}*{T2Vol}+{T3ARGConc}*{T3Vol}+{T4ARGConc}*{T4Vol}+{T5ARGConc}*{T5Vol}+{T6ARGConc}*{T6Vol}}{Event TotalVol}
\]</span></p>
<!-- # ```{r} -->
<!-- # # Load copy number from HT-qPCR runs -->
<!-- # # If you haven't calculated copy numbers yet, -->
<!-- # # see: \#\#\#\#\#\#\# -->
<!-- # water_counts <- read_csv("Data/WORWATERARGCOUNTS.csv") -->
<!-- #  -->
<!-- # head(water_counts) -->
<!-- # ``` -->
<div class="cell" data-hash="ManuscriptWalkthrough_cache/html/unnamed-chunk-104_651c63b11e413ae0d270cf08fd4bfa3e">
<div class="sourceCode" id="cb137"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb137-1"><a href="#cb137-1" aria-hidden="true" tabindex="-1"></a>watervols <span class="ot">&lt;-</span> df <span class="sc">%&gt;%</span></span>
<span id="cb137-2"><a href="#cb137-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(treatment.x, plot, sample_number, time_after_runoff_start, collection_time_sec, jare_tare_wt_g, jare_plus_runoff_wt_g, <span class="st">`</span><span class="at">flow_g/s</span><span class="st">`</span>) <span class="sc">%&gt;%</span></span>
<span id="cb137-3"><a href="#cb137-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">sample_volume =</span> jare_plus_runoff_wt_g <span class="sc">-</span> jare_tare_wt_g)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<!-- <!-- ```{r} -->
<p>–&gt; <!-- <!-- # Join the two by plot and sample_number --> –&gt; <!-- <!-- # Select wanted columns and convert joining variables to factor --> –&gt;</p>
<!-- <!-- # left side -->
<p>–&gt; <!-- <!-- ls <- watervols %>% --> –&gt; <!-- <!--   group_by(plot) %>% --> –&gt; <!-- <!--   mutate(plot_volume = sum(sample_volume), plot = as.factor(plot), sample_number = as.factor(sample_number)) %>% --> –&gt; <!-- <!--   select(treatment.x, plot, sample_number, sample_volume, plot_volume, collection_time_sec, `flow_g/s`) %>% --> –&gt; <!-- <!--   ungroup() --> –&gt;</p>
<!-- <!-- # right side -->
<p>–&gt; <!-- <!-- rs <- norm %>% --> –&gt; <!-- <!--   select(plot, sample_number, Assay, counts) %>% --> –&gt; <!-- <!--   mutate(plot = as.factor(plot), sample_number = as.factor(sample_number)) --> –&gt;</p>
<!-- <!-- # joined -->
<p>–&gt; <!-- <!-- df <- ls %>% --> –&gt; <!-- <!--   left_join(rs, by = c("plot", "sample_number"))  --> –&gt;</p>
<!-- <!-- plotvolumes <- df %>% -->
<p>–&gt; <!-- <!--   select(Plot = plot, plot_volume) %>% --> –&gt; <!-- <!--   distinct() --> –&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- ```{r} -->
<!-- df <- df %>%  -->
<!--   group_by(plot, Assay, sample_number) %>% -->
<!--   mutate(copies_mL = counts / sample_volume, -->
<!--          copies_sec = counts / collection_time_sec, -->
<!--          fcopies = sample_volume * copies_mL, -->
<!--          tcopies = collection_time_sec * copies_sec, -->
<!--          unitload = copies_mL * `flow_g/s` * collection_time_sec, # Unit load, sum this for load calculation for each plot -->
<!--          rep = seq_along(Assay)) %>% -->
<!--   ungroup()  -->
<!-- ``` -->
<!-- We had three technical replicates for each assay and water sample. -->
<!-- Event mean treatment concentration -->
<!-- ```{r} -->
<!-- mycomps <- list(c("Crop + Strip", "Crop + Strip + Manure"), c("Crop + Strip", "Crop + Manure"), c("Crop + Strip + Manure", "Crop + Manure")) -->
<!-- df %>% -->
<!--   filter(Assay == "16S_Eub_338_518") %>% -->
<!--   group_by(plot, rep) %>% -->
<!--   mutate(plot_sum_mean_copies_mL = sum(fcopies), -->
<!--          plot_load_copies = plot_sum_mean_copies_mL / plot_volume) %>% -->
<!--   select(treatment.x, plot, rep, plot_load_copies) %>% -->
<!--   distinct() %>% -->
<!--   arrange(plot) %>% -->
<!--   ggboxplot(x = "treatment.x", y = "plot_load_copies") + -->
<!--   ggtitle("Event mean conc. 16S (copies/mL)") + -->
<!--   stat_compare_means(comparisons = mycomps) + -->
<!--   stat_compare_means(label.y = 3.5) -->
<!-- ``` -->
<!-- ```{r} -->
<!-- EMC <- function(df, gene){ -->
<!--   mycomps <- list(c("Crop + Strip", "Crop + Strip + Manure"),  -->
<!--                   c("Crop + Strip", "Crop + Manure"),  -->
<!--                   c("Crop + Strip + Manure", "Crop + Manure")) -->
<!--   df %>% -->
<!--     filter(Assay == gene) %>% -->
<!--     group_by(plot, rep) %>% -->
<!--     mutate(plot_sum_mean_copies_mL = sum(fcopies), -->
<!--            plot_load_copies = plot_sum_mean_copies_mL / plot_volume) %>% -->
<!--     select(treatment.x, plot, rep, plot_load_copies) %>% -->
<!--     distinct() %>% -->
<!--     arrange(plot) %>% -->
<!--     ggboxplot(x = "treatment.x", y = "plot_load_copies") + -->
<!--     scale_y_continuous(trans = "log10") + -->
<!--     ggtitle(paste0("Event mean concentration for ", gene, " (copies/mL)")) + -->
<!--     labs(x="Treatment",y="gene copies/mL") + -->
<!--     stat_compare_means(comparisons = mycomps) + -->
<!--     stat_compare_means() -->
<!-- } -->
<!-- PlotLoad <- function(df, gene){ -->
<!--     mycomps <- list(c("Crop + Strip", "Crop + Strip + Manure"),  -->
<!--                   c("Crop + Strip", "Crop + Manure"),  -->
<!--                   c("Crop + Strip + Manure", "Crop + Manure")) -->
<!--   df %>% -->
<!--     filter(Assay == gene) %>% -->
<!--     group_by(plot, rep) %>% -->
<!--     mutate(load = sum(unitload)) %>% -->
<!--     select(treatment.x, plot, rep, load) %>% -->
<!--     distinct() %>% -->
<!--     arrange(plot) %>% -->
<!--     ggboxplot(x = "treatment.x", y = "load") + -->
<!--     ggtitle(paste0("Event load for ", gene, " (copies)")) + -->
<!--     labs(x="Treatment",y="Event gene load (copies)") + -->
<!--     stat_compare_means(comparisons = mycomps) + -->
<!--     stat_compare_means() -->
<!-- } -->
<!-- ``` -->
<!-- Event mean concentration -->
<!-- ```{r} -->
<!-- AllGeneEMC <- unique(df$Assay) %>% -->
<!--   map( ~ EMC(df, gene = .x))  -->
<!-- AllGeneEMC -->
<!-- ``` -->
<!-- Event load -->
<!-- For a plot, the sum of concentration * flow * time -->
<!-- ```{r} -->
<!-- AllGeneloads <- unique(df$Assay) %>% -->
<!--   map( ~ PlotLoad(df, gene = .x))  -->
<!-- AllGeneloads -->
<!-- ``` -->
<!-- ```{r} -->
<!-- plot <- df %>% -->
<!--   filter(!treatment.x == "Crop + Strip", Assay == "tetL") %>% -->
<!--   group_by(treatment.x, sample_number) %>% -->
<!--   summarise(  -->
<!--     n = n(), -->
<!--     mean = mean(fcopies), -->
<!--     sd = sd(fcopies) -->
<!--   ) %>% -->
<!--   mutate(se = sd/sqrt(n))  %>% -->
<!--   mutate(ic = se * qt((1-0.05)/2 + .5, n-1)) %>% -->
<!--   ggplot(aes(x = sample_number,  -->
<!--              y = mean,  -->
<!--              ymin = mean - ic,  -->
<!--              ymax = mean + ic, -->
<!--              color = treatment.x, -->
<!--              group = treatment.x)) + -->
<!--   geom_point(position = position_dodge(width = .1), size = 2) + -->
<!--   geom_line(position = position_dodge(width = .1), size = 2) + -->
<!--   geom_errorbar(position = position_dodge(width = .1), width = .2) + -->
<!--   labs(y = "tetL copies", x = "Sample Number", title = "Volume normalized tetL", subtitle = "Divide tetL by volume") + -->
<!--   theme(axis.text.x = element_text(angle = 45, hjust = 1)) + -->
<!--   scale_color_manual(values = c("darkgreen", "tan4"), name = "Treatment") + -->
<!--   theme_bw() -->
<!-- plot -->
<!-- ``` -->
<!-- <!-- 16S copies per mL -->
<p>–&gt;</p>
<!-- <!-- ```{r} -->
<p>–&gt; <!-- <!-- plot <- df %>% --> –&gt; <!-- <!--   filter(!treatment.x == "Crop + Strip", Assay == "16S_Eub_338_518") %>% --> –&gt; <!-- <!--   group_by(treatment.x, sample_number) %>% --> –&gt; <!-- <!--   summarise(  --> –&gt; <!-- <!--     n = n(), --> –&gt; <!-- <!--     mean = mean(fcopies), --> –&gt; <!-- <!--     sd = sd(fcopies) --> –&gt; <!-- <!--   ) %>% --> –&gt; <!-- <!--   mutate(se = sd/sqrt(n))  %>% --> –&gt; <!-- <!--   mutate(ic = se * qt((1-0.05)/2 + .5, n-1)) %>% --> –&gt; <!-- <!--   ggplot(aes(x = sample_number,  --> –&gt; <!-- <!--              y = mean,  --> –&gt; <!-- <!--              ymin = mean - ic,  --> –&gt; <!-- <!--              ymax = mean + ic, --> –&gt; <!-- <!--              color = treatment.x, --> –&gt; <!-- <!--              group = treatment.x)) + --> –&gt; <!-- <!--   geom_point(position = position_dodge(width = .1), size = 2) + --> –&gt; <!-- <!--   geom_line(position = position_dodge(width = .1), size = 2) + --> –&gt; <!-- <!--   geom_errorbar(position = position_dodge(width = .1), width = .2) + --> –&gt; <!-- <!--   labs(y = "16S copies", x = "Sample Number", title = "Volume normalized 16S", subtitle = "Divide 16S by volume") + --> –&gt; <!-- <!--   theme(axis.text.x = element_text(angle = 45, hjust = 1)) + --> –&gt; <!-- <!--   scale_color_manual(values = c("darkgreen", "tan4"), name = "Treatment") + --> –&gt; <!-- <!--   theme_bw() --> –&gt; <!-- <!-- plot --> –&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- Copies/mL heatmap -->
<p>–&gt;</p>
<!-- <!-- ```{r} -->
<p>–&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- ```{r} -->
<p>–&gt; <!-- <!-- df$treatment.x <- factor(df$treatment.x, levels = c("Crop + Strip", "Crop + Strip + Manure", "Crop + Manure")) --> –&gt;</p>
<!-- <!-- df$copies_mL[df$copies_mL <= 7.151370e-142] <- NA -->
<p>–&gt;</p>
<!-- <!-- heatmap <- df %>% -->
<p>–&gt; <!-- <!--   filter(!Assay %in% c("intI1F165_clinical")) %>% --> –&gt; <!-- <!--   ggplot(aes(as.factor(sample_number), Assay)) + --> –&gt; <!-- <!--   geom_tile(aes(fill = copies_mL), colour = "white", size = 0.1) + --> –&gt; <!-- <!--   scale_fill_viridis(name = "Gene copies per mL", option = "plasma")  + --> –&gt; <!-- <!--   labs(x = "Sample Number", y = "Gene") + --> –&gt; <!-- <!--   facet_nested(~ treatment.x + plot, scales = "free", space = "free") + theme_classic() + --> –&gt; <!-- <!--   theme( --> –&gt; <!-- <!--     legend.background = element_rect( --> –&gt; <!-- <!--       fill = (alpha = 0), --> –&gt; <!-- <!--       color = 'black', --> –&gt; <!-- <!--       size = 0.1), --> –&gt; <!-- <!--     panel.background = element_rect(color = 'black', size = 1.4), --> –&gt; <!-- <!--     strip.text.x = element_text(size = 10, face = 'bold'), --> –&gt; <!-- <!--     strip.background = element_rect(colour = 'black', size = 1.4), --> –&gt; <!-- <!--     axis.text.x = element_text(angle = 90) --> –&gt; <!-- <!--   )  --> –&gt;</p>
<!-- <!-- heatmap -->
<p>–&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- ![](images/WaterARGnormcountHeat.png) -->
<p>–&gt;</p>
<!-- <!-- Heatmap of ARG counts normalized by 16s counts -->
<p>–&gt;</p>
<!-- <!-- Relative abundance of ARGs -->
<p>–&gt;</p>
<!-- <!-- DeltaCt = Ct(ARG) - Ct(16S) -->
<p>–&gt; <!-- <!-- Relative Abundance = 2 ^ (-DeltaCt) --> –&gt; <!-- <!-- Genes below detection limit were given a delatCt of 20, higher than any other  --> –&gt; <!-- <!-- deltaCt --> –&gt;</p>
<!-- <!-- ```{r setwd and read data, include=FALSE} -->
<p>–&gt; <!-- <!-- df <- readRDS("data/metawithbiomark.RDS")  --> –&gt; <!-- <!-- # Make Value column numeric --> –&gt; <!-- <!-- df$Value <- as.numeric(df$Value) --> –&gt; <!-- <!-- unique(df$Assay) --> –&gt; <!-- <!-- unique(df$Sample_Name) --> –&gt; <!-- <!-- # Rename two Assays and Sample_Names, tetG_F and sul1 NEW --> –&gt; <!-- <!-- df$Assay <- recode(df$Assay, --> –&gt; <!-- <!--                    "sul1 NEW" = "sul1", --> –&gt; <!-- <!--                    "tetG_F" = "tetG") --> –&gt; <!-- <!-- df$Sample_Name <- recode(df$Sample_Name, --> –&gt; <!-- <!--                    "sul1 NEW" = "sul1", --> –&gt; <!-- <!--                    "tetG_F" = "tetG") --> –&gt; <!-- <!-- df$treatment --> –&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- ```{r subset biomark} -->
<p>–&gt; <!-- <!-- water <- df %>% --> –&gt; <!-- <!--   filter(sample_type %in% "water")  --> –&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- ```{r} -->
<p>–&gt; <!-- <!-- WaterCT <- water %>% --> –&gt; <!-- <!--   select(Sample_Name, Assay, Value, treatment, plot, sample_number) %>% --> –&gt; <!-- <!--   group_by(Sample_Name, Assay) %>% --> –&gt; <!-- <!--   mutate(rep = seq_along(Assay)) %>% --> –&gt; <!-- <!--   ungroup() %>% --> –&gt; <!-- <!--   group_by(Sample_Name, rep) %>% --> –&gt; <!-- <!--   mutate(deltaCt = Value - Value[Assay == "16S_Eub_338_518"]) # Delta CT calculation --> –&gt;</p>
<!-- <!-- WaterCT$deltaCt[WaterCT$deltaCt > 35] <- 20 -->
<p>–&gt; <!-- <!-- WaterCT$deltaCt[WaterCT$deltaCt == 0] <- 20 --> –&gt;</p>
<!-- <!-- WaterCT <- WaterCT %>% -->
<p>–&gt; <!-- <!--   group_by(Sample_Name, rep) %>% --> –&gt; <!-- <!--   mutate(relative_abundance = 2 ^ (- deltaCt), --> –&gt; <!-- <!--          log10relative_abundance = log10(relative_abundance + 0.0001)) %>%# relative abundance calculation --> –&gt; <!-- <!--   ungroup() %>% --> –&gt; <!-- <!--   filter(!Assay %in% c("16S_Eub_338_518", "intI1F165_clinical", "aac3-Via"), --> –&gt; <!-- <!--          !Sample_Name == "Supply_Water")  --> –&gt;</p>
<!-- <!-- ``` -->
<p>–&gt; <!-- <!-- Relative abundance ARG heatmap --> –&gt;</p>
<!-- <!-- ```{r} -->
<p>–&gt; <!-- <!-- WaterCT <- WaterCT %>% # Add classes to ARGs --> –&gt; <!-- <!--   mutate(gene_class = case_when(Assay %in% unique(WaterCT$Assay[grepl("^[aa]", WaterCT$Assay)]) ~ "AMG", --> –&gt; <!-- <!--                                 Assay %in% unique(WaterCT$Assay[grepl("^[erm]", WaterCT$Assay)]) ~ "MLSB", --> –&gt; <!-- <!--                                 Assay %in% unique(WaterCT$Assay[grepl("^[sul]", WaterCT$Assay)]) ~ "Sulfa", --> –&gt; <!-- <!--                                 Assay %in% unique(WaterCT$Assay[grepl("tet", WaterCT$Assay)]) ~ "Tet", --> –&gt; <!-- <!--                                 Assay %in% c("intl1", "intl2", "intl3", "IS1247", "IS6100", "tnpA2", --> –&gt; <!-- <!--                                               "tnpA5", "tnpA6", "tnpA3", "intI1F165_clinical", "tnpA1", "int1-a-marko", "tnpA7") ~ "MGE",  --> –&gt; <!-- <!--                                 Assay %in% c("cadC", "cmr") ~ "HM", --> –&gt; <!-- <!--                                 Assay == c("16S_Eub_338_518") ~ "16S")) --> –&gt;</p>
<!-- <!-- WaterCT$gene_class_f = factor(WaterCT$gene_class, # Order the classes -->
<p>–&gt; <!-- <!--                                 levels = c('AMG', 'MLSB', 'Sulfa', "Tet", "MGE", "HM", "16S")) --> –&gt;</p>
<!-- <!-- test <- WaterCT %>% # Order ARGs based on class -->
<p>–&gt; <!-- <!--   arrange(gene_class)  --> –&gt;</p>
<!-- <!-- levels <- dput(levels(factor(test$Assay))) # dput ftw!!! -->
<p>–&gt;</p>
<!-- <!-- heatmap <- ggplot(WaterCT, aes(as.factor(sample_number), ordered(Assay, levels = levels))) + -->
<p>–&gt; <!-- <!--   geom_tile(aes( --> –&gt; <!-- <!--     fill = relative_abundance),  --> –&gt; <!-- <!--     colour = "white",  --> –&gt; <!-- <!--     size = 0.1) + --> –&gt; <!-- <!--   labs( --> –&gt; <!-- <!--     x = "Sample Time (min)",  --> –&gt; <!-- <!--     y = "Gene") + --> –&gt; <!-- <!--   scale_fill_gradient( --> –&gt; <!-- <!--     name = "Relative Abundance", # changes legend title --> –&gt; <!-- <!--     low = "white", --> –&gt; <!-- <!--     high = "red", --> –&gt; <!-- <!--     limit = c(min(WaterCT$relative_abundance), max(WaterCT$relative_abundance))) + --> –&gt; <!-- <!--   facet_nested( --> –&gt; <!-- <!--     gene_class_f ~ treatment + plot,  --> –&gt; <!-- <!--                scales = "free",  --> –&gt; <!-- <!--                space = "free") + theme_classic() + --> –&gt; <!-- <!--   theme( --> –&gt; <!-- <!--     legend.background = element_rect( --> –&gt; <!-- <!--       fill = (alpha = 0), --> –&gt; <!-- <!--       color = 'black', --> –&gt; <!-- <!--       size = 0.1), --> –&gt; <!-- <!--     panel.background = element_rect(color = 'black', size = 1.4), --> –&gt; <!-- <!--     strip.text.x = element_text(size = 10, face = 'bold'), --> –&gt; <!-- <!--     strip.background = element_rect(colour = 'black', size = 1.4) --> –&gt; <!-- <!--   )  --> –&gt; <!-- <!-- heatmap --> –&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- tetL  -->
<p>–&gt;</p>
<!-- <!-- ```{r} -->
<p>–&gt;</p>
<!-- <!-- sum_stats <- WaterCT %>% -->
<p>–&gt; <!-- <!--   group_by(treatment, Assay) %>% --> –&gt; <!-- <!--   get_summary_stats(relative_abundance, type = "mean_se") --> –&gt;</p>
<!-- <!-- sum_stats -->
<p>–&gt;</p>
<!-- <!-- res.kruskal <- WaterCT %>% -->
<p>–&gt; <!-- <!--   filter(Assay == "tetL") %>% --> –&gt; <!-- <!--   kruskal_test(relative_abundance ~ treatment) --> –&gt;</p>
<!-- <!-- pwc <- WaterCT %>% -->
<p>–&gt; <!-- <!--   filter(Assay == "tetL") %>% --> –&gt; <!-- <!--   dunn_test(relative_abundance ~ treatment, p.adjust.method = "hochberg", detailed = T) --> –&gt;</p>
<!-- <!-- pwc -->
<p>–&gt;</p>
<!-- <!-- pwc <- pwc %>% add_xy_position(x = "treatment") -->
<p>–&gt;</p>
<!-- <!-- WaterCT %>% -->
<p>–&gt; <!-- <!--   filter(Assay == "tetL") %>% --> –&gt; <!-- <!--   ggboxplot(x = "treatment", y = "relative_abundance") + --> –&gt; <!-- <!--   stat_pvalue_manual(pwc) + --> –&gt; <!-- <!--   labs( --> –&gt; <!-- <!--     subtitle = get_test_label(res.kruskal, detailed = TRUE), --> –&gt; <!-- <!--     caption = get_pwc_label(pwc) --> –&gt; <!-- <!--     )  --> –&gt;</p>
<!-- <!-- pwc %>% -->
<p>–&gt; <!-- <!--   select(group1, group2, p.adj, p.adj.signif) %>% --> –&gt; <!-- <!--   mutate(P = round(p.adj, digits = 10))  --> –&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- <!-- log10 transform?  -->
<p>–&gt; <!-- <!-- ```{r} --> –&gt;</p>
<!-- <!-- sum_stats <- WaterCT %>% -->
<p>–&gt; <!-- <!--   group_by(treatment, Assay) %>% --> –&gt; <!-- <!--   get_summary_stats(log10relative_abundance, type = "mean_se") --> –&gt;</p>
<!-- <!-- sum_stats -->
<p>–&gt;</p>
<!-- <!-- res.kruskal <- WaterCT %>% -->
<p>–&gt; <!-- <!--   filter(Assay == "tetL") %>% --> –&gt; <!-- <!--   kruskal_test(log10relative_abundance ~ treatment) --> –&gt;</p>
<!-- <!-- pwc <- WaterCT %>% -->
<p>–&gt; <!-- <!--   filter(Assay == "tetL") %>% --> –&gt; <!-- <!--   dunn_test(log10relative_abundance ~ treatment, p.adjust.method = "hochberg", detailed = T) --> –&gt;</p>
<!-- <!-- pwc -->
<p>–&gt;</p>
<!-- <!-- pwc <- pwc %>% add_xy_position(x = "treatment") -->
<p>–&gt;</p>
<!-- <!-- WaterCT %>% -->
<p>–&gt; <!-- <!--   filter(Assay == "tetL") %>% --> –&gt; <!-- <!--   ggboxplot(x = "treatment", y = "log10relative_abundance") + --> –&gt; <!-- <!--   stat_pvalue_manual(pwc) + --> –&gt; <!-- <!--   labs( --> –&gt; <!-- <!--     subtitle = get_test_label(res.kruskal, detailed = TRUE), --> –&gt; <!-- <!--     caption = get_pwc_label(pwc) --> –&gt; <!-- <!--     )  --> –&gt;</p>
<!-- <!-- pwc %>% -->
<p>–&gt; <!-- <!--   select(group1, group2, p.adj, p.adj.signif) %>% --> –&gt; <!-- <!--   mutate(P = round(p.adj, digits = 10))  --> –&gt; <!-- <!-- ``` --> –&gt; <!-- <!-- ```{r} --> –&gt; <!-- <!-- plotrelafunction <- function(data, gene){ --> –&gt; <!-- <!--   res.kruskal <- data %>% --> –&gt; <!-- <!--     filter(Assay == gene) %>% --> –&gt; <!-- <!--     kruskal_test(relative_abundance ~ treatment) --> –&gt;</p>
<!-- <!--   pwc <- data %>% -->
<p>–&gt; <!-- <!--     filter(Assay == gene) %>% --> –&gt; <!-- <!--     dunn_test(relative_abundance ~ treatment, p.adjust.method = "hochberg", detailed = T) --> –&gt;</p>
<!-- <!--   pwc -->
<p>–&gt;</p>
<!-- <!--   pwc <- pwc %>% add_xy_position(x = "treatment") -->
<p>–&gt;</p>
<!-- <!--   plot <-  data %>% -->
<p>–&gt; <!-- <!--     filter(Assay == gene) %>% --> –&gt; <!-- <!--     ggboxplot(x = "treatment", y = "relative_abundance") + --> –&gt; <!-- <!--    stat_pvalue_manual(pwc) + --> –&gt; <!-- <!--    labs( --> –&gt; <!-- <!--       title = paste0("Relative abundance of ", gene), --> –&gt; <!-- <!--       subtitle = get_test_label(res.kruskal, detailed = TRUE), --> –&gt; <!-- <!--       caption = get_pwc_label(pwc) --> –&gt; <!-- <!--       )  --> –&gt;</p>
<!-- <!-- return(plot) -->
<p>–&gt; <!-- <!-- } --> –&gt; <!-- <!-- ``` --> –&gt; <!-- <!-- ```{r} --> –&gt; <!-- <!-- allgenerela <- unique(WaterCT$Assay) %>% --> –&gt; <!-- <!--   map( ~ plotrelafunction(WaterCT, gene = .x))  --> –&gt;</p>
<!-- <!-- allgenerela -->
<p>–&gt; <!-- <!-- ``` --> –&gt;</p>
<!-- ```{r} -->
<!-- water_flow_data <- read_csv("Data/water_flow_data.csv") %>%  -->
<!--    mutate(Plot = as.factor(Plot)) -->
<!-- ``` -->
<!-- We collected all runoff, save that collected for each sample. Per plot runoff totals: -->
<!-- ```{r} -->
<!-- colnames(water_flow_data) -->
<!-- water_flow_data <- water_flow_data %>% -->
<!--   left_join(plotvolumes, by = "Plot") -->
<!-- ``` -->
<!-- Volume applied to plot based on average reading of 4 rain gauges multiplied by plot area (3 m^2): -->
<!-- ```{r} -->
<!-- ggplot(water_flow_data, aes(x = Plot, y = `WaterVolumeApplied(mL)`)) + -->
<!--   geom_bar(stat = "identity") + -->
<!--      geom_text(aes(label = `WaterVolumeApplied(mL)`), position=position_dodge(width=0.9), vjust=-0.25) -->
<!-- ``` -->
<!-- mass/volume of runoff collected in each plot, the sum of collected samples and runoff bucket weights    -->
<!-- ```{r} -->
<!-- plotrunoff <- ggplot(water_flow_data, aes(x = Plot, y = `TotalRunoff(g)` + plot_volume)) + -->
<!--   geom_bar(stat = "identity") + -->
<!--      geom_text(aes(label = `TotalRunoff(g)` + plot_volume), position=position_dodge(width=0.9), vjust=-0.25) -->
<!-- plotrunoff -->
<!-- MAB <- read_csv("Data/water_flow_with_MAB.csv") -->
<!-- MABRUN <- MAB %>% -->
<!--   select(treatment.x, Plot = plot, Abundance) %>% -->
<!--   mutate(Plot = as.factor(Plot)) %>% -->
<!--   left_join(water_flow_data, by = "Plot") %>% -->
<!--   group_by(Plot) %>% -->
<!--   mutate(Abundance = mean(Abundance)) %>% -->
<!--   select(treatment.x, Plot, Abundance, `TotalRunoff(g)`, plot_volume, `WaterVolumeApplied(mL)`) %>% -->
<!--   distinct() -->
<!-- plot <- MABRUN %>% -->
<!--   filter(!treatment.x == "Crop + Strip") %>% -->
<!--   ggplot(aes(x = plot_volume + `TotalRunoff(g)`, y = Abundance, color = treatment.x)) + -->
<!--   geom_point() + -->
<!--   geom_smooth(method = lm, se = FALSE)  + -->
<!--   scale_color_manual(values = c("darkgreen", "tan4"), name = "Treatment") + -->
<!--   theme_bw() -->
<!-- plot -->
<!-- plot <- MABRUN %>% -->
<!--   filter(!treatment.x == "Crop + Strip") %>% -->
<!--   ggplot(aes(x = ((`TotalRunoff(g)` + plot_volume) / `WaterVolumeApplied(mL)`), y = Abundance, color = treatment.x)) + -->
<!--   geom_point() + -->
<!--   geom_smooth(method = loess, se = FALSE)  + -->
<!--   scale_color_manual(values = c("darkgreen", "tan4"), name = "Treatment") + -->
<!--   theme_bw() + -->
<!--   labs(x = "Percentage of water as runoff", y = "Relative abundance of MAB") -->
<!-- plot -->
<!-- ``` -->
<!-- divide runoff volume by volume applied, runoff rate -->
<p>–&gt;</p>
<!-- i.e. 9.5% of water applied to plot 4 was collected as runoff -->
<!-- ```{r} -->
<!-- ggplot(water_flow_data, aes(x = Plot, y = round((`TotalRunoff(g)` + plot_volume) / `WaterVolumeApplied(mL)`, digits = 4) * 100)) + -->
<!--   geom_bar(stat = "identity") + -->
<!--      geom_text(aes(label = round((`TotalRunoff(g)` + plot_volume) / `WaterVolumeApplied(mL)`, digits = 4) * 100) , position=position_dodge(width=0.9), vjust=-0.25) + -->
<!--   labs(y = "Rate of runoff (%)") -->
<!-- ggsave(filename = "Figures/runoffrate.png", plot = last_plot(), device = "png", width = 6.5, height = 7.5, units = "in", dpi = 350) -->
<!-- ``` -->


</section>
</main> <!-- /main -->
<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    setTimeout(function() {
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
  </div>
  <div class="nav-page nav-page-next">
  </div>
</nav>
</div> <!-- /content -->


</body></html>